<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Lit Digest - Non-coding variants in regulatory regions &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../../../../css/poole.css">
  <link rel="stylesheet" href="../../../../css/syntax.css">
  <link rel="stylesheet" href="../../../../css/lanyon.css">
  <link rel="stylesheet" href="../../../../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../../../../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../../assets/R-icon.png">
  <link rel="shortcut icon" href="../../../../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../../../../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../../../../">Home</a>
    <a class="sidebar-nav-item " href="../../../../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../../../../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../../../../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../../../../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="post">
  <h1 class="post-title">Lit Digest - Non-coding variants in regulatory regions</h1>
  <span class="post-date">Sep 22 2017
  
  
  
  
  
  <a href='../../../../tags/literature/' class='label'>literature</a>
  
  
  
  </span>
  


<div id="genome-wide-analysis-of-noncoding-regulatory-mutations-in-cancer" class="section level2">
<h2>Genome-wide analysis of noncoding regulatory mutations in cancer</h2>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25261935">Weinhold &amp; Jacobsen et al., Nature Genetics 2014</a></p>
<ul>
<li>863 tumors from TCGA and co.</li>
<li>Identify <strong>recurrent non-coding variants in cancer</strong>.</li>
<li>PLEKHS1, WDR74, SDHD had such variants in upstream regulatory elements.</li>
</ul>
<p>Promoter/enhancer have <em>similar rate of somatic mutation</em> as transcribed regions (~3 per Mb), and lower than intergenic (~6 per Mb), supporting the presence of functional variants.</p>
<p>They used three approaches to identify functional ncSNV candidates:</p>
<ol style="list-style-type: decimal">
<li>A <strong>hotspot</strong> analysis, aiming at cluster of variants within 50 bp of each other.</li>
<li>A <strong>regional recurrence</strong> analysis, grouping the variants by regions and detecting recurrence compared to neighboring regions or the entire genome.</li>
<li>A <strong>TFBS</strong> analysis, to find mutations that create/disrupts ETS binding sites. Apparently ETS plays a role in cancer.</li>
</ol>
<p>Many of the top hits were located in regulatory regions of known cancer genes (pv=3.4e-4).</p>
<p>The most significant hotspot is in <strong>TERT promoter</strong> and consists of two recurrent mutations across 38 and 15 tumors, and <strong>7 cancer types</strong>.
TERT tended to be <strong>more expressed</strong> in mutated samples.
Fun fact: the C-&gt;T transitions create novel ETS binding sites.</p>
<p>The second most significant hotspot in in <strong>PLEKHS1 promoter</strong>.
Again two sites, with 11 and 12 mutations over 20 samples and <strong>5 cancer types</strong>.
The gene’s <strong>expression was lower</strong> in mutated samples.
In bladder cancer, 40% of the samples carried one of these two mutations.
Fun fact: also a C-&gt;T transition; each site is flanked by 10 bp palindromes.</p>
<p>The regional recurrence identified enrichment in larger regions, grouping the variants by functional class (enhancer, promoter, 5’ UTR, 3’UTR).
They tested both <strong>local enrichment</strong>, comparing the mutation rate in neighboring regions, and <strong>global enrichment</strong>, comparing the mutation rate in the same functional class and with similar DNA replication timing.
<em>I guess DNA replication timing affects the mutation rate ?</em></p>
<p>TERT was again the strongest hit.
The second and third hits were the promoter and 5’ UTR of <strong>WDR74</strong>.
This gene had no coding mutation but 35 non-coding variants in a ~1 Kb region at the promoter region.
No gene expression changes this time.
Overall, 5% of the tumors had a mutation in this region.</p>
<p>The TFBS approach intended to find mutation similar to the TERT examples, that either create or disrupt ETS binding sites.
From a short list of candidates, they focused on <strong>SDHD</strong> which was also high in the regional recurrence list.
In this case, C-&gt;T transitions disrupted a existing ETS binding sites, consistent with a <strong>reduced expression</strong> of the gene.
The mutations occurred <strong>exclusively in melanoma samples</strong>.
Because the variants were close enough to a coding region, they could use exome sequencing data to investigate more samples and even identified a third putative ETS site.
They showed that around 10% of melanoma samples have such a mutation and provided further support that these sites regulate the gene’s expression.
Samples with SDHD mutation had a significantly shorter overall survival.</p>
<p><strong>Interesting study</strong>.
I like how they used the strong/known association in TERT to generalize and identify other with weaker effect.
It also showed the usefulness of performing <strong>different analysis</strong> to better narrow down the list of leads to follow.
The way they <strong>used orthogonal data</strong> (RNA-seq, WES or Encode) to add support to the original findings was good too.
The paper was pleasant to read as well, not too dense and not too jargon heavy.</p>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../../../../assets/highlight.min.js"></script>
    <script src="../../../../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

