<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Genome graphs &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../../../../css/poole.css">
  <link rel="stylesheet" href="../../../../css/syntax.css">
  <link rel="stylesheet" href="../../../../css/lanyon.css">
  <link rel="stylesheet" href="../../../../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../../../../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../../assets/R-icon.png">
  <link rel="shortcut icon" href="../../../../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../../../../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../../../../">Home</a>
    <a class="sidebar-nav-item " href="../../../../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../../../../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../../../../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../../../../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="post">
  <h1 class="post-title">Genome graphs</h1>
  <span class="post-date">Sep 23 2017
  
  
  
  
  
  <a href='../../../../tags/literature/' class='label'>literature</a>
  
  
  
  </span>
  


<div id="why-genome-graphs" class="section level2">
<h2>Why genome graphs ?</h2>
<ul>
<li>Better genome reference.
<ul>
<li>Higher mapping rate in the presence of variation.</li>
<li>Less “biased” toward the fixed linear European reference genome.</li>
<li>Easier to update. No more major version.</li>
</ul></li>
<li>Better annotation for non-SNV variants.
<ul>
<li>More general and intuitive.</li>
<li>Only one way to place indels.</li>
<li>Complex variants involving several canonical types.</li>
</ul></li>
<li>Improved variant calling.
<ul>
<li>Integrating known variants. Would be useful for SVs with breakpoint resolution.</li>
<li>In highly polymorphic regions.</li>
</ul></li>
</ul>
</div>
<div id="challenges" class="section level2">
<h2>Challenges</h2>
<ul>
<li>Map efficiently reads to graph.</li>
<li>Predict the sequence of the allele(s).</li>
<li>Detect new variation.</li>
<li>Project annotation to graph.</li>
</ul>
<div id="drawbacks" class="section level3">
<h3>Drawbacks</h3>
<ul>
<li>Difficult to integrate CNVs or SVs with imprecise breakpoints.</li>
</ul>
</div>
<div id="other-ideas" class="section level3">
<h3>Other ideas</h3>
<ul>
<li>Use haplotype information for better mapping.</li>
<li>Phasing from graph-wide estimation.</li>
<li>To represent/use a de novo assembly without the need linearize it.</li>
<li>To analyze different types of variants together.</li>
</ul>
</div>
</div>
<div id="types-of-graphs" class="section level2">
<h2>Types of graphs</h2>
<ul>
<li>A-Bruijn graphs. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC525693/">Used for MSA</a></li>
<li>Cactus graph: every edge is in at most one cycle.</li>
</ul>
</div>
<div id="tools" class="section level2">
<h2>Tools</h2>
<ul>
<li><a href="https://github.com/vgteam/vg">vg toolbox</a>.</li>
<li><a href="https://github.com/vgteam">Other tools from the vg team</a>. E.g. sequenceTubeMap.</li>
<li><a href="https://ccb.jhu.edu/software/hisat2/index.shtml">HISAT2</a>, graph-based alignment of next generation sequencing reads to a population of genomes.</li>
</ul>
</div>
<div id="talks" class="section level2">
<h2>Talks</h2>
<ul>
<li><a href="https://www.slideshare.net/GenomeRef/variation-reference-graphs-and-the-variation-graph-toolkit-vg">Variation reference graphs and the variation graph toolkit vg</a></li>
<li><a href="https://docs.google.com/presentation/d/1bbl2zY4qWQ0yYBHhoVuXb79HdgajRotIUa_VEn3kTpI/edit#slide=id.p">Slides: Resequencing against a human whole genome variation graph (vg)</a></li>
<li><a href="https://www.youtube.com/watch?v=BdaFFOD50og">Video: The Human Genome Variation Map Graph Bake-Off</a></li>
</ul>
</div>
<div id="tweets" class="section level2">
<h2>Tweets</h2>
<ul>
<li><a href="https://twitter.com/PlantEvolution/status/932983550818816000">A workflow for 1001 genomes project</a></li>
</ul>
</div>
<div id="papers" class="section level2">
<h2>Papers</h2>
<div id="cactus-algorithms-for-genome-multiple-sequence-alignment" class="section level3">
<h3>Cactus: Algorithms for genome multiple sequence alignment</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166836/">Paten, Genome Research 2011</a></p>
<p>The new <strong>Cactus alignment program</strong> is introduced.
It allows for <strong>alignment of multiple genomes using a Cactus graph representation</strong>.
Cactus graphs are particularly good at representing/embedding large rearrangements and copy number variation.</p>
<p><strong>Needs more reading.</strong></p>
</div>
<div id="improved-genome-inference-in-the-mhc-using-a-population-reference-graph" class="section level3">
<h3>Improved genome inference in the MHC using a population reference graph</h3>
<p><a href="https://www.pubmedcentral.nih.gov/articles/PMC4449272">Dilthey, Nature Genetics 2015</a></p>
<p>They introduced the concept of a <strong>population reference graph</strong> integrating known variants and multiple “alternate” reference sequences.</p>
<p>They used this approach on the MHC region of chromosome 6.
This is the famously polymorphic HLA region.
They build the graph with 8 alternate reference haplotypes, known HLA alleles, and ~90K SNPs.</p>
<p>In this proof of concept, they show that the genome inference is improved.</p>
<p>Their workflow to build the graph:</p>
<ol style="list-style-type: decimal">
<li>Multiple sequence alignment of reference sequences.</li>
<li>Collapse MSA into graph if sequence identity.</li>
<li>Add SNPs where possible.</li>
</ol>
<p>Then to infer the allele of each sample:</p>
<ol style="list-style-type: decimal">
<li>HMM from kmer counts to infer the best path in graph.</li>
<li>Build a chromotype from the best paths.</li>
<li>Use variant caller on chromotype to find new variants.</li>
<li>Add new variants to chromotype.</li>
</ol>
<p>In their comparison they found a gain in regions with paralogous.
The inclusion of variants called from the chromotype didn’t improve the performance much, most likely because this regions has been extensively studied and genotyped.</p>
<p>The graph approach makes use of a greater proportion of the reads, hence analyzing more of the region.</p>
<p>They also compared the chromotypes derived from different approach do Moleculo long reads.
They showed that the graph-based chromotypes are more similar, although Platypus looks very good also.
It had more perfectly aligned reads, but also more of the very badly aligned.</p>
<p>Although only 1% of the genome could gain from such approach, they mention other regions of high diversity:</p>
<ul>
<li>KIR region</li>
<li>Olfactory gene clusters</li>
<li>Inversion on 17q21.31</li>
</ul>
<p>Limitations:</p>
<ul>
<li>Kmer approach not suited for low-complexity regions.</li>
<li>HMM doesn’t integrate long range haplotype information.</li>
</ul>
</div>
<div id="canonical-stable-general-mapping-using-context-schemes" class="section level3">
<h3>Canonical, stable, general mapping using context schemes</h3>
<p><a href="https://www.pubmedcentral.nih.gov/articles/PMC4757953/">Novak, Bioinformatics 2015</a></p>
</div>
<div id="building-a-pan-genome-reference-for-a-population" class="section level3">
<h3>Building a pan-genome reference for a population</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424974/">Nguyen, J Comput Biol 2015</a></p>
<p>An extension of the Cactus program to a set of closely related genomes.
The goal is to align genomes together and find the best ordering of the homology blocks that best represent the underlying genome.
The results is a representation of the pan-genome of the population, particularly useful for visualization, variant description and potentially read mapping.</p>
</div>
<div id="genome-graphs" class="section level3">
<h3>Genome Graphs</h3>
<p><a href="https://doi.org/10.1101/101378">Novak, bioRxiv 2017</a></p>
<p>Several genome graphs structures are compared and shown to be superior to linear representations when mapping reads.</p>
<p>They tested 5 regions of the human genome: the major histocompatibility complex (MHC), the killer cell immunoglobulin-like receptors (LRC_KIR) region, the spinal muscular atrophy (SMA) locus, and the BRCA1 and BRCA2 genes.</p>
<p>Genome graphs improves the amount of read mapped but especially the amount of perfect mapping.</p>
</div>
<div id="coordinates-and-intervals-in-graph-based-reference-genomes" class="section level3">
<h3>Coordinates and intervals in graph-based reference genomes</h3>
<p><a href="https://www.pubmedcentral.nih.gov/articles/PMC5437615">Rand, BMC bioinformatics 2017</a></p>
<p>Different strategies to define coordinates and intervals in a graph are presented.
The goal is to find a good way to annotate genome graphs.</p>
<p>For coordinates we would want:</p>
<ul>
<li><em>spatiality</em>: nearby bases have similar coordinates.</li>
<li><em>readability</em>: concise and interpretable.</li>
<li><em>monotonicity</em>: increment along the genome.</li>
<li><em>backward compatibility</em>: supports updates somewhere else.</li>
</ul>
<p>They propose two offset-based coordinate systems.
One way is to hierarchically partition the graph by choosing main regions and alternate loci as baseline for the offsets.
This is backward compatible but there is no spatiality.
The other way, sequential partitioning, is to divide the main path every time there is a fork.</p>
<p>For naming regions we would want:</p>
<ul>
<li><em>vertical spatiality</em>: are two regions variants of each other.</li>
<li><em>horizontal spatiality</em>: are two regions close.</li>
<li><em>monotonicity</em>: which region comes first.</li>
</ul>
<p>Finally they propose two ways to represent intervals in the graph:</p>
<ul>
<li><em>single-path</em> intervals contains the start, end and information about the path in the graph.</li>
<li><em>multipath</em> intervals uses either a <em>critical</em> subpaths (where the path must go through) or a <em>fuzzy</em> path (central path and threshold).</li>
</ul>
</div>
<div id="a-graph-extension-of-the-positional-burrows-wheeler-transform-and-its-applications" class="section level3">
<h3>A graph extension of the positional Burrows-Wheeler transform and its applications</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5505026/">Novak, Algorithms Mol Biol 2017</a></p>
<p>A generalization of the PBWT (positional BWT) to genome graph is introduced, called gPBWT.
It’s a compressible representation of haplotypes in a graph that can be efficiently queried.
They conclude that the haplotype information could be incorporated into graph-based read mappers.</p>
</div>
<div id="modelling-haplotypes-with-respect-to-reference-cohort-variation-graphs" class="section level3">
<h3>Modelling haplotypes with respect to reference cohort variation graphs</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/28881971">Rosen, Bioinformatics 2017</a></p>
<p>Continuing on the haplotype representation from Novak 2017, they model recombination and haplotypes variation in the population.</p>
</div>
<div id="genome-graphs-and-the-evolution-of-genome-inference" class="section level3">
<h3>Genome graphs and the evolution of genome inference</h3>
<p><a href="https://www.pubmedcentral.nih.gov/articles/PMC5411762">Paten, Genome Research 2017</a></p>
</div>
<div id="comparative-annotation-toolkit-cat---simultaneous-clade-and-personal-genome-annotation" class="section level3">
<h3>Comparative Annotation Toolkit (CAT) - simultaneous clade and personal genome annotation</h3>
<p><a href="https://www.biorxiv.org/content/early/2017/12/08/231118.full.pdf+html">Fiddes, bioRxiv 2017</a></p>
<p>A toolkit to compare assemblies and genomes.
It uses Cactus graphs and Augustus under the hood to respectively align and annotate the assemblies.</p>
</div>
<div id="sequence-variation-aware-genome-references-and-read-mapping-with-the-variation-graph-toolkit" class="section level3">
<h3>Sequence variation aware genome references and read mapping with the variation graph toolkit</h3>
<p><a href="https://www.biorxiv.org/content/early/2017/12/15/234856">Garrison, bioRxiv 2017</a></p>
<p>Paper presenting the <strong>vg</strong> toolkit that provides ways to <strong>create/edit variation graphs</strong> and perform <strong>read mapping</strong> and <strong>variant calling</strong> on it.
<strong>vg scales to</strong> large genome like <strong>the human genome</strong>.
Contrary to existing approaches, vg doesn’t use or need an initial linear reference to build and use the graph.
A vg graph can be created from a linear reference + VCF, or from a set of sequences to align to each other.
New variant can be added easily.</p>
<p>A variation graph is a set of nodes, edges and paths.
Nodes can be traversed in two direction, the second representing the reverse complement of the first.</p>
<p>For alignment, vg uses a classic <strong>seed-extend</strong> through indexes.
The main difference is that it uses <strong>two representations</strong> for the two steps.
The super-maximal exact match (<strong>SMEM</strong>) seed identification is performed from the <strong>indexed</strong> xg graph using the GCSA2 library.
For paired reads, the seeds for both reads are searched at the same time and SMEMs that match the distance are preferred.
The best SMEMs are chosen using the ML though a Markov model.
Secondary seeds are retrieved by iteratively masking the best paths.
For each seed the subgraph is retrieved and converted into a directed acyclic graph (<strong>DAG</strong>) to perform the full read alignment.</p>
<p>vg was <strong>compared to bwa mem</strong> on simulated and real read from a Ashkenazi Jewish male genome.
In regions with <strong>no variants</strong>, vg performed slightly <strong>worse</strong> <strong>but</strong> gave a <strong>better</strong> performance in <strong>regions with variants</strong>.
This superiority was stronger for single ends as the paired end information helped figure out the correct mapping in many cases.
On real reads, vg mapped a similar number of reads but with better quality, especially more perfect matches.</p>
<p>More impressively, the <strong>coverage at known indel location</strong> (from GIAB annotation) was <strong>much more balanced</strong> with vg and independent of the variant size.</p>
<p>The other application uses graphs built from genome alignments of different yeast strains.
Again they show that reads map better in the vg graph than the reference from another strain.
Even removing the relevant strain from the graph showed good mapping performance.
I’m just a bit surprised that aligning to the relevant strain only is much better than the graph with several strains (including the relevant strain).</p>
<p>Among the future developments:</p>
<ul>
<li>Improving <strong>mapping</strong> and <strong>variant calling</strong> algorithms, e.g. using long range <strong>haplotype</strong> information.</li>
<li><strong>Allele-specific</strong> expression, benefiting from unbiased mapping.</li>
<li><strong>Splice-aware</strong> RNAseq mapping.</li>
<li>Integration of <strong>structural variants</strong>.</li>
</ul>
<p>Other comments:</p>
<ul>
<li>vg uses a bidirected DNA sequence graph structure.</li>
<li>Variant callers that use information of known variant: FreeBayes and GATK HaplotypeCaller (1000GP).</li>
<li>vg is multithreaded C++11. It uses protobuf to represent graphs.</li>
<li>They introduce a protobuf alignment format: <strong>GAM</strong> for Graph Alignment Map.</li>
<li><code>xg</code> is a <strong>succinct representation of a vg graph</strong>, more memory and time efficient, used for read mapping and other operations.</li>
<li>How is the <strong>distance</strong> measure defined ? It’s used during read mapping. They mention the use of nearby paths.</li>
<li>Some reads were mapped at different locations using linear vs graph approaches. Many of them because of repeats such as satellite DNA.</li>
<li>GraphTyper is mentioned in the discussion. It’s quite different as it only realigns locally reads into a graph.</li>
</ul>
</div>
</div>
<div id="snv-caller-that-uses-known-variantshaplotypes" class="section level2">
<h2>SNV caller that uses known variants/haplotypes</h2>
<div id="freebayes" class="section level3">
<h3>FreeBayes</h3>
<ul>
<li>By Erik Garrison.</li>
<li><a href="https://github.com/ekg/freebayes">GitHub</a></li>
<li><a href="https://arxiv.org/abs/1207.3907">arXiv</a></li>
</ul>
<blockquote>
<p>Bayesian statistical framework which is capable of modeling multiallelic loci in sets of individuals with non-uniform copy number.</p>
</blockquote>
<blockquote>
<p>FreeBayes is haplotype-based, in the sense that it calls variants based on the literal sequences of reads aligned to a particular target, not their precise alignment.</p>
</blockquote>
<ul>
<li>Should be used in a population of samples.</li>
<li>Finds the most-likely combination of genotype in the population.</li>
<li>Haplotype is short-range haplotype (max 10s of bp).</li>
</ul>
</div>
<div id="gatk-haplotype-caller" class="section level3">
<h3>GATK Haplotype Caller</h3>
<p><em>From <a href="https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145">GATK documentation</a>.</em></p>
<blockquote>
<p>The HaplotypeCaller is capable of calling SNPs and indels simultaneously via local de-novo assembly of haplotypes in an active region. In other words, whenever the program encounters a region showing signs of variation, it discards the existing mapping information and completely reassembles the reads in that region. This allows the HaplotypeCaller to be more accurate when calling regions that are traditionally difficult to call, for example when they contain different types of variants close to each other. It also makes the HaplotypeCaller much better at calling indels than position-based callers like UnifiedGenotyper.</p>
</blockquote>
<p><a href="https://gatkforums.broadinstitute.org/gatk/discussion/4148/hc-overview-how-the-haplotypecaller-works">How it works</a>:</p>
<ol style="list-style-type: decimal">
<li>Define active regions, i.e. with sign of variation. Using a per-bp activity score.</li>
<li>Re-assemble reads to list possible haplotypes.</li>
<li>Align reads to each haplotype to derive per-read haplotype likelihoods. Using a HMM alignment.</li>
<li>Assign sample genotypes.</li>
</ol>
</div>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../../../../assets/highlight.min.js"></script>
    <script src="../../../../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

