<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Local assembly &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../../../../css/poole.css">
  <link rel="stylesheet" href="../../../../css/syntax.css">
  <link rel="stylesheet" href="../../../../css/lanyon.css">
  <link rel="stylesheet" href="../../../../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../../../../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../../assets/R-icon.png">
  <link rel="shortcut icon" href="../../../../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../../../../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../../../../">Home</a>
    <a class="sidebar-nav-item " href="../../../../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../../../../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../../../../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../../../../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="post">
  <h1 class="post-title">Local assembly</h1>
  <span class="post-date">Aug 17 2016
  
  
  
  
  
  <a href='../../../../tags/literature/' class='label'>literature</a>
  
  
  
  </span>
  


<div id="tigra-a-targeted-iterative-graph-routing-assembler-for-breakpoint-assembly" class="section level2">
<h2>TIGRA: A targeted iterative graph routing assembler for breakpoint assembly</h2>
<p>by Ken Chen et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24307552">Genome Research 2014</a>.</p>
<p><em>Targeted Iterative Graph Routing Assembler</em> was widely used in large-scale projects (e.g. 1KGP) for <strong>breakpoints assembly of deletions and mobile element insertion</strong>.
It’s an old effort, that they started in 2008, which was optimized and improved across the years.</p>
<p>To clarify, TIGRA assembles the breakpoint region of SV, <strong>not the entire SV region</strong>.
Another clarification: TIGRA is used to <strong>validate existing calls</strong> and reduce the FDR, not so much to characterize SV mechanism or features.</p>
<ol style="list-style-type: decimal">
<li><strong>Reads are extracted</strong> around the predicted breakpoint (+- 500bp-1Kbp).
<ul>
<li>Mapped reads <em>and their mate</em> are retrieved from the BAM files.</li>
<li>If multiple samples are available and their genotype is known, reads from the variant-containing samples can be pooled.</li>
</ul></li>
<li>Reads are <strong>assembled</strong> using a de Bruijn graph.
<ul>
<li>Multiple k-mer sizes are <em>iteratively</em> used (starting from small k-mers).</li>
<li>Previously assembled contigs taken as <em>pseudo-reads</em> in the next iterations.</li>
<li>Tips, likely created by sequencing errors, are pruned.</li>
<li>Bubbles are joined when the size difference is lower than 3 nucleotides.</li>
<li><em>Entire reads</em> are used to distinguish repeats, instead of k-mers.</li>
</ul></li>
<li>Assembled sequences are <strong>mapped back to the reference genome</strong> using <a href="http://www.phrap.org/phredphrapconsed.html">cross_match</a>.
<ul>
<li>Every contig is tested although they don’t explain explicitly how these contigs are retrieved from the graph.</li>
<li>Contigs with more than two hits or substitution rate higher than 0.5% (low isn’t it ?) are not reported.</li>
<li>A <em>glocal</em> alignment is used.</li>
</ul></li>
</ol>
<p>They tested TIGRA assembly step against Velvet, SGA, Phrap and SPAdes.
They used datasets with known deletions breakpoints and mobile element insertions.
TIGRA shows the <em>highest proportion of validated SVs</em> in any situations.
The validation rates are lower than I thought (40%-80%) but it might reflect the low sequencing depth.
They also managed to detect quasi-full length Alu insertion (Was it because they knew it was there ?).
TIGRA is more robust to low-coverage alleles than other existing methods, thanks to its multiple k-mer size process.</p>
<p>It seems mainly <strong>focused on deletions, duplication (not benchmarked) and mobile insertion</strong>, although they mention being able to detect any type of de novo SV and having specific strategies to deal with inversion or reciprocal translocations.</p>
<p>While it handles repeats slightly better than other methods, by using full reads instead of k-mers, repeated sequences longer than the read length will not be resolved and are bound to create ambiguous graphs.
Indeed they observed a reduction of assembled breakpoints with higher breakpoint homology.
It is implemented to assemble small breakpoint regions, hence <strong>require almost nucleotide resolution as input</strong>.</p>
<ul>
<li>They mention an important drop in validation rate for RD-based regions. Is it because of the input breakpoints being less precise ?</li>
<li>I didn’t understand how they define their contigs (used in the iterative process; aligned against the reference). Is it all possible paths in the graph ? Only unambiguous paths ?</li>
</ul>
<p>TIGRA is available here: <a href="http://bioinformatics.mdanderson.org/main/TIGRA">http://bioinformatics.mdanderson.org/main/TIGRA</a>.</p>
</div>
<div id="breakpoint-profiling-of-64-cancer-genomes-reveals-numerous-complex-rearrangements-spawned-by-homology-independent-mechanisms." class="section level2">
<h2>Breakpoint profiling of 64 cancer genomes reveals numerous complex rearrangements spawned by homology-independent mechanisms.</h2>
<p>by Malhotra et al, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3638133&amp;tool=pmcentrez&amp;rendertype=abstract">Genome Research 2013</a>.</p>
<p>In this study, they looked at 64 genomes from 7 different cancers and tried to characterize the breakpoints of somatic SVs.
Here, I’ll focus on the local assembly part of the paper.</p>
<p>They <strong>modified SGA assembler</strong> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3290790/">Simpson and Durbin 2012</a>) to <strong>report all paths</strong> from all connected components from the string graph.</p>
<ol style="list-style-type: decimal">
<li>Extracted read pairs around the breakpoint (+- 500 bp).</li>
<li>Assembly: <em>sga preprocess</em>, <em>sga index</em>, <em>sga correct</em>, <em>sga index</em>, <em>sga rmdup</em>, <em>sga overlap</em>, <em>sga assemble</em>, modified <em>sga walk</em>.</li>
<li>Contig aligned on reference using BWA-SW.</li>
<li>Split-mapping with &gt;25bp non-overlap with adjacent mapping.</li>
<li>Comparison with original SV signature</li>
</ol>
<p>Consistent with TIGRA benchmark, they estimate their validation rate around 80% for deletion.
They mention that deletion are the easiest to validate and expect lower overall validation rate.</p>
<ul>
<li>How does this perform in regions with repeats ?</li>
<li>How does it scale up when assembling larger regions ?</li>
</ul>
</div>
<div id="discovery-and-characterization-of-alu-repeat-sequences-via-precise-local-read-assembly" class="section level2">
<h2>Discovery and characterization of Alu repeat sequences via precise local read assembly</h2>
<p>by Wildschutte et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/26503250"><em>Nucleic Acids Res.</em> 2015</a>.</p>
<p>Jeff Kidd’s group built on existing approaches to <strong>detect Alu insertion</strong> and attempted to <strong>assemble the full allelic sequence</strong>.</p>
<ol style="list-style-type: decimal">
<li>Retro-Seq selected <strong>insertion-supporting reads</strong>.
<ul>
<li>One pair mapped uniquely, the other in Alu consensus or instance.</li>
<li>Minimum of 2 supporting reads per call.</li>
<li>Call <em>within 500 bp of existing Alu were removed</em>.</li>
</ul></li>
<li>Reads for each insertion were <strong>assembled with CAP3</strong> (overlap-layout-consensus).
<ul>
<li>Read pair within 200 bp and with soft-clip &gt; 20 bp were added.</li>
<li>CAP3 uses read-pair information and links contigs.</li>
<li>They keep sequence with at least 30 bp of Alu and 30 bp of non-gapped sequence..</li>
</ul></li>
<li><strong>Breakpoint determination</strong> using a suite of (old) programs.
<ul>
<li>Orientation guessed by a <em>BLAT</em> on the reference.</li>
<li>Candidate breakpoints from <em>miropeats</em> (semi-automated graphical alignment).</li>
<li>Global alignment of the insertion breakpoints using <em>stretcher</em> (to see TSD).</li>
<li>A lot of <em>manual review/curation</em> in the end.</li>
</ul></li>
</ol>
<p>Pipeline available on GitHub: <a href="https://github.com/KiddLab/insertion-assembly">https://github.com/KiddLab/insertion-assembly</a>.</p>
</div>
<div id="enhanced-structural-variant-and-breakpoint-detection-using-svmerge-by-integration-of-multiple-detection-methods-and-local-assembly." class="section level2">
<h2>Enhanced structural variant and breakpoint detection using SVMerge by integration of multiple detection methods and local assembly.</h2>
<p>by Wong et al, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3046488%7B&amp;%7Dtool=pmcentrez%7B&amp;%7Drendertype=abstract">Genome Biology 2010</a>.</p>
<p>SVMerge is a pipeline that:</p>
<ol style="list-style-type: decimal">
<li>Makes SV prediction with a collection of callers.</li>
<li>Filter them</li>
<li>Merge them.</li>
<li>Validate them using local assembly.</li>
</ol>
<p>For the assembly step:</p>
<ol style="list-style-type: decimal">
<li>Reads around the breakpoint are extracted (+- 1-2 kbp).
<ul>
<li>Mapped read and mate pair.</li>
<li>Reads were written in FASTA with interleaved read pairs (Velvet input?).</li>
<li>SV discarded if too many reads.</li>
</ul></li>
<li>Assembly using ABySS or Velvet.
<ul>
<li>With and without scaffolding.</li>
<li>All contigs were considered.</li>
</ul></li>
<li>Contig alignment using Exonerate.</li>
<li>Alignment parsing.
<ul>
<li>To verify the breakpoint location.</li>
<li>To verify read depth change (het deletions).</li>
</ul></li>
</ol>
<p>Perl scripts available here: <a href="http://svmerge.sourceforge.net/">http://svmerge.sourceforge.net/</a>.</p>
</div>
<div id="other-sv-assembler-to-check" class="section level2">
<h2>Other SV-assembler to check</h2>
<ul>
<li>laSV: <a href="http://nar.oxfordjournals.org/lookup/doi/10.1093/nar/gkv831">Local sequence assembly reveals a high-resolution profile of somatic structural variations in 97 cancer genomes</a>.</li>
<li>AsmVar: <a href="http://www.gigasciencejournal.com/content/4/1/64">Discovery, genotyping and characterization of structural variation and novel sequence at single nucleotide resolution from de novo genome assemblies on a population scale</a>.</li>
<li>CREST: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3527068&amp;tool=pmcentrez&amp;rendertype=abstract">CREST maps somatic structural variation in cancer genomes with base-pair resolution. Wang et al. Nat Methods. 2011</a>.</li>
<li>SV-STAT: <a href="http://scfbm.biomedcentral.com/articles/10.1186/s13029-016-0051-0">SV-STAT accurately detects structural variation via alignment to reference-based assemblies</a></li>
<li>PopIns: <a href="http://bioinformatics.oxfordjournals.org/cgi/doi/10.1093/bioinformatics/btv273">PopIns: population-scale detection of novel sequence insertions</a>.</li>
</ul>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../../../../assets/highlight.min.js"></script>
    <script src="../../../../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

