<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Lit Digest - SV and long-read sequencing &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../../../../css/poole.css">
  <link rel="stylesheet" href="../../../../css/syntax.css">
  <link rel="stylesheet" href="../../../../css/lanyon.css">
  <link rel="stylesheet" href="../../../../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../../../../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../../assets/R-icon.png">
  <link rel="shortcut icon" href="../../../../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../../../../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../../../../">Home</a>
    <a class="sidebar-nav-item " href="../../../../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../../../../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../../../../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../../../../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="post">
  <h1 class="post-title">Lit Digest - SV and long-read sequencing</h1>
  <span class="post-date">Sep 20 2017
  
  
  
  
  
  <a href='../../../../tags/literature/' class='label'>literature</a>
  
  
  
  </span>
  


<ul>
<li>How many more SVs ?</li>
<li>Which types of SVs ?</li>
</ul>
<div id="identifying-structural-variants-using-linked-read-sequencing-data" class="section level2">
<h2>Identifying structural variants using linked-read sequencing data</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/09/18/190454">Elyanow, biorXiv 2017</a></p>
<ul>
<li>Better than other methods…</li>
</ul>
</div>
<div id="mapping-and-phasing-of-structural-variation-in-patient-genomes-using-nanopore-sequencing" class="section level2">
<h2>Mapping And Phasing Of Structural Variation In Patient Genomes Using Nanopore Sequencing</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/04/24/129379">Stancu, bioRxiv 2017</a></p>
<ul>
<li>2 patients with congenital abnormalities, with MinION (11x/16x).</li>
<li>Better at finding complex chromothripsis.</li>
<li>3K novel SVs, majority being duplication &lt;200 bp.</li>
<li>Phasing showed the de novo chromothripsis occurred in paternal chromosomes.</li>
<li>Helps a lot for complex SVs.</li>
</ul>
</div>
<div id="accurate-detection-of-complex-structural-variations-using-single-molecule-sequencing" class="section level2">
<h2>Accurate detection of complex structural variations using single molecule sequencing</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/07/28/169557">Sedlazeck, bioRxiv 2017</a></p>
<ul>
<li><ol start="2" style="list-style-type: decimal">
<li>Method paper.</li>
</ol></li>
<li>Find much more novel than TGP ?</li>
<li>Show that many translocation calls from short read caller are wrong (85%).</li>
</ul>
</div>
<div id="hysa-a-hybrid-structural-variant-assembly-approach-using-next-generation-and-single-molecule-sequencing-technologies" class="section level2">
<h2>HySA: a Hybrid Structural variant Assembly approach using next-generation and single-molecule sequencing technologies</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411774">Fan, Genome Research 2016</a></p>
<ul>
<li>Method paper.</li>
<li>Better job than Chaisson’s SMRT-SV ?</li>
</ul>
</div>
<div id="discovery-and-genotyping-of-structural-variation-from-long-read-haploid-genome-sequence-data" class="section level2">
<h2>Discovery and genotyping of structural variation from long-read haploid genome sequence data</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411763">Huddleston, Genome Research 2016</a></p>
<ul>
<li>2 haploid genomes.</li>
<li>Long-read is much more sensitive.</li>
<li>Highest gain for “small” SVs: 7bp - 1 Kbp.</li>
<li>90% SVs are novel.</li>
<li>60% can be genotyped with short reads once alternate allele is included.</li>
<li>Genotyped in 30 TGP samples. Some SVs look fixed.</li>
<li>Haploid is very useful for detection.</li>
</ul>
</div>
<div id="resolving-the-complexity-of-the-human-genome-using-single-molecule-sequencing" class="section level2">
<h2>Resolving the complexity of the human genome using single-molecule sequencing</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4317254">Chaisson, Nature 2015</a></p>
<ul>
<li>Resolve inversions, complex insertions and long tandem repeats.</li>
<li>Enrichment in long short STR regions and complex insertions (?).</li>
<li>Strong enrichment of STRs (15x) among closed gaps. 78% of the gaps were covered by STR more than 10%.</li>
<li>85% of the CNVs were novel, especially for insertions (92%).</li>
<li>2.8x enrichment of STR insertions compared to deletions. STR insertions were 2.9x longer.</li>
<li>STR insertions clustered at the last 5 Mbp of the chromosomes.</li>
<li>Thousands of STR insertions within genes, 2 in exons.</li>
<li>Other complex insertions enriched in regions already enriched in MEI.</li>
<li>L1 and SVA insertions are found adjacent to repeats, more than in 1KGP or short read catalogs.</li>
<li>68% of inversions flanked by inverted repeats.</li>
</ul>
</div>
<div id="assembly-and-diploid-architecture-of-an-individual-human-genome-via-single-molecule-technologies" class="section level2">
<h2>Assembly and diploid architecture of an individual human genome via single-molecule technologies</h2>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4646949">Pendleton, Nature Methods 2015</a></p>
<ul>
<li>Identified complex SVs</li>
<li>Phased SNVs and SVs</li>
<li>Identified large tandem repeats and suggest that TR are larger than in the reference genome.</li>
<li>Observed DNA transduction when MEI.</li>
<li>Complex STR-like insertion within STRs.</li>
<li>4.2% of SVs are complex.</li>
<li>Inversions are enriched for complexity (55%).</li>
</ul>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../../../../assets/highlight.min.js"></script>
    <script src="../../../../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

