<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Diseases &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../css/poole.css">
  <link rel="stylesheet" href="../css/syntax.css">
  <link rel="stylesheet" href="../css/lanyon.css">
  <link rel="stylesheet" href="../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../assets/R-icon.png">
  <link rel="shortcut icon" href="../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../">Home</a>
    <a class="sidebar-nav-item " href="../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item  active " href="../diseases/">Diseases</a>
      
    
      
        <a class="sidebar-nav-item " href="../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2023 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="page">
  <h1 class="page-title">Diseases</h1>
  

<div id="TOC">
<ul>
<li><a href="#hemochromatosis" id="toc-hemochromatosis">Hemochromatosis</a>
<ul>
<li><a href="#genetics" id="toc-genetics">Genetics</a></li>
<li><a href="#iron-regulation" id="toc-iron-regulation">Iron regulation</a></li>
<li><a href="#next" id="toc-next">Next</a></li>
</ul></li>
<li><a href="#inflammatory-bowel-disease" id="toc-inflammatory-bowel-disease">Inflammatory bowel disease</a>
<ul>
<li><a href="#genetics-1" id="toc-genetics-1">Genetics</a>
<ul>
<li><a href="#functional-fine-mapping" id="toc-functional-fine-mapping">Functional fine-mapping</a></li>
</ul></li>
</ul></li>
<li><a href="#cancer" id="toc-cancer">Cancer</a>
<ul>
<li><a href="#leiomyosarcoma-lms" id="toc-leiomyosarcoma-lms">Leiomyosarcoma (LMS)</a>
<ul>
<li><a href="#structural-variants-of-lms" id="toc-structural-variants-of-lms">Structural variants of LMS</a></li>
<li><a href="#to-read" id="toc-to-read">To read</a></li>
</ul></li>
<li><a href="#neoantigens" id="toc-neoantigens">Neoantigens</a>
<ul>
<li><a href="#to-read-1" id="toc-to-read-1">To read</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<div id="hemochromatosis" class="section level1">
<h1>Hemochromatosis</h1>
<p><a href="https://en.wikipedia.org/wiki/Iron_overload">Hemochromatosis</a> is caused by an excess of iron.
There are five types of hemochromatosis, two of which are juvenile forms.
The classical type, <a href="https://en.wikipedia.org/wiki/Hereditary_haemochromatosis">hereditary hemochromatosis type 1</a>, is by far the most frequent.</p>
<p>Multiple organs are affected, notably liver and heart.
Symptoms include:</p>
<ul>
<li>liver disease, e.g. cirrhosis or liver cancer</li>
<li>heart failure</li>
<li>chronic fatigue</li>
<li>arthritis</li>
</ul>
<p>The disease onset is later in women, usually after menopause.
Men aged 40-50 and more should show symptoms.</p>
<p>The treatment is usually blood draws (<a href="https://en.wikipedia.org/wiki/Phlebotomy">phlebotomy</a>).</p>
<div id="genetics" class="section level3">
<h3>Genetics</h3>
<p>The main risk factor is a <a href="https://www.ncbi.nlm.nih.gov/snp/rs1800562">C282Y mutation</a> in the HFE gene which is frequent but has an incomplete penetrance.
This mutation is observed in about 5% of European populations, and to a lower extent in south Europe and the US.
It was spread through Viking and then Normans migrations (and the <a href="https://bmcmedgenet.biomedcentral.com/articles/10.1186/1471-2350-5-25">celtic haplotype HLA-A3/B7</a>).
Carriers of the homozygous risk alleles (3-8 out of 1000) have about 10-20% chance of developing the disease.</p>
<p>A second HFE mutation is sometimes H63D, which has 17% allele frequency.</p>
<p>HFE is the gene for the hepcidin protein, which is involved in iron absorption regulation by regulating the concentration of ferroportin, the only iron exporter.
Hepcidin also controls the release of iron by macrophages or hepatocytes.</p>
<p>Heritability of hemochromatosis is unknown.
Among HFE-hom, transferrin levels and serum ferritin levels had an estimated heritability of ~0.2 (<a href="https://doi.org/10.1182/blood.V112.11.115.115">McLaren et al. 2008</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/20095037/">McLaren et al. 2010</a>).
While the chances of developing symptoms is ~20% in general, it increases to ~50% if another member of the family suffers from hemochromatosis.</p>
<p>De Tayrac et al. (<a href="https://pubmed.ncbi.nlm.nih.gov/25457201/">2015</a>) tested 500K SNPs across 474 unrelated HFE-homs and found a SNP in intron 11 of TF (transferrin gene) associated with levels of serum iron (serum transferrin and serum iron), which correlated with severity of the disease (iron removed by blood draws, clinical stage).
The association was replicated in 748 C282Y-homozygotes.</p>
<p>Milet et al. (<a href="https://pubmed.ncbi.nlm.nih.gov/17847004/">2007</a>) tested just 10 genes (75 SNPs) across 592 C282Y-homozygotes, and found an association between a SNP near BMP2 and pretherapeutic serum ferritin level (marker penetrance).
They also detected a potential interaction between the BMP2 SNP and a SNP in HJV.
Overall, they concluded that the BMP pathway might be a good candidate when searching for modifier genes.</p>
</div>
<div id="iron-regulation" class="section level3">
<h3>Iron regulation</h3>
<ul>
<li>Enterocyte: intestinal absorptive cell</li>
<li>Hepatocyte: liver cell</li>
<li>Macrophages can contain iron too after recycling erythrocytes.</li>
<li>Erythrocytes: red blood cells.</li>
<li>Erythropoiesis is the process of making red blood cells, which uses iron.</li>
<li>Duedenum: first section of the small intestine, where most iron absorption occurs.</li>
<li>Ferroportin exports iron absorbed by enterocytes, into blood stream. Regulated by hepcidin.</li>
<li>Hepcidin regulates iron homeostasis. Inhibits iron transport by binding to ferroportin.</li>
<li>Transferrin: protein that binds to iron to help its transport to tissues. Transferrin saturation is a symptom of hemochromatosis.</li>
<li>HFE: gene encoding a transmembrane protein that is thought to down-regulate the absorption of iron from transferrin.</li>
<li>Ferritin: intracellular protein that stores iron.</li>
</ul>
</div>
<div id="next" class="section level3">
<h3>Next</h3>
<ul>
<li><a href="https://www.nature.com/articles/gim2015140">The global prevalence of HFE and non-HFE hemochromatosis estimated from analysis of next-generation sequencing data</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4215164/">Novel loci affecting iron homeostasis and their effects in individuals at risk for hemochromatosis</a></li>
<li><a href="https://www.omim.org/entry/613609">HFE in OMIM</a></li>
</ul>
</div>
</div>
<div id="inflammatory-bowel-disease" class="section level1">
<h1>Inflammatory bowel disease</h1>
<p>Inflammatory bowel disease (IBD) is characterized by inflammation of the colon and small intestine.
The two main forms are Crohn’s disease (CD) and ulcerative colitis (UC).
The unregulated inflammation is thought to be caused by a combination of environmental, genetic and bacterial factors interacting with the immune system.
Most IDB types are classified as an autoimmune disease because thought to be caused by a disfunctional immune response.</p>
<p>In CD, the inflammation tend to be spread in local clusters (skip lesions) and to extend deep into the tissues.
The inflammation of UC usually only affects the outer (mucosal) layer and starts in the rectal/sigmoid area (proctitis) before spreading.</p>
<div id="genetics-1" class="section level2">
<h2>Genetics</h2>
<p>Many associated loci in IDB, are associated to both CD and UC.
For example, 110 out of 163 loci associated with IBD conferred risk to both CD and UC (review <span class="citation">McGovern, Kugathasan, and Cho (<a href="#ref-mcgovern2015g">2015</a>)</span>).
In many cases, the effect is stronger in CD than UC, for example the Arg381Gln protective allele in IL23R (review <span class="citation">McGovern, Kugathasan, and Cho (<a href="#ref-mcgovern2015g">2015</a>)</span>).
Associated loci are sometimes also associated with other phenotypes like primary sclerosing cholangitis, psoriasis, ankylosing spondylitis, type 1 diabetes, or multiple sclerosis (review <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span>).</p>
<p>Explained variance tends to be low.
The 163 loci, from the meta-analysis mentioned in the <span class="citation">McGovern, Kugathasan, and Cho (<a href="#ref-mcgovern2015g">2015</a>)</span> review, explained only 13.6% of CD and 7.5% of UC total disease variance.
According to <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span> (review), on 26% of CD variance is accounted for by disease-associated SNPs.
This could be due to epistasis (gene-gene interactions) or gene-environment interactions not being modeled, or because some regions are not well studied (repeats, sex chromosomes), e.g. the Fc receptor gene cluster or the KIR genes.
Rare variants might have a large effect, although the first studies on these variants failed to show it (review <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span>).
Of course, the heritability estimates might be inflated too.</p>
<p>A missense allele in NOD2 was one of the first associated loci found with CD.
NOD2 was originally one of the few examples of successes by linkage mapping approaches.
It supported the hypothesis that CD may be caused by innate immune cell function deficiencies, and pointed at innate lymphoid cells as key players in CD.
GWAS hits related to autophagy seem to be specific to CD.</p>
<p>The discovery of a missense allele in ATG16L1 lead the field to explore the role of autophagy in CD.
An exonic variant in IRGM, a gene related to autophagy also identified early, affects the binding site of a microRNA (miR-196) expressed in intestinal epithelia and likely repressing IRGM.</p>
<p>Multiple independent risk alleles for IBD were found in the IL23R gene region.
Associated loci provided evidence that anti-inflammatory mechanisms may be disrupted in CD.
Several GWAS hits are located near genes involved in IL-10 signaling, an anti-inflammatory cytokine, for example IL10, IL10RB, STAT3, TYK2.
GWAS hits were also found near other cytokines, e.g. IL22, SMAD3, SMAD7.
Several of these SNPs related IL-23/IL-17 have been associated with other autoimmune diseases such as psoriasis, ankylosing spondylitis, or multiple sclerosis (review <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span>).
However, the signals the SNPs might be independent or have opposing effects across those traits.
For example, a missense risk allele for CD in TYK2 look protective for psoriasis.</p>
<p>W134X in FUT2 is a CD-specific association that suggested a potential importance of the mucus layer.</p>
<p>Some associations are not replicated in all human populations.
For example, NOD2 and ATG16L1 were not found to be associated with CD in East Asians (review <span class="citation">McGovern, Kugathasan, and Cho (<a href="#ref-mcgovern2015g">2015</a>)</span>).
However, ATG16L2 seemed to be, according to GWAS studies in Korean and Japanese populations.</p>
<p>In 2010, a large meta-analysis included ~22K cases and 29K controls from 13 different countries (review <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span>).
It found 71 susceptibility loci.</p>
<ul>
<li>Summary table of known loci: see table 1 in review <span class="citation">Verstockt, Smith, and Lee (<a href="#ref-verstockt2018c">2018</a>)</span></li>
<li>Consortium of note: International Inflammatory Bowel Disease Genetic Consortium (IIBDGC).</li>
</ul>
<div id="functional-fine-mapping" class="section level3">
<h3>Functional fine-mapping</h3>
<p>Like many other complex diseases, it is difficult to identify the causal gene driving the association signal.
This is due, among other factors, to large LD blocks, missing variants, cell type specific effects,</p>
<p>Targeted sequencing around GWAS hits can also help identify likely causal genes.
For example, it uncovered rare missense alleles in PRDM1 or CARD9.</p>
<p>Expression QTLs can also help narrow down the list of gene candidate in a region.</p>
<p>Single-cell RNA seq studies may help understand the different cell types involved in IDB ( see <a href="http://scibd.cn/">scIBD</a>).</p>
<p>Of note, the set of associated loci can still provide useful information, even if causal variants remain uncharacterized.
For example, the loci can be compared to epigenetic profiles across cell types to identify the ones that might be involved.
For IBD, the strongest enrichment was observed within CD4+ T cells.
CD showed an enrichment in immune cells, while UC was also enriched in colonic mucus.</p>
</div>
</div>
</div>
<div id="cancer" class="section level1">
<h1>Cancer</h1>
<div id="leiomyosarcoma-lms" class="section level2">
<h2>Leiomyosarcoma (LMS)</h2>
<p>Leiomyosarcoma (LMS) is the most common soft tissue and uterine sarcoma.
It’s the malignant tumor of smooth muscles, which are found in hollw structures like intestines, bladder, uterus, blood vessels.
Unpredictable and usually not responsive to chemotherapy, the most effective treatment seems to be surgical removal.</p>
<p>Subtypes include:</p>
<ul>
<li>Uterine leiomyosarcoma</li>
<li>Cutaneous leiomyosarcoma</li>
<li>Gastrointestinal leiomyosarcoma</li>
</ul>
<p>SRF: Serum Response Factor (SRF), transcription factor involved in cell proliferation, cell death resistance, metastasis</p>
<p>Alternative Lengthening of Telomeres (ALT) has been associated with poorer outcome.
Instead of using telomerase to extend/maintain telomeres, the ALT pathway uses an homologous recombination-based pathway and allow cancerous cells to evade senescence due to telomere shortening.</p>
<p>Low mutation burden of ~40 somatic non-silent (missense or nonsense) coding SNVs per sample (RevChibon2019).
Cancer with one of the highest number of rearrangements, but they are mostly private and don’t typically create gene fusions.</p>
<p>We expect multiple activation/inactivation of oncogenes/tumor suppressors with combined effects rather than one event.</p>
<p>Recurrent events:</p>
<ul>
<li>TP53 pathway altered</li>
<li>p16/RB1 pathway altered</li>
<li>PI3K/mTOR pathway dysregulation</li>
<li>ATRX frequently mutated by SNVs, and associated with ALT phenotype</li>
</ul>
<div id="structural-variants-of-lms" class="section level3">
<h3>Structural variants of LMS</h3>
<p>More somatic losses than gains (RevChibon2019).
LMS are often tetraploid.</p>
<p>Exclusive genomic alterations:</p>
<ul>
<li>MYOCD amplification</li>
<li>PTEN homozygous deletion</li>
<li>DMD structural rearrangement</li>
</ul>
<p>Gene fusions:</p>
<ul>
<li>TRIO-TERT?</li>
</ul>
</div>
<div id="to-read" class="section level3">
<h3>To read</h3>
<ul>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/29100075/">Comprehensive and Integrated Genomic Characterization of Adult Soft Tissue Sarcomas. Cell 2017</a></li>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/29321523/">Integrative genomic and transcriptomic analysis of leiomyosarcoma. Nat Commun 2018</a></li>
</ul>
</div>
</div>
<div id="neoantigens" class="section level2">
<h2>Neoantigens</h2>
<p>Neoantigens are antigens produced by tumors and that could be targeted for treatment.
They can arise from tumor viruses integrating in the cancerous genome, or as products of mutant proteins.</p>
<p>Most specific neoantigens might be non-coding and will require better tools to be identified.</p>
<div id="to-read-1" class="section level3">
<h3>To read</h3>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8085349/">Neoantigen: A New Breakthrough in Tumor Immunotherapy. Front Immunol 2021</a></li>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/30518613/">Noncoding regions are the main source of targetable tumor-specific antigens. Sci Transl Med 2018</a></li>
</ul>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-mcgovern2015g" class="csl-entry">
McGovern, Dermot P. B., Subra Kugathasan, and Judy H. Cho. 2015. <span>“Genetics of <span>Inflammatory Bowel Diseases</span>.”</span> <em>Gastroenterology</em>. <a href="https://doi.org/10.1053/j.gastro.2015.08.001">https://doi.org/10.1053/j.gastro.2015.08.001</a>.
</div>
<div id="ref-verstockt2018c" class="csl-entry">
Verstockt, Bram, Kenneth Gc Smith, and James C Lee. 2018. <span>“Genome-Wide Association Studies in <span>Crohn</span>’s Disease: <span>Past</span>, Present and Future.”</span> <em>Clinical &amp; Translational Immunology</em>. <a href="https://doi.org/10.1002/cti2.1001">https://doi.org/10.1002/cti2.1001</a>.
</div>
</div>
</div>
</div>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../assets/highlight.min.js"></script>
    <script src="../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

