---
date: 2016-06-18
title: SV, gene expression, cancer types, Alu insertions.
tags: ['literature']
output: blogdown::html_page
---



<div id="the-impact-of-structural-variation-on-human-gene-expression" class="section level2">
<h2>The impact of structural variation on human gene expression</h2>
<p>by Chiang et al, <a href="http://biorxiv.org/content/early/2016/06/09/055962?rss=1%2522">biorXiv 2016</a>.</p>
<p>The group from Ira Hall and the GTEx consortium investigated how much SVs affect gene expression and put it in context with existing SNV eQTLs work.
The jointly analyze the effect of SNVs, indels and SVs across 13 tissues in 147 individuals.
Their main results are:</p>
<ul>
<li>SVs look like the causal variants of <strong>3.3-7% of eQTLs</strong>. Much more than predicted before and a large enrichment considering their genomic scarcity.</li>
<li><strong>88%</strong> of the putative causal SVs are <strong>non-coding</strong>, but enriched in enhancer and regulatory regions.</li>
<li>49 SVs are predicted to be causal of known GWAS hits.</li>
<li><strong>30% of common SV-eQTLs</strong> are <strong>not well tagged</strong> by flanking SNPs</li>
</ul>
</div>
<div id="classification-of-cancers-based-on-copy-number-variation-landscapes" class="section level2">
<h2>Classification of cancers based on copy number variation landscapes</h2>
<p>by Zhang et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/27266344?dopt=Abstract"><em>BBA</em> 2016</a>.</p>
<p>They built a classifier taking as input CNVs of 23K genes and <strong>classifying six cancer types</strong>.
They reached <strong>0.75 accuracy with only 19 genes</strong>.
For classification, they used the Dagging algorithm, which chunks the data and feed them to a base classifier.
Then the consensus over the chunks is used as final classification.
The accuracy represents the proportion correctly classified samples.
In figure 1 we see how the accuracy increases rapidly with the number of features to then plateau around 75%.
Some cancer types are more accurately classified.</p>
<p>The 19 genes are: RPS15, IL17RC, CUL2, SMPD3, MIR4703, CDKN2A, RFFL, CTBP2, MMD2, SEMA6A, ZFPM1, CDC25A, ZMYND11, KBTBD6, CELF5, EGFR, PIGL, ZNF503-AS1, RBFOX1.</p>
<p>The introduction has some interesting sentences:</p>
<blockquote>
<p>One study based on SNP microarray data from 2500 individuals with no known disease showed that a CNV that is at least 100 kb is approximately harbored by 65% to 80% of individuals</p>
</blockquote>
<blockquote>
<p>[…] copy number variants (CNV) that could alter the diploid status of particular locus of the genome.</p>
</blockquote>
<p>One thing I didn’t know about the <strong>relation between stress and CNV</strong>.
Under stress (e.g. hypoxia) repair of broken replication may switch from homologous recombination to non-homologous repair.</p>
<div id="further-readings" class="section level3">
<h3>Further readings</h3>
<p>From the introduction:</p>
<ul>
<li><a href="http://www.annualreviews.org.proxy3.library.mcgill.ca/doi/abs/10.1146/annurev-genet-102209-163544">Human Copy Number Variation and Complex Genetic Disease</a> (review from E. Eichler).</li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166990">Population Analysis of Large Copy Number Variants and Hotspots of Human Genetic Disease</a> (SNP-array across 2500 healthy individuals).</li>
</ul>
</div>
</div>
<div id="discovery-and-characterization-of-alu-repeat-sequences-via-precise-local-read-assembly" class="section level2">
<h2>Discovery and characterization of Alu repeat sequences via precise local read assembly</h2>
<p>by Wildschutte et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/26503250"><em>Nucleic Acids Res.</em> 2015</a>.</p>
<p>Jeff Kidd’s group built on existing approaches to <strong>detect Alu insertion</strong> and attempted to <strong>assemble the full allelic sequence</strong>.
<em>ME-insertion supporting reads</em> are assembled using a classic <em>overlap-layout-consensus</em> strategy.
<strong>Low-coverage WGS</strong> from 53 samples (Human Genome Diversity Project) was <strong>pooled</strong>.
They also devised a <em>genotyping experiment</em> of their assembled insertion and validated through PCR.
Briefly, reads are remapped to both the reference and mutated region and mapping used to estimate genotype likelihoods.
With their assembled sequences they could <em>observe and quantify</em> <strong>premature insertion</strong> and <strong>5’ truncation</strong>.
<strong>Some</strong> variable Alu seem to arise <strong>from non-retrotransposition mechanisms</strong>.</p>
<p>One of the <strong>challenge</strong> with short-read data seems to be <strong>colocation of reference and non-reference repeats</strong>.
For example, prediction within 500 bp were removed in this study.
Later they estimate from simulation that they might miss half of the insertions because of this criteria, consistent with the number from Chaisson et al.</p>
<p><em>Breakpoint determination seems tricky</em>.
They had to use a series of (old) programs to align, orient, realign the assembled sequence.</p>
<p>They used <em>PacBio from CHM1 to validate the assembly</em>.
In fact they ran their analysis on short-read data for the haploid cell line.
It’s not exactly the same as their pooled analysis but the best they could do.</p>
<p>Interesting facts:</p>
<ul>
<li>AluY subfamilies is still mobile, especially AluYa5 and AluYb8.</li>
<li>Active Alus contains a RNA Pol III promoter that mobilizes L1 machinery.</li>
<li>Same as L1, inserted Alu are flanked by target site duplication and a poly-A tail.</li>
</ul>
<p>They cite a few modern methods that uses split-read, soft-clipped and unmapped reads to detect repeat insertion:</p>
<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23233656">RetroSeq</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23921633">alu-detect</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745252">Tea</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348035">Mobster</a>.</li>
</ul>
</div>
