---
title: Epilepsy
date: 2017-04-02
tags: ['literature']
output: blogdown::html_page
---



<div id="duplication-hotspots-rare-genomic-disorders-and-common-disease" class="section level2">
<h2>Duplication hotspots, rare genomic disorders, and common disease</h2>
<p>by Mefford et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19477115">Current Opinion in Genetics and Development 2009</a>.</p>
<p><strong>Segmental duplications sensitize ~10% of the genome</strong> to recurrent microdeletions and microduplications.
Mefford et al <strong>review</strong> the association of these <strong>recurrent events with diseases</strong>.
A subset of these hotspots plays an important role in <strong>austism, schizphrenia, and epilepsy</strong>.
In general, the hotspots are characterized by a <strong>stretch of unique sequence</strong> (50 kb-10 Mb) <strong>flanked by large</strong> (&gt;10 kb), highly <strong>homologous</strong> (&gt;95%) <strong>segmental duplications</strong>.
This context provides substrate for <strong>NAHR</strong>.</p>
<p>More than 15% of mental retardation syndromes seem to involve micro-deletion/duplications.
Many are non-recurrent and doesn’t involve NAHR.
However, recurrent <em>de novo</em> microdeletions in 17q21.31 were observed in aCGH screens, but not in controls.
This particular hotspots is interesting because an <em>inversion of ~900 kb</em> with frequency 20% in Europeans, seem to be a <em>prerequisite</em> for the microdeletion.
They also found other similar microdeletions in <em>15q24</em>, <em>16p11.2-p12.2</em>, <em>10q22-q23</em>.
In these studies <em>recurrent usually means just a few cases</em>, i.e. frequencies ~1-2%.</p>
<p><strong>Three hotspots</strong> are enriched in <strong>multiple neuropsychiatric diseases</strong>: <em>16p11.2</em>, <em>1q21.1</em> and <em>15q13.3</em></p>
</div>
<div id="genome-wide-copy-number-variation-in-epilepsy-novel-susceptibility-loci-in-idiopathic-generalized-and-focal-epilepsies" class="section level2">
<h2>Genome-Wide Copy Number Variation in Epilepsy: Novel Susceptibility Loci in Idiopathic Generalized and Focal Epilepsies</h2>
<p>by Mefford et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20502679">PLoS Genetics 2010</a>.</p>
<p>Epilepsy has a <strong>prevalence ~1%</strong> and a <strong>lifetime incidence up to 3%</strong>.
Using <em>custom aCGH</em> in <strong>517 individual</strong> with mixed types of epilepsy, they found one or more <strong>rare genic CNVs in 8.9%</strong> of them.
In addition, 2.9% of the patients carry deletions at genomic hotspots: <strong>15q11.2, 15q13.3 or 16p13.11</strong>.</p>
<p>Individuals had idiopathic generalized epilepsy (IGE), benign epilepsy with centrotemporal spikes, and other types of idiopathic seizure disorders.
The custom aCGH had a higher probe density around 107 regions of the genome flanked by large highly homologous duplications.
The CNVs found in the 8.9% of the probands were not found in any of their 2493 controls.
The <strong>majority are deletions</strong> (69%).
The size range from 13 kb to 15.9 Mb, with an average of 1.2 Mb.</p>
<p>3.9% of the probands had CNVs at known rearrangement hotspots.
2.9% have deletions that are either known pathogenic or significantly enriched in the epilepsy cohort.
Most of the affected probands had IGE, these CNVs actually account for 3.3% of IGE patients.</p>
<p>Looking at non-hotspots CNVs, they found <strong>maximum two patients with the same gene affected</strong>, most of the CNVs being unique to the proband.
Two of these genes are likely important: <em>CNTNAP2</em> and <em>AUTS2</em>.
Although, 5 individuals had 2 rare genic CNVs, it’s not clear how different they are from the rest in term of phenotype.</p>
</div>
<div id="rare-copy-number-variants-are-an-important-cause-of-epileptic-encephalopathies" class="section level2">
<h2>Rare copy number variants are an important cause of epileptic encephalopathies</h2>
<p>by Mefford et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/22190369">Annals of Neurology 2011</a>.</p>
<p>Here, <strong>315 patients</strong> with EE were screened using <em>aCGH</em>.
They found <strong>rare CNVs in 7.9%</strong> of the patients, half of them being clearly or likely pathogenic.
Some were <em>de novo</em>.
They also found some patients with CNVs in the <strong>known “hotspots”</strong>.
Apparently their results “illuminate several novel candidates genes”, but their attempts at getting more support failed (no mutations when resequencing).
Also with this low sample size and CNV recurrence, I can’t help but wonder how many such rare CNV would be there by chance.</p>
<p><strong>Epileptic encephalopathies</strong> are the most devastating group of epilepsy.</p>
<p>CNVs were filtered like this:</p>
<ul>
<li><blockquote>
<p>5 probes</p>
</blockquote></li>
<li>overlap RefSeq genes</li>
<li>not entirely within segmental duplication</li>
<li>not overlapping (&gt;50%) with their 4,519 controls (=rare).</li>
</ul>
<p>All events were also visually inspected in a genome browser.
Some rare CNVS were validated with another array.
They also did some targeted Sanger sequencing.</p>
<p>Again <strong>deletions</strong> in known <strong>epilepsy genes</strong>, or <strong>de novo</strong>, were considered <strong>pathogenic</strong>.
<em>De novo</em> duplications were considered likely pathogenic, same as CNVs &gt;1Mb.</p>
<p>Known epilepsy genes: <em>CDKL5</em>, <em>UBE3A</em>, <em>CNTNAP2</em>.</p>
<p>In the discussion, they mentioned a <strong>significant excess of large deletions</strong> in their cohort compared to controls.
2.2% of their patients have a deletion larger than 1 Mb while only 0.3% in the controls.</p>
<p>The controls came from two independent studies, not using exactly the same arrays…</p>
<p>Many gene stories fillers but nothing really convincing to support the results of their screen.</p>
<p>“Context matters” gem:</p>
<blockquote>
<p>Interestingly, our patient was also obsese.</p>
</blockquote>
</div>
<div id="clinical-significance-of-rare-copy-number-variations-in-epilepsy" class="section level2">
<h2>Clinical Significance of Rare Copy Number Variations in Epilepsy</h2>
<p>by Striano et al, <a href="http://archneur.jamanetwork.com/article.aspx?doi=10.1001/archneurol.2011.1999">Archives of Neurology 2012</a></p>
<p>Defining <em>rare</em> CNV as not seen in DGV, they found similar number of CNVs in epilepsy patients and controls but they were significantly larger and affecting more genes.
For example CNVs larger than 1 Mb and CNVs more than 10 genes were more frequent in epilepsy patients.</p>
</div>
<div id="copy-number-variation-plays-an-important-role-in-clinical-epilepsy" class="section level2">
<h2>Copy number variation plays an important role in clinical epilepsy</h2>
<p>by Olson et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24811917">Annals or Neurology 2014</a>.</p>
<p>In a more <strong>clinical</strong> setting, 805 patients with epilepsy were screened using aCGH arrays.
CNVs were observed in 323 patients.
The epilepsy phenotype was <strong>explained by a CNV</strong> in at least <strong>5%</strong> of the cases.
As usual, some CNVs located in the genomic “hotspots”.</p>
<p>It seems like they kept <strong>any CNV</strong> larger than 18 kb.
They didn’t mention filtering out common CNVs or CNVs found in controls or healthy individuals.
For some patients they screened the parent to assess the mode inheritance.
They did find some <em>de novo</em> CNVs.</p>
<p>They found overlapping CNVs in the <strong>Xp22.3</strong> but <strong>without controls</strong> I’m not completely convinced that it is relevant.
Moreover, looking more into this recurrent locations, there was no distinction between patients with deletion or duplication, and no evidence of epilepsy in relatives carrying the CNV.</p>
<p>They used the set of known epilepsy genes and genomic hotspots to classify CNVs as pathogenic.
They also performed a <strong>pathway analysis</strong>, that nicely pointed at neurodevelopment, synaptic transmission, ion channels.</p>
</div>
<div id="structural-genomic-variation-in-childhood-epilepsies-with-complex-phenotypes" class="section level2">
<h2>Structural genomic variation in childhood epilepsies with complex phenotypes</h2>
<p>by Helbig et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24281369">ELHG 2014</a>.</p>
<p>Rare CNVs, sometimes <em>de novo</em>, were found in patients with unclassified epilepsies and complex phenotypes.
In this study, 222 patients were screened using aCGH/SNP-array for CNVs larger than 100 kb.
88 rare CNVs in 71 patients (31.9%) were discovered.
18 of these were considered <strong>pathogenic</strong>, and 15 <strong>likely pathogenic</strong>.
Out of the 55 CNVs of unknown significance the <strong>deletion/duplication ratio was ~1:2</strong>.
11 were <em>de novo</em> out the 42 that could be assessed, including all four variants larger than 3 Mb.
No novel recurrent CNVs larger than 100 kb were detected.</p>
<p>Patients were recruited in three centers.
One center used aCGH and the other two SNP-arrays.</p>
<p>They looked for CNV:</p>
<ol style="list-style-type: decimal">
<li>larger than 100 kb.</li>
<li>overlapping a known gene.</li>
<li>either:
<ul>
<li>not overlapping a benign CNV (from DGV) (=rare).</li>
<li>overlapping known pathogenic CNV.</li>
</ul></li>
</ol>
<p>They classified the CNVs as:</p>
<ul>
<li><strong>pathogenic</strong> if <em>de novo</em> deletion or known associated with epilepsy.</li>
<li><strong>likely pahotgenic</strong> if <em>de novo</em> duplication or inherited CNVs larger than 1 Mb.</li>
<li><strong>unknown significance</strong> otherwise.</li>
</ul>
<p>Less interesting findings:</p>
<ul>
<li>Some CNVs located in known hot-spot regions (see refs) and were considered pathogenic (<em>so not following their previous definition…</em>).</li>
<li>Patients with dysmorphic features had a higher CNV burden, mostly due to CNVs of unknown significance and likely pathogenic.</li>
</ul>
<p>To go further:</p>
<ul>
<li>CNVs in epilepsy: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2873910/">Genome-wide copy number variation in epilepsy: novel susceptibility loci in idiopathic generalized and focal epilepsies. Mefford et al, 2010</a>.</li>
<li>CNVs in epilepsy: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20398883">Rare deletions at 16p13.11 predispose to a diverse spectrum of sporadic epilepsy syndromes. Heinzen et al, 2010</a>.</li>
<li>CNV hotspots: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19477115">Duplication hotspots, rare genomic disorders, and common disease. Mefford and Eichler, 2009</a>.</li>
</ul>
</div>
<div id="copy-number-variant-analysis-from-exome-data-in-349-patients-with-epileptic-encephalopathy." class="section level2">
<h2>Copy number variant analysis from exome data in 349 patients with epileptic encephalopathy.</h2>
<p>by Mefford et al (Epi4K consortium), <a href="http://www.ncbi.nlm.nih.gov/pubmed/26068938">Annals of Neurology, 2015</a>.</p>
<p><strong>Exome sequencing</strong> of 349 trios with <strong>epileptic encephalopathy</strong> identified <strong>18</strong> <em>de novo</em> CNVs in 17 patients (<strong>4.8%</strong>).
10 of these are likely pathogenic.</p>
<p>CNV calling was performed by <strong>CoNIFER pipeline</strong>.
Exome-reads were mapped to exons (+- 300 bp) using mrsFAST.
30 SVD components were removed to reduce technical bias.
They also <strong>filtered</strong> the segments called based on the number of consecutive bins and signal strength.
They <strong>excluded CNVs</strong> with 50% or more overlapping <strong>repetitive or duplicated regions</strong>.
CNV calls were <strong>manually curated</strong> (<em>whatever that means…no supplementary info…only good ol’ 32 pages of tables in PDF…</em>).
Finally, they filtered out CNVs observed in <strong>more than 1% of controls</strong> (aCGH/WES).</p>
<p>The <em>de novo</em> CNVs and some of the larger inherited ones were <strong>validated</strong> by “oligonucleiotide” (<em>again no supplementary information</em>) and SNP-array.
A <em>de novo</em> CNV was considered <strong>pathogenic</strong> if previously associated to epilepsy or neuro-disorders.</p>
<p>I’m not sure about what/how they did their validation.
It seems that they first took 43 probands and tried to validate all the calls.
There, 53 of 80 (<strong>66%</strong>) inherited CNVs and 5 of 21 (<strong>24%</strong>) <em>de novo</em> CNVs were <strong>confirmed</strong>.
But then they also confirmed 18 <em>de novo</em> in 17 patients.</p>
<p>The confirmed <em>de novo</em> CNVs ranged from 94 kb to 16 Mb, and 1 to 163 genes.
10 patients had a pathogenic <em>de novo</em> CNV.
Some <em>de novo</em> CNVs overlapped known <strong>EE genes</strong>: <em>SCN1A</em>, <em>SCN2A</em> and <em>GABRB3</em>.</p>
<p>They had previously studied <em>de novo</em> <strong>SNV</strong> in more or less the same cohort, and they compared the affected genes.
Some overlapped…</p>
</div>
<div id="analysis-of-rare-copy-number-variation-in-absence-epilepsies" class="section level2">
<h2>Analysis of rare copy number variation in absence epilepsies</h2>
<p>by Addis et al, <a href="http://www.ncbi.nlm.nih.gov/pubmed/27123475">Neurology Genetics 2016</a>.</p>
<p>They looked at CNVs in <strong>144 children</strong> with <strong>absence epilepsy</strong> using SNP-arrays.
<strong>11%</strong> of the patients had a <strong>rare CNVs</strong> that disrupt genes associated with neuronal development and function.
They also found some patients with CNVs in the <strong>known “hotspots”</strong>.</p>
<p>They selected CNVs following those criteria:</p>
<ul>
<li>&gt;12 supporting probes and &gt;20 kb</li>
<li>Not overlapping (&gt;90%) DGV CNVs with frequency &gt; 0.1% (=rare).</li>
<li>CNV overlapping at least one exon.</li>
</ul>
<p>The <strong>pathogenicity</strong> was based on the gene content, size, frequency and overlap with relevant genes (epilepsy and other neuro-diseases).
They don’t detail their criteria more…</p>
<p>CNVs were validated by real-time quantitative PCR.</p>
<p>Apparently, <strong>cell adhesion</strong> is a mechanism that is thought to contribute to epileptogenesis.</p>
</div>
<div id="cenet" class="section level2">
<h2>CENet</h2>
<p><a href="https://biotechnologyfocus.ca/public-private-partnership-to-seek-personalized-treatment-approach-for-drug-resistant-epilepsy/">Information gathered here</a></p>
<ul>
<li><em>Personalized Medicine in the Treatment of Epilepsy</em></li>
<li><strong>Identify variants</strong> that predispose for <strong>epilepsy</strong>.</li>
<li><strong>Identify variants</strong> that predicts <strong>drug response</strong>.</li>
<li>20 anti-epileptic drugs, effective in ~70% of patients.</li>
<li>ID associated with higher resistance.</li>
<li>ID in 25% of the patients.</li>
<li>Particularly problematic in children.</li>
</ul>
<p>The goal is to identify genetic changes that predispose people to epilepsy and changes that predict response to different anti-epileptic drugs.</p>
<blockquote>
<p>Although there are over 20 different anti-epileptic drugs available today, these drugs are ineffective in about one third of patients. Epilepsy is particularly problematic in children since uncontrolled seizures in the developing brain largely contribute to cognitive decline. Managing epilepsy associated with intellectual disability is especially challenging since these symptoms are associated with a high rate of resistance to anti-epileptic drugs.</p>
</blockquote>
<p>ASHG 2014 abstract:</p>
<blockquote>
<p>Epilepsy is a common complex neurological disorder characterized by recurrent episodes of paroxysmal abnormal electrical discharges in the brain (seizure). In 65% of epileptic individuals seizures occur without detectable lesion in the brain. Intellectual disability (ID) is observed in ~25% of epileptic individuals. Although treatment is effective in the majority of cases, more than 30% of patients are resistant to anti-epileptic drugs (AEDs). In this project, we hypothesize that pharmaco-resistant epilepsies (PRE) have different etiologies than other epilepsy syndromes associated with the same clinical manifestations, and posit that genetic factors can be used to predict response to AEDs. In 2013, we united Canadian clinicians and investigators to create the Canadian Epilepsy Network (CENet). In order to identify highly penetrant genetic variants that can predict resistance to AEDs, we will conduct whole genome sequencing (WGS) on 1000 epileptic individuals with extremes of response to AED treatment, including 200 multiplex families, as well as 200 epileptic trios (patient + both parents) with severe epilepsy and ID. Based on our previous studies in patients with ID or autism spectrum disorders, we expect a high rate of de novo mutation for epilepsies associated with these co-morbidities. Potentially causative genes/variants will be validated in additional epilepsy samples available through our large collaborative network with similar but complementary projects in the US (Epi4K - 4,000 epilepsy genomes) and Europe (EpiPGX). We will also assess the clinical utility of whole genome sequencing as a diagnostic test for epilepsy in a prospective cohort of 600 epileptic individuals. So far, pre-screening of 36 known genes for epileptic encephalopathy (EE) in 95 patients led us to identify 11 causative mutations in total (11.6%). At the end of the project, we expect to identify 50 novel genes causing severe forms of epilepsy and PRE. Using this catalog of gene variants, we will develop and validate a diagnostic resource (PRE-GENE) to implement WGS as the gold-standard diagnostic test for PRE in clinical practice.</p>
</blockquote>
</div>
<div id="animal-models-of-epilepsy" class="section level2">
<h2>Animal Models of Epilepsy</h2>
<p>by Kandratovicius et al. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4164293/">Neuropsychaitric Dis Treat. 2014</a></p>
<p>Some models replicate <strong>symptomatic epilepsies</strong>, i.e. where the epilepsy is a symptom of an underlying problem, such as an injury, an infection, a congenital brain malformation, a tumour, or a metabolic disorder</p>
<p>A few other are genetic models of absence seizure but the genetic part is more used for convenience than for investigating the genetic factors.</p>
<ul>
<li>Epilepsy Vs Chronic epilepsy. Are chronic just more recurrent ?</li>
<li><strong>Chemoconvulsant</strong> used on mice models to induce chronic epilepsy. Expensive.</li>
<li><strong>Electrical stimulation</strong> are useful to study brain damage and after effects but are not cell specific and also expensive.</li>
<li>Kindling seems to be another form of electrical stimulation.</li>
<li>Young mice can also be exposed to 30 min of <strong>hot air</strong> (40C) which leads to convulsions. Same with <strong>hypoxia</strong>.</li>
<li>Fluid Percussion Injury is a machine to induce <strong>physical brain trauma</strong> to mice.</li>
<li>Audiogenic rats where acoustic stimulation triggers seizure. Useful but not really a genetic model.</li>
<li><strong>Monogenic mouse</strong> models of abscence epilepsy. Some <strong>rat</strong> models are polygenic and more realistic (e.g. responding to drugs).</li>
<li>They also mention <em>lethargic, tottering, leaner, and stargazer</em> mouse models.</li>
</ul>
</div>
<div id="epilepsygene-resource" class="section level2">
<h2>EpilepsyGene resource</h2>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4384015">Ran et al (NAR 2015)</a></p>
<ul>
<li>Analyzed more than 1000 publications.</li>
<li>Contains information for <strong>499 genes amd 3931 variants</strong> (mostly SNVs).</li>
<li>Variants annotated and scored by the number of tools predicting a <strong>deleterious effect</strong>.</li>
<li><em>154 high-confidence genes</em> with score <span class="math display">\[\ge10\]</span>.</li>
<li>Four co-expression modules from BrainSpan (RNA-seq of brain development).</li>
</ul>
</div>
