---
date: 2017-04-09
title: Family-based designs for gene-association studies
tags: ['literature']
output: blogdown::html_page
---



<p>Different type of family information can be used.
<strong>Trio designs</strong> (affected offspring plus parents) seems to have the most power but having both affected and unaffected siblings is useful too.</p>
<p>A positive result in a family-based test means there is both <strong>linkage</strong> and <strong>association</strong>.
It avoids problems with population stratification or admixture.</p>
<div id="transmission-disequilibrium-test-tdt" class="section level2">
<h2>Transmission Disequilibrium Test (TDT)</h2>
<p>The TDT compares the number of observed allele transmitted to the affected offspring with the expected number following Mendelian transmission.
It is completely non-parametric in the sense that it doesn’t require specifying a disease model.</p>
<p>In its simple form (complete trios), the TDT is more or less a Binomial test on the number of transmitted allele according to the number of heterozygous parents.
It cannot deal with missing parents or more general pedigrees.</p>
</div>
<div id="family-based-association-tests-fbats" class="section level2">
<h2>Family-Based Association Tests (FBATs)</h2>
<p>An extension of the TDT.</p>
<p>The FBAT statistic represents how the observed genotype diverges from the expected genotype in individuals with phenotype:</p>
<p><span class="math display">\[ U = \sum T * (X-E(X|P)) \]</span></p>
<p>with the sum being over all offspring and</p>
<ul>
<li><span class="math display">\[T=Y-\mu\]</span> the centered phenotype</li>
<li><span class="math display">\[X\]</span> the genotype</li>
<li>\(E(X|P)\) the expected genotype.</li>
</ul>
<p>The FBAT statistics is <span class="math display">\[\frac{U^2}{Var(U)}\]</span> and follows a <span class="math display">\[\chi^2\]</span> distribution with one degree of freedom (if at least 10 families).</p>
<ul>
<li>The genotype <span class="math display">\[X\]</span> can be coded to reflect the mode of inheritance.</li>
<li>Both \(E(X|P)\) and <span class="math display">\[Var(U)\]</span> are computed under the null hypothesis from the parents/siblings genotypes.</li>
</ul>
<p><em>TODO empirical estimate of the variance from Lake et al 2000.</em></p>
</div>
<div id="filtering-variants" class="section level2">
<h2>Filtering variants</h2>
<p>Another approach is to focus on (homozygous) segregating variants and annotate them to assess their functional impact (frequency, location).</p>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/16619052"><em>Family-based designs in the age of large-scale gene-association studies.</em> by Laird &amp; Lange (Nature Reviews Genetics 2006)</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11793685"><em>Family-Based Tests of Association in the Presence of Linkage.</em> by Lake et al. (AJHG 200)</a></li>
</ul>
</div>
