---
date: 2017-01-10
title: Lit Digest - SVs and TEs
tags: ['literature']
output: blogdown::html_page
---

## CrowdVariant: a crowdsourcing approach to classify copy number variants

[bioRxiv](http://biorxiv.org/content/early/2016/12/13/093526)

There is no good gold-standard dataset for small CNVs (<10 kb). 
CrowdVariant is a crowdsourcing framework that aim at identifying high quality CNV calls in NA24385, an Ashkenazim reference sample from Genome In A Bottle Consortium.

They show that even annotations from non-expert helps assigning copy number status from putative calls. 
They then provide 1 781 of such high confidence CNVs.

- They only analyze deletions but they have Illumina data for the trio and 10X genomics for the son. With 10X they can separate the haplotype (How exactly ?).
- A user is shown IGV images of the region, but have no information about which region it is, which technology or the weight of their decision.
- Some non-experts didn't do it well or didn't understand but most of the others could accurately distinguish CN0, CN1 or CN2.
- [svviz](https://github.com/svviz/svviz) is a semi-automated visualization tool to look a read mapping around SV breakpoints.


## New SV detection method: DevRO

[bioRxiv](http://biorxiv.org/content/early/2016/12/15/094474)

DevRO uses short insert paired-end reads and long-range mate-pairs. 
It uses PEM and read depth. 
It analyzes multiple samples/populations without the need for pairwise comparisons (like the aCGH-adapted caller). 
It also detects deletions in the reference assembly (novel insertion I guess).

It seems like they first count discordant mapping in 1 kb bins in each population. 
Then they retrieve the bins whose fraction of discordant are different between populations and test them better. 
I'm not sure about the rest of the method.

- In their analysis, they compared two populations (of 2 individuals) of rabbits.
- At first glance, the manuscript (text and figures) does not look convincing and there is no link to a software page.
- References: SVDetect (Zeitouni et al., 2010)

## Detecting large repeat exansion with ExpansionHunter

[bioRxiv](http://biorxiv.org/content/early/2016/12/19/093831)

> A recently discovered hexamer (GGCCCC) repeat expansion in the C9orf72 locus is a major cause of both ALS and frontotemporal dementia.

Repeat expansion are tricky to assess. 
Repeat-primed PCR with fragment length analysis is widely used but challenging to interpret, especially when indels are present. 
And they don't give an estimate of the length of the expansion. 
Southern blotting is the gold standard to estimate repeat length but can be challenging to set up.

The method uses the "in-repeat" reads to estimate the length of large repeat expansion. 
The challenges are finding these IRRs, making sure the mapping is good and estimating the length.

- PCR-free WGS (45x) of two ALS (amyotrophic lateral sclerosis) cohorts for a total of 3 001 samples !!!
- It's a C++ program ([github](https://github.com/Illumina/ExpansionHunter)).


## Detecting TE insertion from WGS data with McClintock pipeline

[bioRxiv](http://biorxiv.org/content/early/2016/12/19/095372)

> The primary goal of the McClintock pipeline is to lower the barrier to installation, use, and evaluation of multiple WGS-based TE  detection  methods.

- Impressive list of existing TE detection method.
- `ngs_te_mapper`, `TE-locate`, `PoPoolationTE`, `RetroSeq`, `RelocaTE` and `TEMP`.
- It was developped for yeast but seems to have in mind all the other organisms.
- [Self-assessment trap: Can we all be better than average ?](http://msb.embopress.org/content/7/1/537)


## Experimental evolution reveals hyperparasitic interactions among transposable elements

[PNAS](http://www.pnas.org/content/113/51/14763.short)

Seems like they experimentally reproduced the "selfish" evolution of TE using *mariner* TE in flies. 
They observed the known selfish properties, even within families where some non-autonomous members hijack autonomous ones (hyperparasitism).


## Detecting DNA methylation in *Alu* elements

[Genome Research](http://genome.cshlp.org/content/27/1/118.abstract)

Cancer cells exhibits widespread hypomethylation affecting large chromosomal domains. 
NSUMA (Next-generation Sequencing of UnMethylated Alu) is a new sequencing approach that enrich for unmethylated *Alu* and a flanking nonrepetitive sequence. 
It detects 10 times more unmethylated *Alu* in cancer cells compared to normal cells. 
Still, *Alu* are less hypomethylated than other TEs. 
They conclude that *Alu* elements contribute to the epigenetic changes but how do they know it originated from the *Alu* ?

+ Alu hypomethylation is associated with higher chromosomal instability.


## Detecting complex SV in cancer using CouGaR

[Genome Research](http://genome.cshlp.org/content/27/1/107.abstract)

CouGaR uses both read depth and discordant mapping. 
It identifies amplified regions across chromosomes and link them using the discordant mapping. 
Then the network-like problem is resolved by estimating the contigs and copy number estimates. 
See the nice figure 1.

They validated 15 out of 17 predicted fusion in pediatric gliomas.

They surveyed complex SVs across samples from TCGA. 
They selected 467 samples that are most likely harboring CGRs. 
Then I don't understand why they are surprised to see more contigs than in previous studies. 
They explain it might be the lack of method or small sample size while it's maybe their specific selection for CGRs...

> CouGaR identified the occurrence of ‘chromoplexy’ in nearly 63% of all prostate cancer samples and 30% of all bladder cancer samples.

Apparently chromoplexy is a complex SV event involving many different chromosomes.

- A stand-alone visualization tool is also available.
- Other similar methods: BAMBAM and nFuse.
