---
date: 2017-09-20
title: Lit Digest - SV and long-read sequencing
tags: ['literature']
output: blogdown::html_page
---



<ul>
<li>How many more SVs ?</li>
<li>Which types of SVs ?</li>
</ul>
<div id="identifying-structural-variants-using-linked-read-sequencing-data" class="section level2">
<h2>Identifying structural variants using linked-read sequencing data</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/09/18/190454">Elyanow, biorXiv 2017</a></p>
<ul>
<li>Better than other methods…</li>
</ul>
</div>
<div id="mapping-and-phasing-of-structural-variation-in-patient-genomes-using-nanopore-sequencing" class="section level2">
<h2>Mapping And Phasing Of Structural Variation In Patient Genomes Using Nanopore Sequencing</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/04/24/129379">Stancu, bioRxiv 2017</a></p>
<ul>
<li>2 patients with congenital abnormalities, with MinION (11x/16x).</li>
<li>Better at finding complex chromothripsis.</li>
<li>3K novel SVs, majority being duplication &lt;200 bp.</li>
<li>Phasing showed the de novo chromothripsis occurred in paternal chromosomes.</li>
<li>Helps a lot for complex SVs.</li>
</ul>
</div>
<div id="accurate-detection-of-complex-structural-variations-using-single-molecule-sequencing" class="section level2">
<h2>Accurate detection of complex structural variations using single molecule sequencing</h2>
<p><a href="http://www.biorxiv.org/content/early/2017/07/28/169557">Sedlazeck, bioRxiv 2017</a></p>
<ul>
<li><ol start="2" style="list-style-type: decimal">
<li>Method paper.</li>
</ol></li>
<li>Find much more novel than TGP ?</li>
<li>Show that many translocation calls from short read caller are wrong (85%).</li>
</ul>
</div>
<div id="hysa-a-hybrid-structural-variant-assembly-approach-using-next-generation-and-single-molecule-sequencing-technologies" class="section level2">
<h2>HySA: a Hybrid Structural variant Assembly approach using next-generation and single-molecule sequencing technologies</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411774">Fan, Genome Research 2016</a></p>
<ul>
<li>Method paper.</li>
<li>Better job than Chaisson’s SMRT-SV ?</li>
</ul>
</div>
<div id="discovery-and-genotyping-of-structural-variation-from-long-read-haploid-genome-sequence-data" class="section level2">
<h2>Discovery and genotyping of structural variation from long-read haploid genome sequence data</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411763">Huddleston, Genome Research 2016</a></p>
<ul>
<li>2 haploid genomes.</li>
<li>Long-read is much more sensitive.</li>
<li>Highest gain for “small” SVs: 7bp - 1 Kbp.</li>
<li>90% SVs are novel.</li>
<li>60% can be genotyped with short reads once alternate allele is included.</li>
<li>Genotyped in 30 TGP samples. Some SVs look fixed.</li>
<li>Haploid is very useful for detection.</li>
</ul>
</div>
<div id="resolving-the-complexity-of-the-human-genome-using-single-molecule-sequencing" class="section level2">
<h2>Resolving the complexity of the human genome using single-molecule sequencing</h2>
<p><a href="https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4317254">Chaisson, Nature 2015</a></p>
<ul>
<li>Resolve inversions, complex insertions and long tandem repeats.</li>
<li>Enrichment in long short STR regions and complex insertions (?).</li>
<li>Strong enrichment of STRs (15x) among closed gaps. 78% of the gaps were covered by STR more than 10%.</li>
<li>85% of the CNVs were novel, especially for insertions (92%).</li>
<li>2.8x enrichment of STR insertions compared to deletions. STR insertions were 2.9x longer.</li>
<li>STR insertions clustered at the last 5 Mbp of the chromosomes.</li>
<li>Thousands of STR insertions within genes, 2 in exons.</li>
<li>Other complex insertions enriched in regions already enriched in MEI.</li>
<li>L1 and SVA insertions are found adjacent to repeats, more than in 1KGP or short read catalogs.</li>
<li>68% of inversions flanked by inverted repeats.</li>
</ul>
</div>
<div id="assembly-and-diploid-architecture-of-an-individual-human-genome-via-single-molecule-technologies" class="section level2">
<h2>Assembly and diploid architecture of an individual human genome via single-molecule technologies</h2>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4646949">Pendleton, Nature Methods 2015</a></p>
<ul>
<li>Identified complex SVs</li>
<li>Phased SNVs and SVs</li>
<li>Identified large tandem repeats and suggest that TR are larger than in the reference genome.</li>
<li>Observed DNA transduction when MEI.</li>
<li>Complex STR-like insertion within STRs.</li>
<li>4.2% of SVs are complex.</li>
<li>Inversions are enriched for complexity (55%).</li>
</ul>
</div>
