---
date: 2017-09-20
title: Lit Digest - SV and long-read sequencing
tags: ['literature']
output: blogdown::html_page
---

- How many more SVs ?
- Which types of SVs ?

## Identifying structural variants using linked-read sequencing data

[Elyanow, biorXiv 2017](http://www.biorxiv.org/content/early/2017/09/18/190454)

- Better than other methods...

## Mapping And Phasing Of Structural Variation In Patient Genomes Using Nanopore Sequencing

[Stancu, bioRxiv 2017](http://www.biorxiv.org/content/early/2017/04/24/129379)

- 2 patients with congenital abnormalities, with MinION (11x/16x).
- Better at finding complex chromothripsis.
- 3K novel SVs, majority being duplication <200 bp.
- Phasing showed the de novo chromothripsis occurred in paternal chromosomes.
- Helps a lot for complex SVs.

## Accurate detection of complex structural variations using single molecule sequencing

[Sedlazeck, bioRxiv 2017](http://www.biorxiv.org/content/early/2017/07/28/169557)

- (2) Method paper.
- Find much more novel than TGP ?
- Show that many translocation calls from short read caller are wrong (85%).

## HySA: a Hybrid Structural variant Assembly approach using next-generation and single-molecule sequencing technologies

[Fan, Genome Research 2016](https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411774)

- Method paper.
- Better job than Chaisson's SMRT-SV ?

## Discovery and genotyping of structural variation from long-read haploid genome sequence data

[Huddleston, Genome Research 2016](https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5411763)

- 2 haploid genomes.
- Long-read is much more sensitive.
- Highest gain for "small" SVs: 7bp - 1 Kbp.
- 90% SVs are novel.
- 60% can be genotyped with short reads once alternate allele is included.
- Genotyped in 30 TGP samples. Some SVs look fixed.
- Haploid is very useful for detection.

## Resolving the complexity of the human genome using single-molecule sequencing

[Chaisson, Nature 2015](https://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4317254)

- Resolve inversions, complex insertions and long tandem repeats.
- Enrichment in long short STR regions and complex insertions (?).
- Strong enrichment of STRs (15x) among closed gaps. 78% of the gaps were covered by STR more than 10%.
- 85% of the CNVs were novel, especially for insertions (92%).
- 2.8x enrichment of STR insertions compared to deletions. STR insertions were 2.9x longer.
- STR insertions clustered at the last 5 Mbp of the chromosomes.
- Thousands of STR insertions within genes, 2 in exons.
- Other complex insertions enriched in regions already enriched in MEI.
- L1 and SVA insertions are found adjacent to repeats, more than in 1KGP or short read catalogs.
- 68% of inversions flanked by inverted repeats.

## Assembly and diploid architecture of an individual human genome via single-molecule technologies

[Pendleton, Nature Methods 2015](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4646949)

- Identified complex SVs
- Phased SNVs and SVs
- Identified large tandem repeats and suggest that TR are larger than in the reference genome.
- Observed DNA transduction when MEI.
- Complex STR-like insertion within STRs.
- 4.2% of SVs are complex.
- Inversions are enriched for complexity (55%).
