---
title: Single Cell RNA-seq and GBM
tags: ['literature']
date: 2017-05-23
output: blogdown::html_page
---



<p>Three studies used single cell RNA-seq on brain cancer tumors.
They also called large CNA from this data.
How ?</p>
<div id="scrna-highlights-intra-tumoral-heterogeneity-in-primary-glioblastoma" class="section level2">
<h2>scRNA highlights intra-tumoral heterogeneity in primary glioblastoma</h2>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4123637/">Patel et al. (Science 2014)</a></p>
<ul>
<li>Intra-tumoral heterogeneity is thought to be an important factor of treatment resistance and relapses.</li>
<li>GBM is one of the most lethal human malignancy.</li>
</ul>
<p>They show that known <strong>suptype</strong> classifier for GBMs are <strong>actually expressed</strong> differently <strong>across the cells</strong> from the <strong>same tumor</strong>.</p>
<p>To infer large CNA they <em>averaged</em> the <em>relative expression</em> across large regions.
They actually average the gene expression using a <em>sliding window</em> of 100 genes.
Hence for each of the 6,000 expressed gene they compute this average expression and use it to <em>cluster cells</em>.
For the <em>heatmap</em>, the signal was normalized using the control (bulk RNA-seq from normal brain samples).</p>
<ul>
<li>They profile 430 cells from 5 glioblastomas using SMART-seq.</li>
<li>They excluded genes and cells with low coverage. They ended up with ~6,000 genes.</li>
<li>They also have bulk RNA-seq from the tumor.</li>
</ul>
</div>
<div id="scrna-supports-developmental-hierarchy-in-human-oligodendroglioma" class="section level2">
<h2>scRNA supports developmental hierarchy in human oligodendroglioma</h2>
<p><a href="http://www.nature.com/nature/journal/v539/n7628/full/nature20123.html">Tirosh et al. (Nature 2016)</a></p>
<p>They used the <strong>same approach</strong> than in the previous study.
However they normalized the expression of each gene by the average across cells of the same tumor.
Then they <em>winsorized</em> these relative expression ([-3,3]) to limit the effect of outliers in the moving average estimation.</p>
<p>Here they didn’t have normals as control so <strong>they guessed which CNV-based cluster of cells were normal</strong>:</p>
<ul>
<li><em>consistent</em> CNV patterns</li>
<li>higher values at 1p/19q which are known losses in this cancer.</li>
</ul>
</div>
<div id="decoupling-genetics-lineages-and-microenvironment-in-idh-mutant-gliomas-by-scrna" class="section level2">
<h2>Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by scRNA</h2>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/28360267">Venteicher et al. (Science 2017)</a></p>
<p>Same lab, same idea.</p>
<p>scRNA QC:</p>
<ol style="list-style-type: decimal">
<li>Remove cells with less than 3000 expressed genes (at least one read).</li>
<li>Remove cells with average <span class="math display">\[log_2(\frac{TPM}{10} + 1)\]</span> below 2.5 in housekeeping genes.</li>
<li>Remove genes with <span class="math display">\[log_2(\bar{TPM}+1)\]</span> below 4.</li>
</ol>
<p>Then the relative expression is the <span class="math display">\[E_r = E - \bar{E}\]</span> for each gene, with <span class="math display">\[E=log_2(\frac{TPM}{10} + 1) \]</span>.</p>
<p>To identify normal cells they first performed a hierarchical clustering of all IDH-A cells and observed three clusters.
One of the cluster is suspected to be malignant cells.
For the CNVs they used the same approach as before but corrected it with two baselines derived from the two “normal” clusters.
So again, moving average of winsorized relative expression across 100 genes.
These values were corrected with the baseline as follows:</p>
<ul>
<li>0 if <span class="math display">\[BaseMin - 0.2 &lt; CNV &lt; BaseMax + 0.2\]</span></li>
<li><span class="math display">\[ CNV - BaseMin \]</span> if <span class="math display">\[CNV &lt; BaseMin - 0.2\]</span></li>
<li><span class="math display">\[ CNV - BaseMax \]</span> if <span class="math display">\[CNV &gt; BaseMax + 0.2\]</span></li>
</ul>
<p>Two strategies were used to classify a cell as malignant:</p>
<ol style="list-style-type: decimal">
<li>If the correlation of the gene expression profile was at least 0.3 superior when comparing with the average profile of the “malignant” cluster compared to the other two..</li>
<li>If the “CNV signal” was higher than 0.05 and correlation with the CNVs in the “malignant” cluster was higher than 0.5.</li>
</ol>
<p>A malignant cell was classified as malignant in both strategies or malignant in one and borderline in the other.</p>
</div>
<div id="pooling-and-deconvoluting-cells-for-normalization" class="section level2">
<h2>Pooling and deconvoluting cells for normalization</h2>
<p>I had an idea of merging cells into meta-cells to call CNVs.
That way the higher depth allows for a much higher resolution.</p>
<p>Lun, Bach &amp; Marioni (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0947-7">Genome Biology 2016</a>) developed a method using this strategy for normalization.</p>
<p>Their objective is to avoid problems created by the high number of genes with null (or extremely low) expression.
They merge cells and compute a normalization factor for each pool.
Then these estimations are deconvoluted to get a normalization factor for each cell.</p>
<p>The result of each pool is used to build a system of linear equations to resolve.</p>
<p>There are a few additional tricks (e.g. how to build the pools or cluster the genes) but the main idea and tools are interesting.</p>
</div>
