---
date: 2017-09-22
title: Lit Digest - Non-coding variants in regulatory regions
tags: ['literature']
output: blogdown::html_page
---



<div id="genome-wide-analysis-of-noncoding-regulatory-mutations-in-cancer" class="section level2">
<h2>Genome-wide analysis of noncoding regulatory mutations in cancer</h2>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25261935">Weinhold &amp; Jacobsen et al., Nature Genetics 2014</a></p>
<ul>
<li>863 tumors from TCGA and co.</li>
<li>Identify <strong>recurrent non-coding variants in cancer</strong>.</li>
<li>PLEKHS1, WDR74, SDHD had such variants in upstream regulatory elements.</li>
</ul>
<p>Promoter/enhancer have <em>similar rate of somatic mutation</em> as transcribed regions (~3 per Mb), and lower than intergenic (~6 per Mb), supporting the presence of functional variants.</p>
<p>They used three approaches to identify functional ncSNV candidates:</p>
<ol style="list-style-type: decimal">
<li>A <strong>hotspot</strong> analysis, aiming at cluster of variants within 50 bp of each other.</li>
<li>A <strong>regional recurrence</strong> analysis, grouping the variants by regions and detecting recurrence compared to neighboring regions or the entire genome.</li>
<li>A <strong>TFBS</strong> analysis, to find mutations that create/disrupts ETS binding sites. Apparently ETS plays a role in cancer.</li>
</ol>
<p>Many of the top hits were located in regulatory regions of known cancer genes (pv=3.4e-4).</p>
<p>The most significant hotspot is in <strong>TERT promoter</strong> and consists of two recurrent mutations across 38 and 15 tumors, and <strong>7 cancer types</strong>.
TERT tended to be <strong>more expressed</strong> in mutated samples.
Fun fact: the C-&gt;T transitions create novel ETS binding sites.</p>
<p>The second most significant hotspot in in <strong>PLEKHS1 promoter</strong>.
Again two sites, with 11 and 12 mutations over 20 samples and <strong>5 cancer types</strong>.
The gene’s <strong>expression was lower</strong> in mutated samples.
In bladder cancer, 40% of the samples carried one of these two mutations.
Fun fact: also a C-&gt;T transition; each site is flanked by 10 bp palindromes.</p>
<p>The regional recurrence identified enrichment in larger regions, grouping the variants by functional class (enhancer, promoter, 5’ UTR, 3’UTR).
They tested both <strong>local enrichment</strong>, comparing the mutation rate in neighboring regions, and <strong>global enrichment</strong>, comparing the mutation rate in the same functional class and with similar DNA replication timing.
<em>I guess DNA replication timing affects the mutation rate ?</em></p>
<p>TERT was again the strongest hit.
The second and third hits were the promoter and 5’ UTR of <strong>WDR74</strong>.
This gene had no coding mutation but 35 non-coding variants in a ~1 Kb region at the promoter region.
No gene expression changes this time.
Overall, 5% of the tumors had a mutation in this region.</p>
<p>The TFBS approach intended to find mutation similar to the TERT examples, that either create or disrupt ETS binding sites.
From a short list of candidates, they focused on <strong>SDHD</strong> which was also high in the regional recurrence list.
In this case, C-&gt;T transitions disrupted a existing ETS binding sites, consistent with a <strong>reduced expression</strong> of the gene.
The mutations occurred <strong>exclusively in melanoma samples</strong>.
Because the variants were close enough to a coding region, they could use exome sequencing data to investigate more samples and even identified a third putative ETS site.
They showed that around 10% of melanoma samples have such a mutation and provided further support that these sites regulate the gene’s expression.
Samples with SDHD mutation had a significantly shorter overall survival.</p>
<p><strong>Interesting study</strong>.
I like how they used the strong/known association in TERT to generalize and identify other with weaker effect.
It also showed the usefulness of performing <strong>different analysis</strong> to better narrow down the list of leads to follow.
The way they <strong>used orthogonal data</strong> (RNA-seq, WES or Encode) to add support to the original findings was good too.
The paper was pleasant to read as well, not too dense and not too jargon heavy.</p>
</div>
