---
date: 2017-08-10
title: Testing the quality of an assembly
tags: ['literature']
output: blogdown::html_page
---



<div id="mapping-reads" class="section level2">
<h2>Mapping reads</h2>
<div id="coverage-analysis" class="section level3">
<h3>Coverage analysis</h3>
<p>Regions with unusual coverage might be mis-assembled (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23358380" title="Sequence assembly demystified">Nagarajan &amp; Pop</a>).</p>
<p>Coverage is too high -&gt; collapse of a repeat.
Coverage is too low -&gt; incorrect join.</p>
</div>
<div id="calling-snps" class="section level3">
<h3>Calling SNPs</h3>
<ul>
<li>An excess of SNPs might be a sign of mis-assemblies.</li>
<li>If assembling an haploid genome, there should not be any SNPs.</li>
</ul>
<p>In <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4248323" title="Single haplotype assembly of the human genome from a hydatidiform mole Karyn">Steinberg et al</a>, ~80K SNVs were found when aligning reads to their haploid assembly.
80% of those overlapped repeat regions.
34 regions (49 Mb) had a high SNV density (per Kbp, &gt;2 SD deviation).
They state that because these regions are duplicated, these SNVs likely come from paralogs.
They also mapped the reads to the reference genome and found ~100K heterozygous SNVs.
This also supports the hypothesis of paralogs and repeat sequences confusing the variant calling.
So not due to mis-assembly ?</p>
</div>
<div id="tools" class="section level3">
<h3>Tools</h3>
<blockquote>
<p>AMOSvalidate statistically analyzes mate-pair orientations and separations, repeat content, depth-of-coverage, correlated polymorphisms in the read alignments, and read alignment breakpoints to identify structurally suspicious regions of the assembly. The suspicious regions identified by individual metrics are then clustered and combined to identify (with high confidence) regions that are mis-assembled.</p>
</blockquote>
</div>
</div>
<div id="using-bac-ends" class="section level2">
<h2>Using BAC ends</h2>
<p>If I understand correctly the end sequences of a BAC clones are sequenced, maybe with Sanger sequencing (500 bp).
The BAC clone is also “fingerprinted” to estimate the insert-size.
So it’s some kind of super-mate pair information as the BAC can range from 80-350 Kbp and 500 bp flanks are sequenced.
The BAC ends are used to validate, order and join contigs (e.g. from finished BAC sequence).</p>
<p>Discordant orientation or size of the BAC ends mapped to the assembly identify mis-assembled regions.</p>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4248323" title="Single haplotype assembly of the human genome from a hydatidiform mole Karyn">Steinberg et al</a> used BAC ends to identify regions with potential deletions or insertions in the assembly.</p>
</div>
<div id="using-long-reads" class="section level2">
<h2>Using long reads</h2>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4248323" title="Single haplotype assembly of the human genome from a hydatidiform mole Karyn">Steinberg et al</a> used Pacbio reads to assess the quality of the assembly. Specifically they looked for regions with:</p>
<ul>
<li>low-coverage.</li>
<li>“cliffs”, i.e. abrupt drop in coverage.</li>
<li>inversions.</li>
</ul>
</div>
<div id="using-a-reference-genome" class="section level2">
<h2>Using a reference genome</h2>
<p><a href="https://gigascience.biomedcentral.com/articles/10.1186/2047-217X-2-10"></a> used COMPASS to compute three metrics from the alignment to a reference genome:</p>
<ul>
<li>validity: the fraction of aligned assembly</li>
<li>multiplicity: “ratio of the length of alignable assembled sequence to covered sequence on the reference; higher multiplicity implies expansion of repeats in the assembly, lower multiplicity implies repeat collapse”</li>
<li>parsimony: multiplicity divided by validity, i.e. size of the assembly divided by the aligned reference.</li>
</ul>
<p><img src="http://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F2047-217X-2-10/MediaObjects/13742_2013_Article_29_Fig7_HTML.jpg" /></p>
</div>
<div id="others" class="section level2">
<h2>Others</h2>
<p>Other approaches include comparison with transcriptome data or reference sequence of closely related organisms (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23358380" title="Sequence assembly demystified">Nagarajan &amp; Pop 2013</a>)</p>
</div>
