---
date: 2018-07-22
title: "Cool tools, 'machine-learning', and a population-specific reference genome"
tags: ['literature']
output: blogdown::html_page
---

I accumulated a huge number of articles in Feedly and I don't know what to do about them. 
One thing I decided to try was to go over them quickly and write down what I want to remember.
I don't think I will read these kind of posts again but maybe it will help me remember or eventually find the reference again.

**genomeview - an extensible python-based genomics visualization engine** ([bioRxiv](https://www.biorxiv.org/content/early/2018/06/26/355636), [docs](http://genomeview.readthedocs.io/en/latest/)).
`genomeview` implements a genomic visualization for python.
It can creates SVG file representing genomics annotation (transcripts/tracks) and reads (short/long) of a region.
This package nicely complement the bioinformatics python environment. 
I like that it integrates with jupyter notebooks or could be used for interactive applications.
Among my projects, this could be integrated to my local SV assembler by showing the alignment of reads on the newly assembled sequence versus the reference sequence.

**EnsembleCNV: An ensemble machine learning algorithm to identify and genotype copy number variation using SNP array data**. 
In [their preprint](https://www.biorxiv.org/content/early/2018/06/27/356667) the ensemble results from three methods are described: PennCNV, QuantiSNP and iPattern.
Part of the ensemble feature, CNV regions are derived from the calls from different methods and samples (using a network analysis and cliques finding), and then (re-)genotyped.
The machine learning part seems to be during the genotyping where a *local likelihood model is trained on both LRR and BAF*. 
Hum reading the methods quickly it seems like the *"machine learning"* is just Gaussian mixture modeling and this types of things...

**RBV: Read balance validator, a tool for prioritising copy number variations in germline conditions**. 
Because BAF information is *underutilised*, RBV package ([python](https://github.com/whitneywhitford/RBV)) model BAF to validate or prioritize CNVs. 
Here "prioritize" is in term of quality, not functional impact.

**The Qatar genome: a population-specific tool for precision medicine in the Middle East** ([Hum Genome Var 2016](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4927697/)).
1,161 Qataris were sequenced (~100 WGS and ~1,300 WES) and used to create a population-specific rederence that improved coverage and reduced the number of variants called.
The benefit of this reference varied in across ancestries.
One benefit of modifying the reference is that less variants needs to be interpreted and that those that are considered are likely rare variants.
Interesting: to liftover their genome they used ProgressiveCactus alignment and HalTools.
It's a bit strange how they computed the benefit of their reference in term of coverage, showing the sum of the bases mapped (overall or at modified bases).
Of note these numbers are shown in bar plots that don't start at 0.
They observed a 23% depth improvement at variant sites when mapping to their new reference.

**GenomicScores: seamless access to genomewide position-specific scores from R and Bioconductor** ([article](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty311/4987140), [Bioconductor](https://bioconductor.org/packages/GenomicScores)).
This R package simplify the utilization of position scores such as conservation.
Essentially two functions `score`/`gscore` can efficiently retrieve a summary statistic (e.g. mean) of scores in a genomic region.

**BioVR: a platform for virtual reality assisted biological data integration and visualization** ([bioRxiv](https://www.biorxiv.org/content/early/2018/04/25/307769)).
A virtual reality tool to manipulate DNA/RNA/protein structure using an Oculus Rift and hand detection.
More of a proof-of-concept than a completely usable/useful tool but cool!

**[The personal and clinical utility of polygenic risk scores](https://www.nature.com/articles/s41576-018-0018-x)**. A Nature Reviews Genetics lead by Eric Topol that looks interesting.
