---
date: 2021-02-24
title: "Functional SVs: GWAS, long-read for NDDs, disruption prediction"
tags: ['literature', 'functional', 'sv']
bibliography: [../../library-small.bib]
link-citations: true
output: blogdown::html_page
---



<div id="association-of-structural-variation-with-cardiometabolic-traits-in-finns" class="section level2">
<h2>Association of Structural Variation with Cardiometabolic Traits in Finns</h2>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.12.13.422502v1">bioRxiv</a> <span class="citation">Chen et al. (<a href="#ref-Chen2020a">2020</a>)</span></p>
<div id="from-the-intro" class="section level3">
<h3>From the intro</h3>
<ul>
<li>Targeted analysis of known SVs across large cohorts associated <em>HP</em> and <em>LPA</em> with cholesterol levels</li>
</ul>
</div>
<div id="in-this-study" class="section level3">
<h3>In this study</h3>
<ul>
<li>&gt;20x WGS</li>
<li>SV calling
<ul>
<li><a href="https://github.com/hall-lab/svtools">svtools</a>: discovery with LUMPY, CNVnator, svtyper; then genotyping.</li>
<li>GenomeSTRiP and CNVnator for cohort-level CNVs.</li>
<li>55k DEL, 32k DUP, 187 INV, 3k MEI, 34k mCNV (, 4k BND).</li>
</ul></li>
<li><strong>~130K SVs across 4-5K Finnish individuals</strong></li>
<li>35% overlap with 1KG and gnomAD</li>
<li>Advantage of the Finnish population: some rare variants enriched due to bottlenecks.</li>
<li>65K SVs tested for <strong>association with 116 traits</strong></li>
<li>Association using raw values when possible, e.g. allele balence from LUMPY or raw decimal copy numbers from CNVnator.</li>
<li><strong>WES and arrays for replication</strong> on 15K more individuals</li>
<li>31 hits at 15 loci
<ul>
<li><strong>2 novel loci</strong>:
<ul>
<li><em>ALB</em> promoter deletion: <span class="math inline">\(\nearrow\)</span> serum albumin, <span class="math inline">\(\searrow\)</span> cholesterol.</li>
<li><em>PDPR</em> multi-allelic CNV: pyruvate and alanine levels.</li>
</ul></li>
</ul></li>
<li>The <em>PDPR</em> CNV is in a complex region: flanked by segmental duplications and were two recurrent CNVs overlap.</li>
<li>5/6 replicated have stronger SNV hits</li>
<li><em>HP</em> recurrent deletion associated with cholesterol and <span class="math inline">\(\nearrow\)</span> glycoprotein.</li>
</ul>
</div>
<div id="questions" class="section level3">
<h3>Questions</h3>
<ul>
<li>What about rare variants? Any burden test attempts?</li>
<li>Did they consider using specialized tools to detect MEI? And to assess STR variation?</li>
</ul>
</div>
<div id="glossary" class="section level3">
<h3>Glossary</h3>
<ul>
<li>cholesterol: essential lipid required to build and maintain membranes, and modulate membrane fluidity.</li>
<li>glycoprotein: protein with carbohydrates attached that are use, for example, for cell-cell interaction.</li>
<li>albumin: transport protein, of which serum albumin is the main protein of blood plasma.</li>
<li>pyruvate: versatile molecule used as intermediate for multiple pathways, including the one to break/make glucose.</li>
<li>alanine: amino acid, maybe relevant here because interchangeable with pyruvate?</li>
</ul>
</div>
</div>
<div id="long-read-genome-sequencing-for-the-diagnosis-of-neurodevelopmental-disorders" class="section level2">
<h2>Long-read genome sequencing for the diagnosis of neurodevelopmental disorders</h2>
<p><a href="https://doi.org/10.1101/2020.07.02.185447">bioRxiv</a> <span class="citation">Hiatt et al. (<a href="#ref-Hiatt2020">2020</a>)</span></p>
<div id="summary" class="section level3">
<h3>Summary</h3>
<p>PacBio CCS found potentially pathogenic SVs in 2 out of 6 proband-parent trios with unexplained neurodevelopmental disorders (NDDs).</p>
</div>
<div id="why-look-for-svs-in-ndds-using-long-reads" class="section level3">
<h3>Why look for SVs in NDDs using long reads?</h3>
<ol style="list-style-type: decimal">
<li>Majority of NDDs are still undiagnosed.</li>
<li>Some NDDs should result from highly penetrant varaints missed by other tests.</li>
</ol>
</div>
<div id="methods" class="section level3">
<h3>Methods</h3>
<ul>
<li>SNV/indels calling with bwa-mem + DeepVariant + GLNexus</li>
<li>SV calling with pbmm2 + pbsv</li>
<li><em>de novo</em> assemblies sandbox included canu, Flacon unzip, HiCanu, hifiasm</li>
<li>Final assembly appoach was trio-binned hifiasm using short reads from the parents.</li>
<li>SV calling from the <em>de novo</em> assemblies just from the minimap2 alignments + bedtools.</li>
<li><a href="https://cube.univie.ac.at/gepard">Gepard</a> to make dotplots</li>
</ul>
<p>They also used a list of 35 known low-complexity repeats associated to diseases which they visualized on IGV.</p>
<p>Note: The (negative) Illumina short reads pipeline had used DRAGEN+GATK, Delly+CNVnator+ERD+Manta+MELT.</p>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<ul>
<li>~56K SVs per trio</li>
<li>~59 candidate <em>de novo</em> SVs per proband</li>
<li><strong>~35-45 Mb N50 for the hifiasm assemblies</strong></li>
<li>MEI that looked <em>de novo</em> in the short reads were real but had been missed in the parents.</li>
<li>Much less <em>de novo</em> MEI calls in the CCS data.</li>
</ul>
<div id="de-novo-l1-mediated-insertion-causing-frameshift" class="section level4">
<h4><em>de novo</em> L1-mediated insertion causing frameshift</h4>
<p>One of the SV is a <em>de novo</em> L1-mediated insertion which result in the duplication of exon 3 of <em>CDK5</em> and creates a frameshift.
Mosaicism is suspected because they also observed paternal reads without the insertion.
The insertion is made up of:</p>
<ul>
<li>~4.3 Kbp of L1HS sequence</li>
<li>~2.6 Kbp of sequence similar to an exon of a nearby gene</li>
<li>119 bp with the duplicated exon 3 of <em>CDKL5</em></li>
</ul>
<p>They confirmed the variant by Sanger and also that the duplicated exon is spliced in the mRNA.
<strong>Very cool!</strong></p>
</div>
<div id="complex-multi-svs-events" class="section level4">
<h4>Complex multi-SVs event(s)</h4>
<p>The other proband has some complex <em>de novo</em> translocations and inversions affecting 7 genes, including one (<em>MLLT3</em>) <strong>whose expression seems disrupted</strong> (~40% decrease).
Another gene, <em>DGKB</em>, is also predicted to be affected by those rearrangements.
The <em>de novo</em> status, number of genes affected (including on predicted haploinsufficient), and total number of SVs are the most compelling evidence of pathogenicity in this proband.
Like in the other patient, all these SVs are predicted to be on the paternal allele.</p>
</div>
</div>
<div id="references-to-read" class="section level3">
<h3>References to read</h3>
<ul>
<li>Torene RI, Galens K, Liu S, et al. Mobile element insertion detection in 89,874 clinical exomes. <a href="doi:10.1038/s41436-020" class="uri">doi:10.1038/s41436-020</a></li>
<li>Gardner EJ, Prigmore E, Gallone G, et al. Contribution of retrotransposition to developmental disorders. <a href="doi:10.1038/s41467-019-12520-y" class="uri">doi:10.1038/s41467-019-12520-y</a></li>
</ul>
<hr />
<blockquote>
<p>Parents were sequenced, at a relatively reduced depth, to facilitate identification of de novo variation.</p>
</blockquote>
<p>Which I read as if it was the reduced depth that helped “finding” de novo variants…</p>
</div>
</div>
<div id="functional-annotation-of-rare-structural-variation-in-the-human-brain" class="section level2">
<h2>Functional annotation of rare structural variation in the human brain</h2>
<p><a href="https://doi.org/10.1038/s41467-020-16736-1">Nature Communications 2020</a> <span class="citation">CommonMind Consortium et al. (<a href="#ref-CommonMindConsortium2020">2020</a>)</span></p>
<div id="summary-1" class="section level3">
<h3>Summary</h3>
<p>WGS and RNA-seq was leveraged to explore the effect of SVs on gene expression and devise a prediction approach to annotate rare CNVs.</p>
</div>
<div id="methods-1" class="section level3">
<h3>Methods</h3>
<ul>
<li>Data from the CommonMind Consortium: &gt;1000 brain bank samples, many with SCZ and bipolar disorder.</li>
<li><strong>WGS and RNA-seq from 629 samples</strong></li>
<li>Same SV pipeline as for gnomAD-SV: Delly, Lumpy, Manta, Wham, MELT, CNVnator, GenomeSTRiP, cn.MOPS</li>
<li>Rare SVs are treated as heterozygous.</li>
<li>Regulatory regions: CTCF sites, enhancers from brain tissue, promoters (2 kbp upstream of TSS).</li>
<li>Control “non-functional” SVs if not overlapping any annotation. Either <em>intronic</em> or <em>intergenic</em>.</li>
<li>Z-score based on expression in non-carriers.</li>
<li>Compare z-score outliers for genic SVs vs non-genic SVs. Note: where did the “60” for the Bonferroni correction come from?</li>
<li>Enhancers linked to genes using Hi-C data (PsychENCODE)</li>
<li>Linear model between z-score and:
<ul>
<li>proportion of exonic sequence</li>
<li>proportion of the promoter</li>
<li>sum proportion of all affected enhancers. <code>what's affected?</code></li>
<li>wheter SV and gene are in same TAD</li>
<li>SV length</li>
</ul></li>
<li>Compare with genic intolerance scores (ExAC/gnomAD)</li>
<li>Aggregate regulatory disruption scoreas <strong>sum z-scores of each affected gene weighted by the intolerance score</strong>.</li>
<li>Annotated CNVs across ~15K genomes with WGS
<ul>
<li>including those overlapping known pathogenic CNVs from NDD ClinGen track.</li>
</ul></li>
</ul>
</div>
<div id="results-1" class="section level3">
<h3>Results</h3>
<ul>
<li>~116K SVs, &gt;90% of which are CNVs or MEI, and ~80% are smaller than 1 Kbp</li>
<li>~89K SVs are rare (AF&lt;1%)</li>
<li>AF is lower for SVs in coding region, enhancer, and CTCF sites (although maybe some SV size bias, see below).</li>
<li>The more exons affected the stronger the expression disruption.</li>
<li>TSS more sensitive than transcription end.</li>
<li>Consistent Z-score shift for genic duplications/deletions…</li>
<li>…although most complete gene deletions/duplications don’t lead to expression outliers.</li>
<li>Expression outliers also enriched in <strong>genic inversions</strong>.</li>
<li>Linear model expression-z-score vs annotation:
<ul>
<li>exon and promoter proportion correlated in expected direction with deletions and duplications.</li>
<li><span class="math inline">\(\nearrow\)</span> enhancer proportion by duplications (and deletion but not significant), <span class="math inline">\(\searrow\)</span> expression</li>
<li>SV-gene co-occurence in same TAD only significant for deletions (<span class="math inline">\(\searrow\)</span> expression)</li>
</ul></li>
<li>I think the comparison with intolerance scores focused on each annotation or the SV frequencies, but didn’t explore their relation to the observed transcriptomic disruption. For example, I’d have liked to know if SVs were more likely to have an impact on intolerant genes.
<ul>
<li>I don’t understand Fig 4 bcd. The way I understand, expecting 20% of CNVs in each gene quintile, wouldn’t be ideal I think.</li>
</ul></li>
<li>Pathogenic deletions and duplications associated with NDDs have higher “disruption score”.</li>
<li>“Disruption score” is better at ranking pathogenic CNVs than the traditional size+frequency considerations.</li>
<li>This study highlights <strong>SVs that alters CTCF sites</strong>.</li>
</ul>
</div>
<div id="figure-2-is-more-about-sv-size-effect-than-the-functional-class" class="section level3">
<h3>Figure 2 is more about “SV size” effect than the functional class?</h3>
<p>The effect of SV size was controlled by matching each functional SV with a non-functional SV of similar size.
That way, the two sets of SVs compared with the Wilcoxon test have a similar size distribution.
Although this works for the test comparing two sets of SVs, the AF numbers in the text and Figure 2 don’t take this adjustment into account (I think).
If so there is not one “intergenic AF”, there should be one adjusted for each comparison with the different functional cases.</p>
<p>Suprisingly, Fig 2 shows stronger selection patterns for enhancers and CTCF compared to coding regions.
If the size is not (well) controlled, it would make sense that these regulatory elements which are more spread in the genome, more often overlap larger and rarer SVs.
The fact that the singleton proportion for SVs in enhancers drops that much when focusing on SVs overlapping one functional class only (CMC unique) would suggest a confounding effect of SV size in the other two panel also.</p>
<p>My best guess is that the test is kind of correct but the figure 2 is biased by the size effect.
The fact that the p-value of the test is smallest for enhancers might be because there more SVs to test than coding SVs for example.</p>
<p>Thinking about this adjustment strategies, I can imagine two problems.
First, it might be difficult if not impossible to match some large SVs.
In particular, is it really possible to find Kbp-long intronic SVs, or even large SVs that don’t overlap any CTCF sites or enhancers?
Second, the few large SVs overlapping none of the functional annotations might be enriched in difficult regions to annotate and study in general, like low-complexity, repeat-rich and centro-telomeric regions.
These regions might well have SV calls, and often frequent.
So there might be a bias if we don’t control for repeat context and how they might correlate with lack of annotation (I think).</p>
<p>To finish on this, I can think of a few alternative approaches.
Why not model the frequency of a SV using a linear model with their size and their overlap with functional classes?
If we want intuitive numbers like AF or singleton proportion, maybe a conservative size theshold would give better results (kind of what <em>CMC unique</em> looks like).
Or maybe show Figure 2 for a couple of size classes.
I would be reassured if SVs shorter than 200 or 500 bp showed the same pattern, for example.</p>
</div>
<div id="references-to-read-1" class="section level3">
<h3>References to read</h3>
<ul>
<li>Fudenberg, G. &amp; Pollard, K. S. Chromatin features constrain structural variation across evolutionary timescales. Proc. Natl Acad. Sci. 116, 2175–2180 (2019).</li>
</ul>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<div id="refs" class="references">
<div id="ref-Chen2020a">
<p>Chen, Lei, Haley J. Abel, Indraniel Das, David E. Larson,..., and Ira M. Hall. 2020. “Association of Structural Variation with Cardiometabolic Traits in Finns.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.12.13.422502">https://doi.org/10.1101/2020.12.13.422502</a>.</p>
</div>
<div id="ref-CommonMindConsortium2020">
<p>CommonMind Consortium, Lide Han, Xuefang Zhao, Mary Lauren Benton,..., and Douglas M. Ruderfer. 2020. “Functional Annotation of Rare Structural Variation in the Human Brain.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-020-16736-1">https://doi.org/10.1038/s41467-020-16736-1</a>.</p>
</div>
<div id="ref-Hiatt2020">
<p>Hiatt, Susan M., James M.J. Lawlor, Lori H. Handley, Ryne C. Ramaker,..., and Gregory M. Cooper. 2020. “Long-Read Genome Sequencing for the Diagnosis of Neurodevelopmental Disorders.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.07.02.185447">https://doi.org/10.1101/2020.07.02.185447</a>.</p>
</div>
</div>
</div>
