---
date: 2022-07-12
title: "SV detection???"
tags: ['literature', 'sv']
slug: litreviews-svdetection
bibliography: [../../library-small.bib]
link-citations: true
draft: true
output: blogdown::html_page
---



<div id="pansvr-pan-genome-augmented-short-read-realignment-for-sensitive-detection-of-structural-variations" class="section level2">
<h2>PanSVR: Pan-Genome Augmented Short Read Realignment for Sensitive Detection of Structural Variations</h2>
<p><a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.731515/full">Li et al. Frontiers in Genetics 2021</a></p>
<p>Pan-genome augmented Structure Variation calling tool with read Re-alignment (PanSVR) is a SV caller method is <strong>more sensitive to SVs</strong>, including in repeat-rich regions or <strong>for insertions</strong>.
Compared to other pangenomic approaches, PanSVR is supposed to be <strong>more robust to the sample SV diverging from the known SV</strong>.
I guess this is true because the pangenome is used only to get longer and more informative seeds that will be used for a more traditional read alignment, clustering, and assembly afterwards.</p>
<p><strong>Methods</strong>:</p>
<ol style="list-style-type: decimal">
<li>The pan-genome SV reference contains the sequence around SV sites and the new sequence (for insertions).
<ol start="2" style="list-style-type: decimal">
<li>SV breakpoints +- 250 bp</li>
<li>Those “anchor” sequences of all SVs are concatenated?</li>
<li>Index with a de Bruijn graph (RdBG-index??). <em>k=22</em></li>
</ol></li>
<li>Chimeric short reads are aligned to this SV reference to make long “match blocks”
<ol start="2" style="list-style-type: decimal">
<li>Keep all read pairs with at least on reads not perfectly aligned. (Really, that’s a lot of reads??)</li>
<li>Realigned using the RdBG index with unique 20-mers seeds (or repetitive seeds at ends of the read).</li>
<li>Seeds are extended within graph unitigs.</li>
</ol></li>
<li>“Match blocks” are used as seeds for a re-alignment to the reference sequence
<ol start="5" style="list-style-type: decimal">
<li>Chaining using sparse dynamic programming</li>
<li>Align the gaps with Smith-Waterman</li>
</ol></li>
<li>Reads are clustered by position in the SV reference</li>
<li>In each cluster, reads are assembled into a consensus sequence
<ol start="3" style="list-style-type: decimal">
<li>Only consider cluster with good quality distribution.</li>
<li>Fragmented assembly with a modified version of Manta.</li>
</ol></li>
<li>Consensus sequence is aligned back to the SV reference with NW, and the</li>
</ol>
<p><strong>They use the pangenome just to make longer seeds.</strong>
It surprises me because the breakpoint information is not really used.
They could have just indexed the regions surrounding each known SV (and the inserted sequences) and selected the alignment seeds based on how uniquely they map, and get exactly the same.
Well, I guess that’s what they did.
The other pangenomic methods I know use the adjacency information in the graph (GraphTyper, vg, BayesTyper, Paragraph).
It feels like a lost oppotunity to not use the fact that reads <strong>span</strong> the known breakpoints.
On the other hand, it makes their method more robust when the SVs are different.</p>
<p><strong>Benchmark</strong>:</p>
<ul>
<li>Compare to Manta and Delly</li>
<li>Truthset: minimap2+SVIM-asm to call SVs from phased HGSVC assembly.</li>
<li>SV comparison to truthset using SURVIVOR</li>
<li>Higher F1 scores for both insertions and deletions</li>
<li>Overall better at calling long insertions (~20x more 500+ bp insertions than Manta).</li>
<li>Alu insertions are still challenging</li>
<li>No comparison to other pangenomic SV genotypers (cited in Discussion) :(</li>
</ul>
<p>Multiple typos/inconsistencies:
- “Both <em>two the</em> values can be used”
- Figure 2C legend says “NW” algorithm but the figure and text talk about SW.
- “the improved alignments are sort by their positions”
- “There are two haplotypes <em>of</em> for each of the samples”
- “within the inserted <em>strings</em> of the sequenced sample”</p>
</div>
