<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Family-based designs for gene-association studies &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../../../../css/poole.css">
  <link rel="stylesheet" href="../../../../css/syntax.css">
  <link rel="stylesheet" href="../../../../css/lanyon.css">
  <link rel="stylesheet" href="../../../../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../../../../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../../assets/R-icon.png">
  <link rel="shortcut icon" href="../../../../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../../../../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../../../../">Home</a>
    <a class="sidebar-nav-item " href="../../../../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../../../../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../../../../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../../../../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../../../../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="post">
  <h1 class="post-title">Family-based designs for gene-association studies</h1>
  <span class="post-date">Apr 9 2017
  
  
  
  
  
  <a href='../../../../tags/literature/' class='label'>literature</a>
  
  
  
  </span>
  


<p>Different type of family information can be used.
<strong>Trio designs</strong> (affected offspring plus parents) seems to have the most power but having both affected and unaffected siblings is useful too.</p>
<p>A positive result in a family-based test means there is both <strong>linkage</strong> and <strong>association</strong>.
It avoids problems with population stratification or admixture.</p>
<div id="transmission-disequilibrium-test-tdt" class="section level2">
<h2>Transmission Disequilibrium Test (TDT)</h2>
<p>The TDT compares the number of observed allele transmitted to the affected offspring with the expected number following Mendelian transmission.
It is completely non-parametric in the sense that it doesn’t require specifying a disease model.</p>
<p>In its simple form (complete trios), the TDT is more or less a Binomial test on the number of transmitted allele according to the number of heterozygous parents.
It cannot deal with missing parents or more general pedigrees.</p>
</div>
<div id="family-based-association-tests-fbats" class="section level2">
<h2>Family-Based Association Tests (FBATs)</h2>
<p>An extension of the TDT.</p>
<p>The FBAT statistic represents how the observed genotype diverges from the expected genotype in individuals with phenotype:</p>
<p><span class="math display">\[ U = \sum T * (X-E(X|P)) \]</span></p>
<p>with the sum being over all offspring and</p>
<ul>
<li><span class="math display">\[T=Y-\mu\]</span> the centered phenotype</li>
<li><span class="math display">\[X\]</span> the genotype</li>
<li>\(E(X|P)\) the expected genotype.</li>
</ul>
<p>The FBAT statistics is <span class="math display">\[\frac{U^2}{Var(U)}\]</span> and follows a <span class="math display">\[\chi^2\]</span> distribution with one degree of freedom (if at least 10 families).</p>
<ul>
<li>The genotype <span class="math display">\[X\]</span> can be coded to reflect the mode of inheritance.</li>
<li>Both \(E(X|P)\) and <span class="math display">\[Var(U)\]</span> are computed under the null hypothesis from the parents/siblings genotypes.</li>
</ul>
<p><em>TODO empirical estimate of the variance from Lake et al 2000.</em></p>
</div>
<div id="filtering-variants" class="section level2">
<h2>Filtering variants</h2>
<p>Another approach is to focus on (homozygous) segregating variants and annotate them to assess their functional impact (frequency, location).</p>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/16619052"><em>Family-based designs in the age of large-scale gene-association studies.</em> by Laird &amp; Lange (Nature Reviews Genetics 2006)</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11793685"><em>Family-Based Tests of Association in the Presence of Linkage.</em> by Lake et al. (AJHG 200)</a></li>
</ul>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../../../../assets/highlight.min.js"></script>
    <script src="../../../../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

