<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  
  <meta name='robots' content='noindex,follow' />

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Structural Variation &middot; My Reviews
    
  </title>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113785126-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    
    gtag('config', "UA-113785126-1");
  </script>
  
  
  <link rel="stylesheet" href="../css/poole.css">
  <link rel="stylesheet" href="../css/syntax.css">
  <link rel="stylesheet" href="../css/lanyon.css">
  <link rel="stylesheet" href="../css/github.min.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
  <link rel="stylesheet" href="../fa/css/font-awesome.min.css">
  
  
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../assets/R-icon.png">
  <link rel="shortcut icon" href="../assets/R-icon.png">

  
  <link rel="alternate" type="application/rss+xml" title="RSS" href="../atom.xml">
</head>

  
  <body class="theme-base-0d">

    
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">


<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>A place to review papers I read. Views are my own.</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item " href="../">Home</a>
    <a class="sidebar-nav-item " href="../post">Posts</a>

    
    
      
        <a class="sidebar-nav-item " href="../genome-assembly/">Genome Assembly</a>
      
    
      
        <a class="sidebar-nav-item " href="../genome-graphs/">Genome Graphs</a>
      
    
      
        <a class="sidebar-nav-item  active " href="../structural-variation/">Structural Variation</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-impact/">Structural Variation Impact</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-mechanisms-of-formation/">Structural Variation Mechanisms of Formation</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-methods/">Structural Variation Methods</a>
      
    
      
        <a class="sidebar-nav-item " href="../structural-variation-and-repeats/">Structural Variation and Repeats</a>
      
    
      
        <a class="sidebar-nav-item " href="../whole-genome-sequencing/">Whole-Genome Sequencing</a>
      
    
      
        <a class="sidebar-nav-item " href="../glossary/">Glossary</a>
      
    
      
        <a class="sidebar-nav-item " href="../about/">About</a>
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    
      
    

    
    
  </nav>

  <div class="sidebar-item">
  </div>
  
  <div class="sidebar-item">
    <div><a href="../index.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS</a></div>
    <div><a href="https://twitter.com/JMonlong"><i class="fa fa-twitter" aria-hidden="true"></i> @JMonlong</a></div>
    <p>&copy; 2020 Jean Monlong. All rights reserved.</p>
  </div>
</div>


    
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="../" title="Home">My Reviews</a>
            <small>Views are my own</small>
          </h3>
        </div>
      </div>

      <div class="container content">


<div class="page">
  <h1 class="page-title">Structural Variation</h1>
  

<div id="TOC">
<ul>
<li><a href="#introduction---importance-of-structural-variation">Introduction - Importance of Structural Variation</a></li>
<li><a href="#different-sv-types">Different SV types</a><ul>
<li><a href="#copy-number-variation">Copy Number Variation</a><ul>
<li><a href="#deletions-and-insertions">Deletions and insertions</a></li>
<li><a href="#tandem-repeats">Tandem repeats</a></li>
</ul></li>
<li><a href="#inversions">Inversions</a></li>
<li><a href="#transchromosomal-svs">Transchromosomal SVs</a></li>
<li><a href="#complex-svs">Complex SVs</a><ul>
<li><a href="#somatic-svs">Somatic SVs</a></li>
<li><a href="#germline-svs">Germline SVs</a></li>
</ul></li>
<li><a href="#sv-hotspots">SV hotspots</a></li>
</ul></li>
<li><a href="#bibliography">Bibliography</a></li>
</ul>
</div>

<div id="introduction---importance-of-structural-variation" class="section level1">
<h1>Introduction - Importance of Structural Variation</h1>
<p>With our current knowledge about genomic variation, <em>the</em> human genome has ceased to exist as a defined entity and is instead becoming a fluid continuum <span class="citation">(Dear <a href="#ref-Dear2009" role="doc-biblioref">2009</a>)</span>.</p>
<p><strong>5% of the genome</strong> is known to display CNVs, i.e. more than SNVs.
CNVs represent around 50 Mbp of difference per human genome <span class="citation">(Baker <a href="#ref-Baker2012" role="doc-biblioref">2012</a>)</span>.
The amount of the genome that have been seen hosting CNVs varies across studies.
CNVs spanned 136.6 Mb/4.8% (273 Mb/9.5%) of the genome in a recent stringent (inclusive) CNV map constructed from DGV <span class="citation">(Zarrei et al. <a href="#ref-Zarrei2015" role="doc-biblioref">2015</a>)</span>.
CNVs spanned 112.7 Mb/3.7% of the genome in an aCGH survey across 41 samples <span class="citation">(Conrad et al. <a href="#ref-Conrad2010" role="doc-biblioref">2010</a>)</span>.
On average, 24 Mb/0.78% of the genome were CNV between two samples.
<span class="citation">Pang et al. (<a href="#ref-Pang2010" role="doc-biblioref">2010</a>)</span> compared directly the assemblies of the human genome and found <strong>48.8 Mb</strong> (1.2%) of the genome to be affected by SVs.
More recentlty, the 1000 Genomes Project Consortium observed on average <strong>18.4 Mb of SV</strong> per diploid genome <span class="citation">(Sudmant et al. <a href="#ref-Sudmant2015a" role="doc-biblioref">2015</a>)</span>.
mCNVs (11.3 Mb) and large deletions (5.6 Mb) contribute the most to this number.
Although SVs represent less than <em>0.1%</em> of the variant sites in a typical genome, they affect more bases <span class="citation">(Auton et al. <a href="#ref-Auton2015" role="doc-biblioref">2015</a>)</span>.</p>
<p>In the 1000 Genome Project, <strong>a typical genome contains 2,100-2,500 SVs</strong>: a thousands large deletions, 160 CNV, 915 Alu insertions, 128 L1 insertions, 51 SVA insertions, 4 nuclear mitochondrial DNA variants, and 10 inversions <span class="citation">(Auton et al. <a href="#ref-Auton2015" role="doc-biblioref">2015</a>)</span>.</p>
<p>Similar to SNPs, the <strong>majority of SVs in the population are rare</strong> (65% with VAF <em>&lt;0.2%</em>) <span class="citation">(Auton et al. <a href="#ref-Auton2015" role="doc-biblioref">2015</a>; Sudmant et al. <a href="#ref-Sudmant2015a" role="doc-biblioref">2015</a>)</span>.
Nevretheless, the majority of variants in one genome are common.</p>
<table>
<thead>
<tr class="header">
<th>Study</th>
<th>Data</th>
<th>SV</th>
<th>Mb</th>
<th>SV/sample</th>
<th>Mb/sample</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="citation">Conrad et al. (<a href="#ref-Conrad2010" role="doc-biblioref">2010</a>)</span></td>
<td>41 aCGH</td>
<td>8,600</td>
<td>112.7</td>
<td>1,100</td>
<td>24</td>
</tr>
<tr class="even">
<td><span class="citation">Pang et al. (<a href="#ref-Pang2010" role="doc-biblioref">2010</a>)</span></td>
<td>2 assemblies</td>
<td>?</td>
<td>48.8</td>
<td>?</td>
<td>48.8</td>
</tr>
<tr class="odd">
<td><span class="citation">Altshuler et al. (<a href="#ref-Altshuler2010" role="doc-biblioref">2010</a>)</span></td>
<td>?</td>
<td>?</td>
<td>?</td>
<td>?</td>
<td>?</td>
</tr>
<tr class="even">
<td><span class="citation">Mills et al. (<a href="#ref-Mills2011" role="doc-biblioref">2011</a>)</span></td>
<td>185 WGS</td>
<td>28,000</td>
<td><em>?</em></td>
<td><em>?</em></td>
<td><em>?</em></td>
</tr>
<tr class="odd">
<td><span class="citation">Altshuler et al. (<a href="#ref-Altshuler2012" role="doc-biblioref">2012</a>)</span></td>
<td>?</td>
<td>?</td>
<td>?</td>
<td>?</td>
<td>?</td>
</tr>
<tr class="even">
<td><span class="citation">Francioli et al. (<a href="#ref-Francioli2014" role="doc-biblioref">2014</a>)</span></td>
<td>750 WGS</td>
<td>27,500</td>
<td><em>?</em></td>
<td><em>?</em></td>
<td><em>?</em></td>
</tr>
<tr class="odd">
<td><span class="citation">Zarrei et al. (<a href="#ref-Zarrei2015" role="doc-biblioref">2015</a>)</span></td>
<td>DGV</td>
<td>12,000</td>
<td>136.6</td>
<td>-</td>
<td>-</td>
</tr>
<tr class="even">
<td><span class="citation">Zarrei et al. (<a href="#ref-Zarrei2015" role="doc-biblioref">2015</a>)</span> inclusive</td>
<td>DGV</td>
<td>24,000</td>
<td>273</td>
<td>-</td>
<td>-</td>
</tr>
<tr class="odd">
<td><span class="citation">Handsaker et al. (<a href="#ref-Handsaker2015" role="doc-biblioref">2015</a>)</span></td>
<td>849 WGS</td>
<td>8,659</td>
<td><em>?</em></td>
<td><em>?</em></td>
<td><em>?</em></td>
</tr>
<tr class="even">
<td><span class="citation">Auton et al. (<a href="#ref-Auton2015" role="doc-biblioref">2015</a>)</span> Sudmant2015a</td>
<td>2,504 WGS</td>
<td>69,000</td>
<td>?</td>
<td>2,300</td>
<td>18.4</td>
</tr>
</tbody>
</table>
<p>SV estimates. <em>italic</em>: computed manually from their results.</p>
</div>
<div id="different-sv-types" class="section level1">
<h1>Different SV types</h1>
<ul>
<li>What are the different types of SV ?</li>
<li>What the expected/known size distribution of the SV in the genome ?</li>
<li>What is the expected/known location of SV in the genome ?</li>
<li>Is it different for different SV types ?</li>
<li>Does it look like we are missing something ?</li>
</ul>
<p>The different SV type are commonly <strong>defined with respect to the reference genome</strong>.
Another definition takes into account the ancestral state of the genome.
An insertion compared to the reference genome could also be a deletion in the reference genome.
While 21% of the deletions in <span class="citation">Mills et al. (<a href="#ref-Mills2011" role="doc-biblioref">2011</a>)</span> are actually insertions in the reference genome, 60% were clear deletions relative to the ancestral state.</p>
<p>Simple SV types includes <strong>insertions, deletions, duplications, inversion and translocations</strong>.
Deletions and duplications are also called <strong>Copy Number Variation</strong> (CNV).
Complex SVs look like a mixture simple SVs.</p>
<div id="copy-number-variation" class="section level2">
<h2>Copy Number Variation</h2>
<p>A comprehensive study of the selection bias on Human CNVs was performed by <span class="citation">Nguyen, Webber, and Ponting (<a href="#ref-Nguyen2006" role="doc-biblioref">2006</a>)</span>.
They started from the null hypothesis that CNVs arise uniformly in the genome and are selectively neutral.
They found that <strong>CNVs occur more frequently near telomeres and centromeres</strong>, are <strong>enriched in protein-coding genes and tandem repeats</strong> but <strong>not in high GC content regions</strong>.
Moreover CNV genes display a higher synonymous and non-synonymous substitution rate and are <strong>under-represented in Mendelian disease genes</strong>.
In fact human <strong>CNV genes</strong> encodes a disproportionately large number of <strong>secreted, olfactory and immunity proteins</strong>.
To be precise, the enrichment in tandem repeat (e.g. micro-satellites) is specific to the CNVs within 2 Mb of the centromeres and telomeres.
They also noticed no significant enrichment of RNA genes, interspersed repeats, CpG islands or GC content.
They concluded from this that the observed patterns were not due to sequence context.</p>
<p><span class="citation">Zarrei et al. (<a href="#ref-Zarrei2015" role="doc-biblioref">2015</a>)</span> found <em>immune response, olfactory receptors, starch and sucrose metabolism, vesicle and Golgi apparatus pathways</em> to be <strong>enriched in CNVs</strong>.
Conversely, <strong>copy number stable</strong> regions were enriched in <em>protein phosphorylation, signal transduction, protein degradation, transcript regulation, intra-cellular transport, cell differentiation</em> genes.
They also found lincRNA and promoters enriched while enhancer were under-represented.</p>
<div id="deletions-and-insertions" class="section level3">
<h3>Deletions and insertions</h3>
<p>Deletions and insertions are the most frequent and best characterized type of SVs in the human genome.
Data suggests that the <strong>number of insertions/deletions increases exponentially with decreasing event size</strong> <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>As expected, many deletions display <strong>signs of negative selection</strong>.
Deletion are significantly under-represented in coding genes, pointing at a deleterious effect and purifying selection <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>; Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.
In <span class="citation">Mills et al. (<a href="#ref-Mills2011" role="doc-biblioref">2011</a>)</span>, high-frequency deletions were under-represented in protein-coding sequence.
Exceptions exist, e.g. molecular and environmental interaction genes.
Rare deletions seem to be present more often than SNPs, again because of a potentially stronger purifying selection <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>There are more deletions detected than duplication <span class="citation">(Mills et al. <a href="#ref-Mills2011" role="doc-biblioref">2011</a>; Fu et al. <a href="#ref-Fu2010" role="doc-biblioref">2010</a>)</span>.
Still it is not clear how much this trend is affected by detection power.</p>
</div>
<div id="tandem-repeats" class="section level3">
<h3>Tandem repeats</h3>
<p>Tandem repeats are composed of <strong>variable numbers of serially repeated cassettes</strong> <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.
Some repeats units can be several Kbp long.
Several <strong>genes</strong> with wide CNV are located <strong>within tandemly repeated arrays</strong>, e.g. AMY1A, GSTM1, \(-\)/\(-\) defensins.
Micro-satellites expansion has also been documented within such arrays.</p>
<p>Many of the tandem repeat arrays might be highly variable in copy number between individuals <span class="citation">(Warburton et al. <a href="#ref-Warburton2008" role="doc-biblioref">2008</a>)</span>.</p>
<p><strong>Tandem repeats in centromere</strong> have <strong>essential functions</strong> for establishing <strong>heterochromatin formation</strong> and <strong>chromosome segregation</strong> <span class="citation">(Warburton et al. <a href="#ref-Warburton2008" role="doc-biblioref">2008</a>)</span>.
Their function also seem to involve RNA interference-mediated chromatin modifications.</p>
<p>Some large tandem repeat arrays are locus specific.
The largest found by <span class="citation">Warburton et al. (<a href="#ref-Warburton2008" role="doc-biblioref">2008</a>)</span> was 246 kb large, and <em>“reflects the heroic efforts made in sequencing the Y chromosome”</em>.</p>
<p>Microsatellites and variable number of tandem repeats can be highly polymorphic and <strong>used as genetic markers</strong> <span class="citation">(Warburton et al. <a href="#ref-Warburton2008" role="doc-biblioref">2008</a>)</span>.</p>
<p><strong>Short tandem repeats</strong> (STR) or microsatellites are repetitive elements of <strong>2-6 nucleotides</strong> and consist of approximately a <em>quarter million loci</em> in the human genome <span class="citation">(Gymrek et al. <a href="#ref-Gymrek2012" role="doc-biblioref">2012</a>)</span>.
They create secondary structure <strong>prone to replication slippage</strong> and results in <strong>high variability</strong>.
STRs are involved in different <strong>genetic disorders</strong> and used in <strong>forensics</strong>, genetic anthropology <span class="citation">(Gymrek et al. <a href="#ref-Gymrek2012" role="doc-biblioref">2012</a>)</span>.
They have also been used to trace cell lineages in cancer samples.</p>
<p>The <strong>rate of STR polymorphism is highly correlated with the repeat unit length</strong>, as well as with the length in the reference genome <span class="citation">(Gymrek et al. <a href="#ref-Gymrek2012" role="doc-biblioref">2012</a>)</span>.</p>
</div>
</div>
<div id="inversions" class="section level2">
<h2>Inversions</h2>
<p>Inversions might be one of the <em>least characterized SV type</em>.
Again, detected inversions are associated with the presence of segmental duplications at their flanking regions.
Moreover, <strong>insertion/deletion at their breakpoint location are present in many inversions</strong>, hence this SV type is often not fully unbalanced.</p>
<p>Inversions are <strong>enriched in chromosome X</strong>, maybe due to its enrichment in large inverted repeat structures <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>Although inversions are mainly balanced, they can induce phenotypic effect by disrupting coding regions spanning their breakpoints, or genes dependent on their relative positions.
Eventually it is also recognized that <strong>inversions can confer a predisposition for further chromosomal rearrangements</strong> <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>; Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.
Such effects have been observed with deletions but also translocations.</p>
<p>Inversion have an interesting effect on a region haplotype conservation.
Indeed, local recombination seems suppressed after an inversion.
This phenomenon is thought to derive from a failure to recover from lethal recombination in inverted regions, rather than an actual reduction in recombination frequency.
Suppressed local recombination could actually be used to detect inversions by identifying and genotyping haplotypes with abnormal segregation.</p>
</div>
<div id="transchromosomal-svs" class="section level2">
<h2>Transchromosomal SVs</h2>
<p>Translocations have been <strong>challenging</strong> to study.
Even with recent sequencing, potential translocation are often filtered to limit false-positive rate <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>Some fixed transchromosomal SVs have been identified but they are likely more often deleterious, affecting meiotic segregation.
For example, olfactory receptor are located in nearly half of all human sub-telomeres.
Other genes, e.g. zinc-finger and immunoglobulin heavy-chain gene families, are also located in multiple human sub-telomeres <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>Common translocations usually involve p arms of acrocentric chromosomes.
For example, 15p pericentromeric satellite III into chromosome 14 short arm.</p>
</div>
<div id="complex-svs" class="section level2">
<h2>Complex SVs</h2>
<p>Complex SVs are observed <strong>in tumors</strong> as well as in the normal population.</p>
<div id="somatic-svs" class="section level3">
<h3>Somatic SVs</h3>
<p>In <span class="citation">Malhotra et al. (<a href="#ref-Malhotra2013" role="doc-biblioref">2013</a>)</span>, complex SVs are defined as SVs with discordant read clusters with <strong>3 or more breakpoints</strong> (distant from at least 1 Mbp).
Moreover, they define <strong>chromothripsis</strong> as a <em>single mutational event with 10 or more breakpoints</em>.
They found complex SVs to be <strong>common in tumors</strong>, with 75% of them exhibiting at least one.
The breakpoints of these complex SVs were <strong>not enriched in known SV hotspots</strong>, segmental duplication or repetitive elements.
The incidence of chromothripsis differed across tumor types (e.g. over-represented in glioblastoma) and apparently not correlated by the number of breakpoints.
They describe chromothripsis as a more extreme manifestation of common events rather than a different mechanism.
NHEJ after DNA double-strand breaks seems to be the predominant mechanism of complex SV formation.</p>
<p>Complex SVs can originate from a single mutational event or the accumulation of mutations in a region.
A <strong>few copy number states</strong> might suggest <strong>one single mutational event</strong> <span class="citation">(Malhotra et al. <a href="#ref-Malhotra2013" role="doc-biblioref">2013</a>)</span>.
74% of single-mutational complex SVs showed copy number alteration, suggesting that many events might be <em>falsely detected as independent events</em> in array or RD-based approaches.</p>
<p>Complex SVs in tumors seemed to be slightly more frequent than simple SVs <span class="citation">(Malhotra et al. <a href="#ref-Malhotra2013" role="doc-biblioref">2013</a>)</span>.
They might appear earlier or be under stronger selection.</p>
</div>
<div id="germline-svs" class="section level3">
<h3>Germline SVs</h3>
<p>Chromosome Y or region 5q13.3 with the Spinal Muscular Atrophy locus, were <strong>very difficult to assemble</strong> and are expected to segregate divergent alleles in the population.
SMA region, 1-2 Mbp in size, has two known variants that diverge by multiple rearrangements and small insertion/deletions.
<em>Only one male donor was used to assemble Y chromosome in the reference genome</em> because of its variability and high repeated/palindromic content.</p>
<p><span class="citation">Collins et al. (<a href="#ref-Collins2017" role="doc-biblioref">2017</a>)</span> describe the distribution of complex SVs and chromothripsis in individuals with ASD.
They used long-insert reads (105X, ~3.5 kb IS) and linked-reads on 689 ASD patients.
They found that complex SVs are larger and rarer and usually involve inversion (84.4%)
A morbid genome has on average 14 complex SVs.
They also observed chromoanagenesis where chromosome is pulverized and reorfanized.
Chromoanagenesis encompasses chromothripsis, chromoplexy and chromoanasynthesis.
In the morbid genome they found exmaples of balanced germline chromothripsis and somatic chromoanasynthesis.</p>
<p>SVs found in <span class="citation">Collins et al. (<a href="#ref-Collins2017" role="doc-biblioref">2017</a>)</span> were compared to six recent WGS studies (1KGP, Chaisson-PacBio, Brand, Pendleton-Pacbio, Handsaker-1KGP, GoNL) DGV and InvFEST.
38.1% of the SVs had not been reported previously.</p>
</div>
</div>
<div id="sv-hotspots" class="section level2">
<h2>SV hotspots</h2>
<p>CNV hotspots cluster in pericentromeric and subtelomeric regions <span class="citation">(Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.
However this represents only ~1/3 of all hotspots <span class="citation">(Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.</p>
<p>CNVs are <strong>enriched in segmental duplication, exons and mobile elements</strong> <span class="citation">(Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.
The enrichment in exons is surprising to me; in <span class="citation">Cooper, Nickerson, and Eichler (<a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span> they computed it in term of density (proportion of affected nucleotide) that might be driven by a few gene-rich regions.
CNVs are also more likely to occur near ancestrally duplicated sequence (<strong>duplication shadowing</strong>) <span class="citation">(Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007" role="doc-biblioref">2007</a>)</span>.</p>
</div>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Altshuler2010">
<p>Altshuler, David L., Richard M. Durbin, Gonçalo R. Abecasis, David R. Bentley,..., and Jane L. Peterson. 2010. “A Map of Human Genome Variation from Population-Scale Sequencing.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature09534">https://doi.org/10.1038/nature09534</a>.</p>
</div>
<div id="ref-Altshuler2012">
<p>Altshuler, David M., Richard M. Durbin, Gonçalo R. Abecasis, David R. Bentley,..., and Phil Lacroute. 2012. “An Integrated Map of Genetic Variation from 1,092 Human Genomes.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature11632">https://doi.org/10.1038/nature11632</a>.</p>
</div>
<div id="ref-Auton2015">
<p>Auton, Adam, Gonçalo R. Abecasis, David M. Altshuler, Richard M. Durbin,..., and Gonçalo R. Abecasis. 2015. “A Global Reference for Human Genetic Variation.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.</p>
</div>
<div id="ref-Baker2012">
<p>Baker, Monya. 2012. “Structural Variation: The Genome’s Hidden Architecture.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1858">https://doi.org/10.1038/nmeth.1858</a>.</p>
</div>
<div id="ref-Collins2017">
<p>Collins, Ryan L, Harrison Brand, Claire E Redin, Carrie Hanscom,..., and Michael E Talkowski. 2017. “Defining the Diverse Spectrum of Inversions, Complex Structural Variation, and Chromothripsis in the Morbid Human Genome.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/s13059-017-1158-6">https://doi.org/10.1186/s13059-017-1158-6</a>.</p>
</div>
<div id="ref-Conrad2010">
<p>Conrad, Donald F, Dalila Pinto, Richard Redon, Lars Feuk,..., and Matthew E Hurles. 2010. “Origins and Functional Impact of Copy Number Variation in the Human Genome.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature08516">https://doi.org/10.1038/nature08516</a>.</p>
</div>
<div id="ref-Cooper2007">
<p>Cooper, Gregory M., Deborah A. Nickerson, and Evan E. Eichler. 2007. “Mutational and Selective Effects on Copy-Number Variants in the Human Genome.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng2054">https://doi.org/10.1038/ng2054</a>.</p>
</div>
<div id="ref-Dear2009">
<p>Dear, Paul H. 2009. “Copy-Number Variation: The End of the Human Genome?” <em>Trends in Biotechnology</em>. <a href="https://doi.org/10.1016/j.tibtech.2009.05.003">https://doi.org/10.1016/j.tibtech.2009.05.003</a>.</p>
</div>
<div id="ref-Francioli2014">
<p>Francioli, Laurent C, Androniki Menelaou, Sara L Pulit, Freerk van Dijk,..., and Cisca Wijmenga. 2014. “Whole-Genome Sequence Variation, Population Structure and Demographic History of the Dutch Population.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3021">https://doi.org/10.1038/ng.3021</a>.</p>
</div>
<div id="ref-Fu2010">
<p>Fu, Wenqing, Feng Zhang, Yi Wang, Xun Gu, and Li Jin. 2010. “Identification of Copy Number Variation Hotspots in Human Populations.” <em>American Journal of Human Genetics</em>. <a href="https://doi.org/10.1016/j.ajhg.2010.09.006">https://doi.org/10.1016/j.ajhg.2010.09.006</a>.</p>
</div>
<div id="ref-Gymrek2012">
<p>Gymrek, Melissa, David Golan, Saharon Rosset, and Yaniv Erlich. 2012. “lobSTR: A Short Tandem Repeat Profiler for Personal Genomes.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.135780.111">https://doi.org/10.1101/gr.135780.111</a>.</p>
</div>
<div id="ref-Handsaker2015">
<p>Handsaker, Robert E, Vanessa Van Doren, Jennifer R Berman, Giulio Genovese,..., and Steven a McCarroll. 2015. “Large Multiallelic Copy Number Variations in Humans.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3200">https://doi.org/10.1038/ng.3200</a>.</p>
</div>
<div id="ref-Malhotra2013">
<p>Malhotra, Ankit, Michael Lindberg, Gregory G Faust, Mitchell L Leibowitz,..., and Ira M Hall. 2013. “Breakpoint Profiling of 64 Cancer Genomes Reveals Numerous Complex Rearrangements Spawned by Homology-Independent Mechanisms.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.143677.112">https://doi.org/10.1101/gr.143677.112</a>.</p>
</div>
<div id="ref-Mills2011">
<p>Mills, Ryan E., Klaudia Walter, Chip Stewart, Robert E. Handsaker,..., and Jan O. Korbel. 2011. “Mapping Copy Number Variation by Population-Scale Genome Sequencing.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature09708">https://doi.org/10.1038/nature09708</a>.</p>
</div>
<div id="ref-Nguyen2006">
<p>Nguyen, Duc-Quang, Caleb Webber, and Chris P Ponting. 2006. “Bias of Selection on Human Copy-Number Variants.” <em>PLoS Genetics</em>. <a href="https://doi.org/10.1371/journal.pgen.0020020">https://doi.org/10.1371/journal.pgen.0020020</a>.</p>
</div>
<div id="ref-Pang2010">
<p>Pang, Andy W, Jeffrey R MacDonald, Dalila Pinto, John Wei,..., and Stephen W Scherer. 2010. “Towards a Comprehensive Structural Variation Map of an Individual Human Genome.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2010-11-5-r52">https://doi.org/10.1186/gb-2010-11-5-r52</a>.</p>
</div>
<div id="ref-Sharp2006">
<p>Sharp, Andrew J., Ze Cheng, and Evan E. Eichler. 2006. “Structural Variation of the Human Genome.” <em>Annual Review of Genomics and Human Genetics</em>. <a href="https://doi.org/10.1146/annurev.genom.7.080505.115618">https://doi.org/10.1146/annurev.genom.7.080505.115618</a>.</p>
</div>
<div id="ref-Sudmant2015a">
<p>Sudmant, P. H., S. Mallick, B. J. Nelson, F. Hormozdiari,..., and E. E. Eichler. 2015. “Global Diversity, Population Stratification, and Selection of Human Copy Number Variation.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aab3761">https://doi.org/10.1126/science.aab3761</a>.</p>
</div>
<div id="ref-Warburton2008">
<p>Warburton, Peter E, Dan Hasson, Flavia Guillem, Chloe Lescale,..., and Gyorgy Abrusan. 2008. “Analysis of the Largest Tandemly Repeated DNA Families in the Human Genome.” <em>BMC Genomics</em>. <a href="https://doi.org/10.1186/1471-2164-9-533">https://doi.org/10.1186/1471-2164-9-533</a>.</p>
</div>
<div id="ref-Zarrei2015">
<p>Zarrei, Mehdi, Jeffrey R. Macdonald, Daniele Merico, and Stephen W. Scherer. 2015. “A Copy Number Variation Map of the Human Genome.” <em>Nature Reviews Genetics</em>. <a href="https://doi.org/10.1038/nrg3871">https://doi.org/10.1038/nrg3871</a>.</p>
</div>
</div>
</div>

</div>

      </div>
    </div>

    
    

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="../assets/highlight.min.js"></script>
    <script src="../assets/r.min.js"></script>
    
    <script>
      hljs.configure({languages: ['r']});
      hljs.initHighlightingOnLoad();
    </script>

    <script src="//yihui.name/js/math-code.js"></script>
    <script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
    </script>
    
  </body>
</html>

