---
sidebar: true
title: Diseases
bibliography: [../../library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---

# Hemochromatosis


[Hemochromatosis](https://en.wikipedia.org/wiki/Iron_overload) is caused by an excess of iron.
There are five types of hemochromatosis, two of which are juvenile forms.
The classical type, [hereditary hemochromatosis type 1](https://en.wikipedia.org/wiki/Hereditary_haemochromatosis), is by far the most frequent.

Multiple organs are affected, notably liver and heart.
Symptoms include:

- liver disease, e.g. cirrhosis or liver cancer
- heart failure
- chronic fatigue
- arthritis

The disease onset is later in women, usually after menopause.
Men aged 40-50 and more should show symptoms.

The treatment is usually blood draws ([phlebotomy](https://en.wikipedia.org/wiki/Phlebotomy)).

### Genetics

The main risk factor is a [C282Y mutation](https://www.ncbi.nlm.nih.gov/snp/rs1800562) in the HFE gene which is frequent but has an incomplete penetrance.
This mutation is observed in about 5% of European populations, and to a lower extent in south Europe and the US.
It was spread through Viking and then Normans migrations (and the [celtic haplotype HLA-A3/B7](https://bmcmedgenet.biomedcentral.com/articles/10.1186/1471-2350-5-25)).
Carriers of the homozygous risk alleles (3-8 out of 1000) have about 10-20% chance of developing the disease.

A second HFE mutation is sometimes H63D, which has 17% allele frequency.

HFE is the gene for the hepcidin protein, which is involved in iron absorption regulation by regulating the concentration of ferroportin, the only iron exporter.
Hepcidin also controls the release of iron by macrophages or hepatocytes.

Heritability of hemochromatosis is unknown.
Among HFE-hom, transferrin levels and serum ferritin levels had an estimated heritability of ~0.2 ([McLaren et al. 2008](https://doi.org/10.1182/blood.V112.11.115.115), [McLaren et al. 2010](https://pubmed.ncbi.nlm.nih.gov/20095037/)).
While the chances of developing symptoms is ~20% in general, it increases to ~50% if another member of the family suffers from hemochromatosis.

De Tayrac et al. ([2015](https://pubmed.ncbi.nlm.nih.gov/25457201/)) tested 500K SNPs across 474 unrelated HFE-homs and found a SNP in intron 11 of TF (transferrin gene) associated with levels of serum iron (serum transferrin and serum iron), which correlated with severity of the disease (iron removed by blood draws, clinical stage).
The association was replicated in 748 C282Y-homozygotes.

Milet et al. ([2007](https://pubmed.ncbi.nlm.nih.gov/17847004/)) tested just 10 genes (75 SNPs) across 592 C282Y-homozygotes, and found an association between a SNP near BMP2 and pretherapeutic serum ferritin level (marker penetrance).
They also detected a potential interaction between the BMP2 SNP and a SNP in HJV.
Overall, they concluded that the BMP pathway might be a good candidate when searching for modifier genes.

### Iron regulation

- Enterocyte: intestinal absorptive cell
- Hepatocyte: liver cell
- Macrophages can contain iron too after recycling erythrocytes.
- Erythrocytes: red blood cells.
- Erythropoiesis is the process of making red blood cells, which uses iron.
- Duedenum: first section of the small intestine, where most iron absorption occurs.
- Ferroportin exports iron absorbed by enterocytes, into blood stream. Regulated by hepcidin.
- Hepcidin regulates iron homeostasis. Inhibits iron transport by binding to ferroportin.
- Transferrin: protein that binds to iron to help its transport to tissues. Transferrin saturation is a symptom of hemochromatosis.
- HFE: gene encoding a transmembrane protein that is thought to down-regulate the absorption of iron from transferrin.
- Ferritin: intracellular protein that stores iron.

### Next

- [The global prevalence of HFE and non-HFE hemochromatosis estimated from analysis of next-generation sequencing data](https://www.nature.com/articles/gim2015140)
- [Novel loci affecting iron homeostasis and their effects in individuals at risk for hemochromatosis](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4215164/)
- [HFE in OMIM](https://www.omim.org/entry/613609)

# Inflammatory bowel disease

Inflammatory bowel disease (IBD) is characterized by inflammation of the colon and small intestine.
The two main forms are Crohn's disease (CD) and ulcerative colitis (UC).
The unregulated inflammation is thought to be caused by a combination of environmental, genetic and bacterial factors interacting with the immune system.
Most IDB types are classified as an autoimmune disease because thought to be caused by a disfunctional immune response.

In CD, the inflammation tend to be spread in local clusters (skip lesions) and to extend deep into the tissues.
The inflammation of UC usually only affects the outer (mucosal) layer and starts in the rectal/sigmoid area (proctitis) before spreading.

## Genetics

Many associated loci in IDB, are associated to both CD and UC.
For example, 110 out of 163 loci associated with IBD conferred risk to both CD and UC (review @mcgovern2015g).
In many cases, the effect is stronger in CD than UC, for example the Arg381Gln protective allele in IL23R (review @mcgovern2015g).
Associated loci are sometimes also associated with other phenotypes like primary sclerosing cholangitis, psoriasis, ankylosing spondylitis, type 1 diabetes, or multiple sclerosis (review @verstockt2018c).

Explained variance tends to be low.
The 163 loci, from the meta-analysis mentioned in the @mcgovern2015g review, explained only 13.6% of CD and 7.5% of UC total disease variance.
According to @verstockt2018c (review), on 26% of CD variance is accounted for by disease-associated SNPs.
This could be due to epistasis (gene-gene interactions) or gene-environment interactions not being modeled, or because some regions are not well studied (repeats, sex chromosomes), e.g. the Fc receptor gene cluster or the KIR genes.
Rare variants might have a large effect, although the first studies on these variants failed to show it (review @verstockt2018c).
Of course, the heritability estimates might be inflated too.

A missense allele in NOD2 was one of the first associated loci found with CD.
NOD2 was originally one of the few examples of successes by linkage mapping approaches.
It supported the hypothesis that CD may be caused by innate immune cell function deficiencies, and pointed at innate lymphoid cells as key players in CD.
GWAS hits related to autophagy seem to be specific to CD.

The discovery of a missense allele in ATG16L1 lead the field to explore the role of autophagy in CD.
An exonic variant in IRGM, a gene related to autophagy also identified early, affects the binding site of a microRNA (miR-196) expressed in intestinal epithelia and likely repressing IRGM.

Multiple independent risk alleles for IBD were found in the IL23R gene region.
Associated loci provided evidence that anti-inflammatory mechanisms may be disrupted in CD.
Several GWAS hits are located near genes involved in IL-10 signaling, an anti-inflammatory cytokine, for example IL10, IL10RB, STAT3, TYK2.
GWAS hits were also found near other cytokines, e.g. IL22, SMAD3, SMAD7.
Several of these SNPs related IL-23/IL-17 have been associated with other autoimmune diseases such as psoriasis, ankylosing spondylitis, or multiple sclerosis (review @verstockt2018c).
However, the signals the SNPs might be independent or have opposing effects across those traits.
For example, a missense risk allele for CD in TYK2 look protective for psoriasis.

W134X in FUT2 is a CD-specific association that suggested a potential importance of the mucus layer.

Some associations are not replicated in all human populations.
For example, NOD2 and ATG16L1 were not found to be associated with CD in East Asians (review @mcgovern2015g).
However, ATG16L2 seemed to be, according to GWAS studies in Korean and Japanese populations.

In 2010, a large meta-analysis included ~22K cases and 29K controls from 13 different countries (review @verstockt2018c).
It found 71 susceptibility loci.

- Summary table of known loci: see table 1 in review @verstockt2018c
- Consortium of note: International Inflammatory Bowel Disease Genetic Consortium (IIBDGC).

### Functional fine-mapping

Like many other complex diseases, it is difficult to identify the causal gene driving the association signal. 
This is due, among other factors, to large LD blocks, missing variants, cell type specific effects, 


Targeted sequencing around GWAS hits can also help identify likely causal genes.
For example, it uncovered rare missense alleles in PRDM1 or CARD9.

Expression QTLs can also help narrow down the list of gene candidate in a region.

Single-cell RNA seq studies may help understand the different cell types involved in IDB ( see [scIBD](http://scibd.cn/)).

Of note, the set of associated loci can still provide useful information, even if causal variants remain uncharacterized. 
For example, the loci can be compared to epigenetic profiles across cell types to identify the ones that might be involved.
For IBD, the strongest enrichment was observed within CD4+ T cells. 
CD showed an enrichment in immune cells, while UC was also enriched in colonic mucus.

# Cancer

## Leiomyosarcoma (LMS) 

Leiomyosarcoma (LMS) is the most common soft tissue and uterine sarcoma.
It's the malignant tumor of smooth muscles, which are found in hollw structures like intestines, bladder, uterus, blood vessels.
Unpredictable and usually not responsive to chemotherapy, the most effective treatment seems to be surgical removal.

Subtypes include:

- Uterine leiomyosarcoma
- Cutaneous leiomyosarcoma
- Gastrointestinal leiomyosarcoma

SRF: Serum Response Factor (SRF), transcription factor involved in cell proliferation, cell death resistance, metastasis

Alternative Lengthening of Telomeres (ALT) has been associated with poorer outcome. 
Instead of using telomerase to extend/maintain telomeres, the ALT pathway uses an homologous recombination-based pathway and allow cancerous cells to evade senescence due to telomere shortening.

Low mutation burden of ~40 somatic non-silent (missense or nonsense) coding SNVs per sample (RevChibon2019).
Cancer with one of the highest number of rearrangements, but they are mostly private and don't typically create gene fusions.

We expect multiple activation/inactivation of oncogenes/tumor suppressors with combined effects rather than one event.

Recurrent events:

- TP53 pathway altered
- p16/RB1 pathway altered
- PI3K/mTOR pathway dysregulation
- ATRX frequently mutated by SNVs, and associated with ALT phenotype

### Structural variants of LMS

More somatic losses than gains (RevChibon2019).
LMS are often tetraploid.

Exclusive genomic alterations:

- MYOCD amplification
- PTEN homozygous deletion
- DMD structural rearrangement

Gene fusions:

- TRIO-TERT?

### To read

- [Comprehensive and Integrated Genomic Characterization of Adult Soft Tissue Sarcomas. Cell 2017](https://pubmed.ncbi.nlm.nih.gov/29100075/)
- [Integrative genomic and transcriptomic analysis of leiomyosarcoma. Nat Commun 2018](https://pubmed.ncbi.nlm.nih.gov/29321523/)

## Neoantigens

Neoantigens are antigens produced by tumors and that could be targeted for treatment.
They can arise from tumor viruses integrating in the cancerous genome, or as products of mutant proteins.

Most specific neoantigens might be non-coding and will require better tools to be identified.

### To read

- [Neoantigen: A New Breakthrough in Tumor Immunotherapy. Front Immunol 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8085349/)
- [Noncoding regions are the main source of targetable tumor-specific antigens. Sci Transl Med 2018](https://pubmed.ncbi.nlm.nih.gov/30518613/)
