---
sidebar: true
title: Structural Variation Methods
bibliography: [../../library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---


<div id="TOC">
<ul>
<li><a href="#cytogenetic-techniques">Cytogenetic techniques</a></li>
<li><a href="#array-cgh-technology">Array-CGH technology</a><ul>
<li><a href="#circular-binary-segmentation">Circular binary segmentation</a></li>
<li><a href="#shifting-level-model">Shifting Level Model</a></li>
</ul></li>
<li><a href="#snp-array">SNP-array</a></li>
<li><a href="#short-read-sequencing">Short-read sequencing</a><ul>
<li><a href="#read-depth-rd">Read-Depth (RD)</a><ul>
<li><a href="#rd-in-one-sample">RD in one sample</a></li>
<li><a href="#case-vs-control">Case vs Control</a></li>
<li><a href="#choosing-the-optimal-window-size">Choosing the optimal window size</a></li>
</ul></li>
<li><a href="#rd-in-several-samples">RD in several samples</a></li>
<li><a href="#methods-for-repeat-rich-regions">Methods for repeat-rich regions</a></li>
<li><a href="#pem-and-split-reads">PEM and Split-reads</a></li>
<li><a href="#assembly-approach">Assembly approach</a></li>
<li><a href="#sv-genotypers">SV genotypers</a></li>
<li><a href="#others">Others</a></li>
<li><a href="#benchmark">Benchmark</a></li>
<li><a href="#misc">Misc</a><ul>
<li><a href="#mapping-algorithm-effect">Mapping algorithm effect</a></li>
<li><a href="#comparing-sv-calls">Comparing SV calls</a></li>
<li><a href="#in-silico-validation">In Silico validation</a></li>
<li><a href="#visualization">Visualization</a></li>
</ul></li>
</ul></li>
<li><a href="#long-read-sequencing">Long-read sequencing</a></li>
<li><a href="#single-cell-sequencing">Single-cell sequencing</a><ul>
<li><a href="#from-scrna-seq">From scRNA-seq</a></li>
</ul></li>
<li><a href="#bibliography">Bibliography</a></li>
</ul>
</div>

<p>Different technologies and methods have been used over the years to detect structural variation.
The first genome-wide assessment tool was the hybridization array.
Recently, high-throughput sequencing has shown better resolution.
The next step should be long-read sequencing technologies which will allow interrogation of the remaining challenging regions and SV types.</p>
<div id="cytogenetic-techniques" class="section level1">
<h1>Cytogenetic techniques</h1>
<p>Aneuploidy and extremely large SVs can be seen in directly in the karyotype, especially after G-banding (~1960-70s).
In theory deletions, translocations and inversions can be seen in the karyotype.</p>
<p>Fluorescent in situ hybridization (FISH) uses fluorescent probes that bind to complementary DNA in the genome.
Developed in the 1980s, it’s used to detect the presence or absence of specific DNA sequences in the sample.
Many probes come from DNA fragments that were isolated previously and kept in the form of bacterial artificial chromosomes.
Briefly, the cells and chromosomes are fixed and the probe is incubated for several hours, after which a microscope inspection reveals the location of the bindings.</p>
</div>
<div id="array-cgh-technology" class="section level1">
<h1>Array-CGH technology</h1>
<p>One could argue that aCGH is a cytogenetic technique?
Anyway, it was such a big improvement in the CNV detection field that it deserves its own section.</p>
<p>DNA from a test sample and normal reference sample are labeled differentially, using different fluorophores, and hybridized to several thousand probes.
The probes are derived from most of the known genes and non-coding regions of the genome, printed on a glass slide.</p>
<div class="figure">
<img src="/Array-CGH_protocol.svg" alt="Protocol" />
<p class="caption">Protocol</p>
</div>
<p>Using this method, copy number changes at a level of <strong>5–10 kilobases</strong> of DNA sequences can be detected.
As of 2006, even high-resolution CGH (<em>HR-CGH</em>) arrays are accurate to detect structural variations (SV) at resolution of <strong>200 bp</strong>.</p>
<p>This method allows one to identify new recurrent chromosome changes such as microdeletions and duplications in human conditions such as cancer and birth defects due to chromosome aberrations.
This technology is not able to detect balanced chromosomal imbalances as translocations or inversions.</p>
<div id="circular-binary-segmentation" class="section level2">
<h2>Circular binary segmentation</h2>
<p>For example improved by <span class="citation">Venkatraman and Olshen (<a href="#ref-Venkatraman2007">2007</a>)</span>, CBS iteratively identify segment with different mean value.</p>
<p>It computes a maximal t-statistic \(T=max_{1 i &lt; j m}|T_{ij}|\), where \(T_{ij}\) compared the mean of observations between <em>i+1</em> and <em>j</em> to the rest using the formula:</p>
<p><span class="math display">\[ T_{ij} = \frac{\bar{Y}_{ij}-\bar{Z}_{ij}}{s_{ij}\{(j-i)^{-1}+(m-j+i)^{-1}\}^{1/2}} \]</span></p>
<p>where <span class="math display">\[\bar{Y}_{ij}\]</span> and <span class="math display">\[\bar{Z}_{ij}\]</span> are the mean values within and outside of <em>i</em> and <em>j</em>.</p>
</div>
<div id="shifting-level-model" class="section level2">
<h2>Shifting Level Model</h2>
<p><span class="citation">Magi et al. (<a href="#ref-Magi2010">2010</a>)</span> model the \(log_2\) ratios as the sum of two independent stochastic processes.
One is the biological process and results from variation in DNA copy number while the second one models the noise due to technical errors.</p>
</div>
</div>
<div id="snp-array" class="section level1">
<h1>SNP-array</h1>
<p>Some methods are implemented for normal genomes (PennCNV <span class="citation">(Chen et al. <a href="#ref-Chen2013">2013</a>)</span>, iPattern, QuantiSNP, Birdsuite, Genotyping Console).
Others are capable of dealing with complex signal in tumoral samples (ASCAT <span class="citation">(Van Loo et al. <a href="#ref-VanLoo2010">2010</a>)</span>, GAP <span class="citation">(Popova et al. <a href="#ref-Popova2009">2009</a>)</span>).</p>
<p>Concordance between the results of different methods is often weak, pushing several studies to adopt an ensemble approach. In this approach, several (2-3 usually) methods are ran on the same SNP-array and discordant calls are filtered <span class="citation">(Walker and Scherer <a href="#ref-Walker2013">2013</a>)</span>.</p>
</div>
<div id="short-read-sequencing" class="section level1">
<h1>Short-read sequencing</h1>
<p>Compared to Array-CGH technologies, High-Throughput Sequencing allows for better breakpoints resolution, identification of a broader range of structural variant types and better estimation of larger Copy-Number Variation.
It also has the potential to identify allele-specific structural variation.</p>
<div id="read-depth-rd" class="section level2">
<h2>Read-Depth (RD)</h2>
<p>In this approach, the <strong>genome is partitioned</strong> into windows, usually non-overlapping, and the <strong>number of reads in each window</strong> is used as a <strong>read depth(RD)</strong> measure.
Most of the time, random distribution of the sequenced reads across the genome is assumed.
Yet several technical factors affect read depth (GC content, open regions, mappability) and are not completely corrected for.</p>
<div id="rd-in-one-sample" class="section level3">
<h3>RD in one sample</h3>
<p>In <span class="citation">Campbell et al. (<a href="#ref-Campbell2008">2008</a>)</span> <strong>circular binary segmentation</strong> algorithm is adapted.
Two lung cancer cell lines were sequenced using GenomeAnalyzer: paired end 29-36bp reads with ~200/400 bp insert size.
The main discovery approach used paired end information; RD was used as additional information.</p>
<p>Coverage was normalized using custom bin size estimated from a simulation analysis to standardize the number of unique mapping.
It resulted in ~425 simulated reads per bin and an average bin size of 15 Kbp.
Region within 1Mb of the centromere or telomere were discarded.
To detect copy number variation, a method was adapted from array CGH approach using a fast version of circular binary segmentation algorithm, implemented in *DNACopy} package from Bioconductor <span class="citation">(Venkatraman and Olshen <a href="#ref-Venkatraman2007">2007</a>)</span>.</p>
<p>The minimum event detected was 30 Kbp long.
They also noticed some amplicon amplification with up to 35-50 fold amplification.</p>
<hr />
<p><strong>RDXplorer</strong> aka Event-Wise Testing is used on single read sequencing results(~30X coverage) from Genome Analyzer <span class="citation">(Yoon et al. <a href="#ref-Yoon2009">2009</a>)</span>.
The method was applied to five samples (CEU trio: NA12878 NA12891 NA12892 + NA18507 YH).</p>
<p>The genome was cut into 100 bp windows in order to ensure breakpoint identification accuracy and small event detection.
The normality was assessed with QQplot and loosely accepted.
GC bias was corrected by determining the deviation in coverage of each GC class and linear correction.
The RD is converted into a Z-score (dividing by the genome average and standard deviation).
Duplication and deletion were separated by the computation of two different P-values \(p^U_i=P(Z&gt;z_i)\) and \(p^L_i=P(Z&lt;z_i)\) in order to join consistent signal.
Based on this significance testing, small events meeting some significance are first identified and clustered together and then grouped into larger events.
Then to assess the significance of <em>l</em> consecutive windows <em>A</em> among <em>L</em> total windows(here in the chromosome), \(max{p^U_i|iA}\) and \(max{p^L_i|iA}\) were compared to \(()^{}\) to control for a false-positive rate FPR.
In practice, to join the small events, the size of window interval was incremented and tested until losing significance(here FDR 5%).
Some additional filters are also necessary to clean up the results: minimum copy number deviation, small clusters merging, significance threshold choice based on manual inspection of results.
The copy number was estimated by rounding up \(\).</p>
<p>The results were compared across the five samples to define poly/monomorphic events.
To validate their call they used the data from the Genome Structural Variation (NimbleGen tiling array) as reference.
However, instead of overlapping the SVs for the correspondent samples, they used the SVs from the whole set of 40 samples.
Finally, they compared their results to the one from PEM approaches on the same samples.
The SVs detected being very different between the two techniques, the conclusion was that both methods had his advantages to different types of SVs.</p>
<hr />
<p><strong>mrFAST</strong> was used to map more robustly reads to duplicated regions of the human genome <span class="citation">(Alkan et al. <a href="#ref-Alkan2009">2009</a>)</span>.
Previous knowledge on the copy number state guided the parameter of the segmentation approach.
In the end, 6 out of 7 consecutive bins (5 Kbp) with a deviation exceeding 3 SD defined a CNV call.
They also corrected for GC bias using a LOESS-based smoothing technique.</p>
<hr />
<p><strong>Control-FREEC</strong> can run on one sample although it is recommended to provide a control.
More on the method <a href="#controlfreec">below</a>.</p>
<hr />
<p><strong>CNVnator</strong> combines mean-shift approach with additional refinements as GC correction and multiple-bandwidth partitioning <span class="citation">(Abyzov et al. <a href="#ref-Abyzov2011">2011</a>)</span>.
It rapidly (a few hours) analyze small bins (around 100bp) to detect CNVs from a wide range of sizes (hundreds to megabases in length).</p>
<p>GC bias was corrected by binning and average adjustment.
The bins were partitioned into segment using a mean-shift technique, that comes from image processing.
Briefly, bins point at the closest neighboring bin in a certain bandwidth and breakpoint are identified by two bins diverging from each other.
They improved the bandwidth estimation strategy.
Then they compare the average bin counts in the segments as well as the neighboring segments.
To deal with the issue of multiple mapping and the potentially duplicated calls, they record the fraction of reads with mapping quality zero.
Additionally they normalized the counts by the global coverage to estimates count, ultimately rounding off these values.</p>
<p>They used two 1KGP trios to test the reproducibility of their calls and stressed the importance of coverage uniformity compared to actual sequencing depth.
A minimum of 50% overlap defined concordance between two calls.
Then they simply checked the general direction when comparing parent-parent and parent-child overlap.
After explaining the challenges of duplication detection, they noticed 50% of their duplications located close to gaps.
Finally, they explored potential de novo or multi-allelic variants.
If variants are present in parent they could create de novo or multi-allelic event.
They didn’t validate any but mentioned that the immunoglobulin lambda locus was potentially true somatic variants.</p>
<hr />
<p><strong>cnD</strong> (Jared Simpson and Richard Durbin) is HMM approach using RD and allele imbalance to call CNVs from homozygous strains of mice.</p>
<hr />
<p><strong>CNV-TV</strong> uses total variation penalized least squares model to detect changes in RD of a sample <span class="citation">(Duan et al. <a href="#ref-Duan2013a">2013</a>)</span>.
It improves the segmentation.
Apparently better across a wider range of sizes and copy numbers compared to other methods.
Also better breakpoint accuracy.</p>
<p>The figures are a bit unusual, maybe the authors’ background leans more toward math/CS?</p>
<hr />
<p><strong>CNVfinder</strong> uses HMM to model bias and coverage variation using a negative binomial <span class="citation">(McCallum and Wang <a href="#ref-McCallum2013">2013</a>)</span>.
After normalization the coverage follows even less a Poisson so a negative binomial is better suited.
The coverage, GC content and mappability are included in the emission probabilities.</p>
<p>The package name changed from the beginning of the paper to the end (CNVwire).</p>
<hr />
<p><strong><span class="citation">Sepúlveda et al. (<a href="#ref-Sepulveda2013">2013</a>)</span></strong> developed a more flexible Poisson model to analyze genomes of micro-organisms, such as malaria.
The rate of the Poisson distribution varies across the genome according to a Gamma (negative binomial) or Lognormal distribution.</p>
<hr />
<p><strong>GENSENG</strong> <span class="citation">(Szatkiewicz et al. <a href="#ref-Szatkiewicz2013">2013</a>)</span> uses a complex HMM strategy to fit different models in order to take into account biases.
A negative binomial is used to account to over-dispersion and the effect of confounders.
Unknown confounders are taken into account by the over-dispersion parameter and a noise component in a mixture model.
The input is a triplet of RD, GC content and mappability for each bin.
They used 500 bp windows with 200 bp overlap (why not).</p>
<hr />
<p><strong>GROM-RD</strong> <span class="citation">(Smith, Kawash, and Grigoriev <a href="#ref-Smith2015">2015</a>)</span> aims at removing genomic biases before performing segmentation.
It’s unique features include excessive coverage masking, GC bias mean and variance normalization, GC weighting, dinucleotide repeat bias correction and usage of bins of mutliple sizes.
Masking regions with excessive coverage is actually used only in the <em>no-repeat</em> mode and the method is also ran on the unmasked genome with both call sets outputted in the end.
The GC content is computed from the GC weights which are computed at the base level using bases within the average insert size.
Quantile normalization the ensures that both the mean and variance in RD are normalized per GC class.</p>
<p>A big “recipe” with different analysis for different regions.
Comprehensive but also a lot of arbitrary thresholds.</p>
</div>
<div id="case-vs-control" class="section level3">
<h3>Case vs Control</h3>
<p>The RD in the case sample is compared to the control sample. The concept is similar to aCGH, but with much more data points and control.</p>
<hr />
<p><strong>SegSeq</strong> <span class="citation">(Chiang et al. <a href="#ref-Chiang2009">2009</a>)</span> uses a different approach, it looks for breakpoints comparing tumor-normal log-ratio of two consecutive windows.
Random distribution of the sequenced read and Poisson is assumed.
Using the normal sample as control should take care of most of the GC bias and mappability.
It’s not clear in the main text how the RD from the two samples are normalized.</p>
<p>In more details, each tumor read position is first tested for being at a breakpoint location by extending two windows to the left and right by <em>w</em> normal read.
A P-value is derived from the differences between the two log-ratios.
Candidate breakpoints (\(P-value&gt;p_{init}\)) are then eliminated by recomputing the P-value using the whole segments and tested against a minimal \(p_{merge}\).</p>
<p>The events detected in their application ranged from 29kb to 582kb.</p>
<hr />
<p><strong>CNV-seq</strong> <span class="citation">(Xie and Tammi <a href="#ref-Xie2009">2009</a>)</span> tries to estimate the best window size and uses \(log_2\) copy ratio between two samples and a sliding-window approach.
It uses a Poisson model and assumes the reads are randomly distributed across the genome.
Heavily inspired by aCGH technology.
Statistical analysis of power according to coverage but practical use not really described/clear.</p>
<hr />
<p><strong>CNAseg</strong> <span class="citation">(Ivakhno et al. <a href="#ref-Ivakhno2010">2010</a>)</span> segments RD signal using a HMM tumor-normal approach.
Notably, the difference in read counts between normal and tumor is used and modeled by Skellam distribution.</p>
<p>Normal and tumor read counts are normalized by their total number of reads beforehand.
GC bias is corrected using LOESS regression and mappability issues are <em>taken care</em> of by smoothing the signal with Discrete Wavelet Transform.</p>
<p>On the last steps, segments are merged using a \(^2\) test.
The threshold for the test is derived by mixing flowcells from normal and tumor and running control segmentation expected to contain only noise.</p>
<hr />
<p><strong>ExomeCNV</strong> <span class="citation">(Sathirapongsasuti et al. <a href="#ref-Sathirapongsasuti2011">2011</a>)</span> looks at the coverage and B allele frequency in exome sequencing data.</p>
<hr />
<p><strong>BIC-seq</strong> <span class="citation">(Xi et al. <a href="#ref-Xi2011">2011</a>)</span> minimize the Bayesian information criterion in order to get the best segmentation of the tumor/normal RD ratio.</p>
<hr />
<p><strong>VarScan 2</strong> <span class="citation">(Koboldt et al. <a href="#ref-Koboldt2012">2012</a>)</span> apply log ratio approach to exome data, controlling carefully for similar sequencing of the normal and tumor sample(similar hybridization).</p>
<p><strong>+++</strong></p>
<hr />
<div id="controlfreec" class="section level6">
<h6></h6>
<p><strong>Control-FREEC</strong> <span class="citation">(Boeva et al. <a href="#ref-Boeva2011">2011</a>, <a href="#ref-Boeva2012">2012</a>)</span> aims at predicting copy number variation along with LOH.
The read depth can be corrected for GC bias and mappability if a control sample is unavailable.
For cancer, the method can also estimate the stromal contamination or ploidy.
Both germline and somatic variants can be called.</p>
<p>The read depth is computed in windows tiling the genome.
The RD in the sample is normalized by dividing by the <em>expected read-depth</em>.
If a control sample is provided, the <em>expected read-depth</em> is computed by fitting the RD of the sample vs control, forcing an intercept of 0 (cubic polynomial recommended).
If not, the GC content is fitted using a cubic polynomial to provide an <em>expected read-depth</em>.
A mappability track, either provided or as the proportion of Ns in the window, is used to scale the <em>expected read-depth</em>.
Of note, centromeric/telomeric regions are excluded using a configurable distance threshold (default 50 kbp).
The normalized RD is segmented using a lasso-based algorithm that is robust to outliers.
The main parameter for the segmentation is the number of breakpoints and the tool will explore an increasing number of breakpoints, stopping once the residual sum of squares stops decreasing.</p>
<p>In its latest version, it integrates allelic imbalance estimates.
The B-allele frequency (BAF) of known SNPs is used for both segmentation and genotyping.
First, the windows are segmented using the same lasso-based algorithm on the median BAF deviation from expectation (0.5).
This new set of breakpoints is merged with the RD-based breakpoints.
The copy number in each segment is based on the RD but the genotyping on the BAF (maximum likelihood using Gaussian mixture models).</p>
<hr />
<p><strong>CNAnorm</strong> <span class="citation">(Gusnanto et al. <a href="#ref-Gusnanto2012">2012</a>)</span> corrects for cancer genome size and tumour cell content to segment tumor/normal RD ratios.</p>
<hr />
<p><strong>CoNVEX</strong> <span class="citation">(Amarasinghe, Li, and Halgamuge <a href="#ref-Amarasinghe2013">2013</a>)</span> is an exome approach using discrete wavelet transform on tumor/normal coverage ratio in each exonic region. The segmentation is performed through Hidden Markov Models.</p>
<hr />
<p><strong>WaveCNV</strong> <span class="citation">(Holt et al. <a href="#ref-Holt2013">2013</a>)</span> uses a translation-invariant discrete wavelet transform to detect CNV breakpoints from tumor/normal RD ratios.</p>
<hr />
<p><strong>AbsCN-seq</strong> <span class="citation">(Bao, Pu, and Messer <a href="#ref-Bao2014">2014</a>)</span> is an exome approach to detect tumoral variant taking into account purity and ploidy.</p>
<hr />
<p><strong>CONSERTING</strong> <span class="citation">(Chen et al. <a href="#ref-Chen2015">2015</a>)</span> uses regression trees to segment the tumor/normal coverage ratio.</p>
<p>Interestingly the breakpoint are fine-tuned using local SV detection around the segmented calls with <em>CREST</em>.</p>
</div>
</div>
<div id="choosing-the-optimal-window-size" class="section level3">
<h3>Choosing the optimal window size</h3>
<p>Control-FREEC defines the optimal window size using the theoretical coefficient of variation in a window <span class="citation">(Boeva et al. <a href="#ref-Boeva2011">2011</a>)</span>.
This CV is computed from the total read count, assuming a Poisson distribution.
The authors suggest a CV from 0.05 and 0.1.</p>
</div>
</div>
<div id="rd-in-several-samples" class="section level2">
<h2>RD in several samples</h2>
<p><strong>JointSLM</strong> <span class="citation">(Magi et al. <a href="#ref-Magi2011">2011</a>)</span> simultaneously analyzes several samples to increase statistical power to detect shared events.
It can detect common CNVs as small as 500 bp in their analysis.</p>
<p>Uniform distribution of the reads is assumed to use and model the count data as the same Poisson for the entire genome.
A specific Hidden-Markov Model, coined Shifting Level Model (SLM) model the log2 ratios as a mixture of a white noise process and a biological process.
100bp bins were corrected for GC, median-normalized and approximated by a Normal distribution.
The user-defined parameters are: the transition probability from a state(here CN across all samples), the proportion of the signal variance due to the biological process, and the number of states.
The other state parameters are first estimated through Baum and Welch algorithm.
Finally the best state sequence is estimated by Viterbi.</p>
<p>To test performance, synthetic chromosomes were constructed by mixing bins from chromosome 1 and X from a male individual.
Tweaking the input parameters they obtain good results…
Then they compare the results from three other methods (ran using their default parameters) and as expected their parameter-tweaked method performs better!
In practice they recommend to call event of size 500 bp or more, but, at this resolution, JointSLM is not that much better.
Although I find this synthetic chromosome approach clever, it’s not that realistic (e.g. doesn’t recapitulate repeat context) and only works for deletions.</p>
<p>For their real data analysis, they mixed 2 trios and 2 unrelated individuals and used a “conservative” (but arbitrary) set of parameters to get a round number of detected CNVs (3000).
They notice an over-representation of calls in centromere/telomeres (many systematic errors I would guess).
Using known annotation (DGV, then GSV) JointSLM had 58% specificity and 43% sensitivity, much lower than the simulation results.</p>
<p>These results were compared to PEM (not a RD approach) and found that JointSLM, designed to find CNV from RD, was better, according to the truth (aCGH CNV calls), compared to PEM approaches designed for deletion/insertions.</p>
<p>Finally, they used the copy number to cluster regions and samples.
They found two clear clusters that represented sample ancestry.</p>
<hr />
<p><strong>Copynumber</strong> <span class="citation">(Nilsen et al. <a href="#ref-Nilsen2012">2012</a>)</span> is a R package that can analyze single of multiple samples together to segment the genome.</p>
<hr />
<p><strong>cn.MOPS</strong> <span class="citation">(Klambauer et al. <a href="#ref-Klambauer2012">2012</a>)</span> integrates in their Bayesian model , read depth signal for all samples.
They use a mixture of Poisson distributions to model the copy number of each sample and technical variation.</p>
<p><strong>+++++</strong></p>
<hr />
<p><strong>XHMM</strong> <span class="citation">(Fromer et al. <a href="#ref-Fromer2012">2012</a>)</span> applies a PCA-based normalization of exome read depth and performs coverage segmentation using Hidden Markov Model. The RD is normalized using the mean RD across all samples; regions are filtered out if too variable across the samples.</p>
<hr />
<p><strong>CoNIFER</strong> <span class="citation">(Krumm et al. <a href="#ref-Krumm2012">2012</a>)</span> is an exome approach that normalize coverage and compute Z-score using information from several samples.</p>
<hr />
<p><strong>CONTRA</strong> <span class="citation">(Li et al. <a href="#ref-Li2012">2012</a>)</span> is another exome approach that segment ratios whose baseline is computed using reference samples.</p>
<hr />
<p><strong>EXCAVATOR</strong> <span class="citation">(Magi et al. <a href="#ref-Magi2013">2013</a>)</span> is another exome approach that segment ratios whose baseline is computed using reference samples. It uses Hidden Markov Models.</p>
<hr />
<p><strong>FishingCNV</strong> <span class="citation">(Shi and Majewski <a href="#ref-Shi2013">2013</a>)</span> analyze exome data using information from a set of reference samples.</p>
<hr />
<p><strong>PatternCNV</strong> <span class="citation">(Wang et al. <a href="#ref-Wang2014">2014</a>)</span> analyze exome data using information from a set of reference samples.</p>
<hr />
<p><strong>ONCOCNV</strong> <span class="citation">(Boeva et al. <a href="#ref-Boeva2014">2014</a>)</span> is an amplicon sequencing approach that includes a multi-factor normalization and annotation of large CNVs. The amplicon technology is particularly useful to detect somatic events.</p>
<hr />
<p><strong>CNVkit</strong> <span class="citation">(Talevich et al. <a href="#ref-Talevich2014">2014</a>)</span> uses off-target reads from targeted sequencing and a series of corrections and normalization.
In a nutshell, non-targeted regions are partitioned in larger bins so that the coverage is comparable to the one in the targeted regions.
Then a series of normalization and corrections are applied and a typical segmentation approach is performed.</p>
<p>First the usefulness of off-targeted reads is verified by showing that copy-number correlated with adjacent targeted regions and aCGH data.
To remove technical noise, the coverage in each bin is normalized by the expected coverage in a set of reference samples.
However they note some biases to vary from sample to sample hence the needs for additional corrections.</p>
<p>For this end, <em>CNVkit</em> uses a rolling median technique to re-center bin counts with bins of similar GC content, repetitiveness, larger size and distance from other targets.
While the normalization using reference samples and GC bias correction seem to improve the signal, the fancy correction at the targets boundaries doesn’t show so much interest.
Moreover, several samples (out of not so many) are still outliers and will be influenced by technical noise.</p>
<p>Of note, bins are previously filtered if the coverage in the reference samples is too low or too variable, or if the repeat content is too high.
They don’t explain how the reference samples are normalized get the average coverage in each bin, they just mention Tuckey’s biweight something.</p>
<p>The analysis is mostly done using a Python library and is quite fast (supposedly seconds once the reads are counted).</p>
<hr />
<p><strong><span class="citation">Glusman et al. (<a href="#ref-Glusman2015">2015</a>)</span></strong> integrates RD information from thousands of samples. Its normalization and reference building may seem simplistic but it is extremely efficient. It is exactly the goal of the pipeline as it targets clinical usage.</p>
<hr />
<p><strong>Genome STRiP 2.0</strong> <span class="citation">(Handsaker et al. <a href="#ref-Handsaker2015">2015</a>)</span> is the extended version of <em>Genome STRiP</em>.
Now the read depth across many samples is used to detect multi-allelic CNVs.</p>
<p>Quickly the methods consists in <em>“carefully normalizing”</em> the coverage information across samples and fitting a Gaussian mixture model.
The normalization seems to be only GC bias correction.
I’m guessing kind of a sequencing depth normalization but it’s not explicit, even in the supplementary material.
The Gaussian mixture model is not described much either, although it should <em>“integrate sample-specific variation”</em>.
In practice the genome is scanned and when a regions deviates from an unimodal distribution, the window is fine-tuned to get the best Gaussian mixture model.
Regions where the model did not result in a series of discrete copy numbers were removed.
Complex regions, e.g. mixing both deletions and duplications, or inversions, are not addressed here.</p>
</div>
<div id="methods-for-repeat-rich-regions" class="section level2">
<h2>Methods for repeat-rich regions</h2>
<p>Tandem copy-number variation reconstruction by improved multi-mapped reads counting approach was proposed by <span class="citation">He et al. (<a href="#ref-He2011">2011</a>)</span>.</p>
<hr />
<p><strong>VariationHunter-CR</strong> <span class="citation">(Hormozdiari et al. <a href="#ref-Hormozdiari2010">2010</a>)</span> improves their previous PEM approach by modeling transposon insertion. It considers and chooses from the potential multi-mapping of a read to support a SV <span class="citation">(Halper-Stromberg et al. <a href="#ref-Halper-Stromberg2014">2014</a>)</span>.</p>
<hr />
<p><strong>CNVeM</strong> <span class="citation">(Wang et al. <a href="#ref-Wang2013">2013</a>)</span> uses an Expectation-Maximization to allocate ambiguous reads to the correct location and compute better copy number estimates in repeat-rich regions.</p>
<hr />
<p><strong>MELT</strong> finds mobile element insertions. It was developed for the 1000 Genomes Project <span class="citation">(Sudmant et al. <a href="#ref-Sudmant2015a">2015</a>)</span>.
It was also used to confirm and characterize a driver somatic L1 insertion in the APC gene in colorectal cancer <span class="citation">(Scott et al. <a href="#ref-Scott2016">2016</a>)</span>.</p>
<p>Insertions are detected using the classical PEM/split-read approach: reads are selected if mapping to the TE consensus, then filtered and monitored for breakpoint patterns (target site duplication).</p>
<p>It generally uses the transducted sequence to identify the source L1 but apparently can also use internal variants (see <span class="citation">Scott et al. (<a href="#ref-Scott2016">2016</a>)</span>).</p>
<hr />
<p><strong>Mobster</strong> detects mobile element insertion using a similar approach to MELT.</p>
<hr />
<p><a href="https://github.com/cancerit/TraFiC"><strong>TraFiC</strong></a> was used in <span class="citation">Tubio et al. (<a href="#ref-Tubio2014">2014</a>)</span> to detect <strong>somatic L1 insertion in cancer</strong>. It can detect <strong>solo-L1</strong> insertions as well as <strong>partnered and orphan transduction</strong>. With this method they could even reconstruct multi-jumps insertions.</p>
</div>
<div id="pem-and-split-reads" class="section level2">
<h2>PEM and Split-reads</h2>
<p><strong><span class="citation">Tuzun et al. (<a href="#ref-Tuzun2005">2005</a>)</span></strong> seem to be the first who used paired-end mapping to detect SVs.</p>
<hr />
<p><strong><span class="citation">Korbel et al. (<a href="#ref-Korbel2007">2007</a>)</span></strong> were one of the first to use paired-end mapping information to detect SVs.
They used outlier from the insert size distribution as well as inverted reads or distal insertions.
Looking at the paired-end information they could detect insertion, insertion of distal locus, deletions and inversions.
Two supporting read were necessary to call an event.</p>
<p><em>NA15510</em> and <em>NA18505</em> were used in the paper.
The samples were sequenced using 454 technology and ~250 bp reads with ~3 kb insert-size.
Using this technique, events of 3kb or larger could be detected.
They found signal for a slight selective constraint on against SVs in genes, enrichment of retrovirus- and transposition-related proteins.
Several SVs were also found in repetitive elements (<em>Alu</em>, L1, L2, SVA, LTRs) with a significant enrichment for L1s.
They estimated the origin of their SVs as mainly non-homologous end-joining (NHEJ) (56%), retrotransposition (30%) and a few non-allelic homologous recombination (NAHR) (14%).</p>
<p>To validate their calls they compared to DGV annotation, Celera assembly, array-CGH and PCR experiments.</p>
<hr />
<p><strong><span class="citation">Campbell et al. (<a href="#ref-Campbell2008">2008</a>)</span></strong> Two lung cancer cell lines were sequenced using GenomeAnalyzer: paired end 29-36 bp reads with ~200/400 bp insert size.</p>
<p>Aberrant paired reads supported by high mapping quality were targeted by a confirmatory screening (PCR amplification and conventional sequencing) that gave them the base-pair breakpoint resolution.</p>
<p>Mostly deletions were detected.
Around 50-60% of the SVs found were germlines (<em>Q?? I still don’t know how they distinguish germline/somatic from one cancer sample.</em>) and several L1 and AluY deletions were found.
Also a bunch of events created fusion transcript or exon duplication.
They observed <em>“genomic shards”</em>, i.e. small DNA fragments inserted around breakpoints, and inverted duplications.</p>
<hr />
<p><strong>BreakDancer</strong> <span class="citation">(Chen et al. <a href="#ref-Chen2009">2009</a>)</span> also monitors anomalous(in term of insert size or orientation) paired reads to detect large structural variants.
The confidence score for the calls integrates the number of pairs supporting it, the sequencing coverage and the insert size deviation.
The significance of the score is compute for each insert type by assuming an uniform distribution across the genome (not very realistic for repeat-rich regions).
It also propose a module to detect small indels, using the the insert size information of reads mapping to a window(of size the mean insert size), sliding it 1bp at a time.
It integrates samples pooling to increase detection power of common variants or flag inherited variation in normal vs tumor studies.</p>
<hr />
<p><strong>SVMiner</strong> <span class="citation">(Pyon, Hayes, and Li <a href="#ref-Pyon2011">2011</a>)</span> looks like a standard approach.</p>
<hr />
<p><strong>CLEVER</strong> <span class="citation">(Marschall et al. <a href="#ref-Marschall2012">2012</a>)</span> organizes all read into <em>read alignment graphs</em> and looks for cliques of consistent reads.</p>
<p><strong>++</strong></p>
<hr />
<p><strong>RetroSeq and SECluster</strong> ++</p>
<hr />
<p><strong>CommonLAW</strong> <span class="citation">(Hormozdiari et al. <a href="#ref-Hormozdiari2011a">2011</a>)</span> looks at PEM signal across several samples to increase the sensitivity and specificity.</p>
<hr />
<p><strong>Hydra-Multi</strong> <span class="citation">(Lindberg, Hall, and Quinlan <a href="#ref-Lindberg2014">2014</a>)</span> also looks at PEM signal across several samples to increase the sensitivity and specificity.</p>
<hr />
<p><strong>Pindel</strong> <span class="citation">(Ye et al. <a href="#ref-Ye2009">2009</a>)</span> uses pattern growth on the unmapped pair of uniquely mapped anchor read.
The mapped read (anchor) is used to reduce the search-space.
The unmapped read is cut in two or three and the ends are mapped around the anchor.
Breakpoints with at least 2 supporting reads are reported.
Depending on the read length, this approach can detected large deletions (1bp-10kb) and small insertions (1-20bp).</p>
<hr />
<p><strong>Splitread</strong> <span class="citation">(Karakoc et al. <a href="#ref-Karakoc2012">2012</a>)</span> in exome sequencing data, splits the unmapped read into balanced or unbalanced subsequences and detect cluster of sub-reads.
Something about <em>weighted set-cover approximation</em>.</p>
<p><strong>++</strong></p>
<hr />
<p><strong>YAHA</strong> <span class="citation">(Faust and Hall <a href="#ref-Faust2012">2012</a>)</span> implement a split read approach for long reads.</p>
<p><strong>++</strong></p>
<hr />
<p><strong>SplazerS</strong> <span class="citation">(Emde et al. <a href="#ref-Emde2012">2012</a>)</span> is another approach.</p>
<hr />
<p><strong>Gustaf</strong> <span class="citation">(Trappe et al. <a href="#ref-Trappe2014">2014</a>)</span> is yet another approach.</p>
</div>
<div id="assembly-approach" class="section level2">
<h2>Assembly approach</h2>
<ul>
<li>How to deal with repeats ? diploidy ?</li>
<li>What is the difference between global/local assembly ? (except computational complexity)</li>
<li>Why is it so much difficult to assemble (tandem) duplication ?</li>
</ul>
<p><strong>Advantages</strong> Firstly, assembled sequence gives extensive information about the breakpoint location and mechanism of formation.
Assembly approaches are also not biased toward the reference genome, which we realize can be significantly different from a studied human genome.
Hence It could theoretically simplify characterization of complex events.
Indeed if a complex event is defined according to its deviation from the reference, it is not particularly complex here.
Of course, other issues will create event that are complex for the assembly exercise.</p>
<p><strong>Disadvantages</strong> It requires more coverage and is computationally more demanding.</p>
<p><strong>Overlap-layout-consensus vs de Bruijn graphs</strong> Overlap-based graphs were first implemented.</p>
<p>In overlap-layout-consensus graphs, fragment assembly problem is cast as finding a path visiting every vertex exactly (or at most) once, a Hamiltonian Path Problem.
Here a vertex is a read; edges join reads that overlap.</p>
<p>In de Bruijn graphs, fragment assembly problem is cast as finding a path visiting every edge once, a Eulerian Path Problem.
De Bruijn graphs typically require less resources.
In theory, the graph resolution is also easier when repeats are present.</p>
<p><strong>Challenges</strong> Diploidy or cell heterogeneity is a major challenge.
Most of the whole-genome assembler aim at producing an haploid version of the sequenced genome, ignoring local structure.</p>
<p>Insertions are also more challenging to assemble.
The full inserted sequence can be problematic to retrieve.
The sequence redundancy is also more complex to assemble (and map/retrieve).</p>
<p>The choice of the k-mer size is very important.
It must be a tradeoff between complexity and coverage.
A small k-mer will create complex structure due to common k-mer in different part of the assembled sequence, but each node will be well covered.</p>
<hr />
<p><strong>Cortex</strong> <span class="citation">(Iqbal et al. <a href="#ref-Iqbal2012">2012</a>)</span> is a <em>de novo</em> assembly method that uses de Bruijn graphs to detect and genotype simple and complex variants in an individual or population.</p>
<hr />
<p><strong>Platypus</strong> <span class="citation">(Rimmer et al. <a href="#ref-Rimmer2014">2014</a>)</span> Platypus integrated mapping, assembly and haplotype approaches to call variants.
It also integrates information across multiple samples.</p>
<p>First a set of candidates is derived from read alignment, local assembly or external sources.
The local assembly uses reads mapping in a genomic window, as well as their mate, and reference sequences to construct a colored de Bruijn graph.
Unique paths that begin and end on reference sequence are extracted.
The candidate variants are then clustered and a list of potential haplotypes is defined.
Using reads from all samples, a diploid segregation model and an expectation-maximization approach the haplotype frequencies are derived from likelihood estimates.
These frequencies are finally used as prior to call haplotype in each sample, again by likelihood approximation.</p>
<p>This method seems to be highly sensitive and specific thanks to its comprehensive set of variants detected from the local assembly and realignment process.
It is supposedly not affected by repeat content, although the assembly is limited by the total read coverage and the mate of locally mapped reads.
Still they managed to detect full Alu insertions.
Here SV is described as an extra feature but the focus in on SNP and indel haplotype calling.
Hence few efforts have been directed toward detection of the wide range of SV types and sizes.</p>
<p>Among the remarkable results, the calling rate for de novo variant and HLA analysis are quite impressive.
Of note, Platypus works similarly with Whole-Exome Sequencing data.</p>
<hr />
<p><strong>TIGRA</strong> <span class="citation">(Chen et al. <a href="#ref-Chen2014">2014</a>)</span> Targeted Iterative Graph Routing Assembler was widely used in large-scale projects (e.g. 1KGP) for breakpoints assembly of deletions and mobile element insertion.
It’s an old effort, that they started in 2008, which was optimized and improved across the years.</p>
<p>To clarify, TIGRA assembles the breakpoint region of SV, not the entire SV region.
Another clarification: TIGRA is used to validate existing calls and reduce the FDR, not so much to characterize SV mechanism or features.</p>
<p>First, reads are extracted around (500bp-1Kbp) the boundaries of the input region.
Mapped reads and their mate are retrieved from the BAM files.
If multiple samples are available and their genotype is known, reads from the variant-containing samples can be pooled.
These reads are then assembled using a de Bruijn graph.
Multiple k-mer sizes are iteratively used, with previously assembled contigs taken as pseudo-reads in the next iterations.
During the contigs construction, tips likely created by sequencing errors are pruned.
Bubbles with less than 3 nucleotides differences are also joined.
Entire reads are used to distinguish repeats, instead of k-mers.
The assembled sequences are then mapped back to the reference genome using cross_match.
Every contig is tested although they don’t explain explicitly their how these contigs are retrieved from the graph.
Contigs with more than two hits or substitution rate higher than 0.5% (a bit low, no?) are not reported.
A glocal alignment is used.</p>
<p>They tested TIGRA assembly step against Velvet, SGA, Phrap and SPAdes.
They used datasets with known deletions breakpoints and mobile element insertions.
TIGRA shows the highest proportion of validated SVs in any situations.
They also managed to detect quasi-full length Alu insertion (was it because they knew it was there ?).
TIGRA is more robust to low-coverage alleles than other existing methods, thanks to its multiple k-mer size process.</p>
<p>It seems mainly focused on deletions, duplication (also not benchmarked) and mobile insertion, although they mention being able to detect any type of de novo SV and having specific strategies to deal with inversion or reciprocal translocations.</p>
<p>While it handles repeats slightly better than other methods, by using full reads instead of k-mers, repeated sequences longer than the read length will not be resolved and are bound to create ambiguous graphs.
Indeed they observed a reduction of assembled breakpoints with higher breakpoint homology.
It is implemented to assemble small breakpoint regions, hence require almost nucleotide resolution in its input regions.</p>
<p>They comment about a important drop in validation rate for RD-based regions.
I didn’t understand how they define their contigs (used in the iterative process; aligned against the reference).
Is it all possible path in the graph ? Only unambiguous paths ?</p>
<hr />
<p><strong>SGA</strong> <span class="citation">(Simpson and Durbin <a href="#ref-Simpson2012">2012</a>)</span> stands for String Graph Assembler and follows a overlap-based string graph approach.
Thanks to its FM-index representation and Burrow-Wheeler compressing transformation, it manages considerable economy in memory.</p>
<p>The overlap graph is simplified before being transformed into the string graph used to resolve contig assembly.
K-mer correction for sequencing error removal, duplicated reads removal and ``bubble-popping’’ are applied beforehand.
Reconstruction of the genome is done by finding a walk through the graph, where each vertices (reads) are connected if the reads overlap with minimum proportion.
Finally, the paired-end reads are aligned to the contigs to guide their ordering and assess their copy-number through coverage analysis.</p>
<p>To benchmark the method, they firstly used C. elegans reads.
This genome easier than a human genome in several sense: haploid, small (100 Mbp).</p>
<p>An interesting metric used to explore the assembler performance was computed by sampling strings of varying sizes from the reference genome and counting the fraction that is found exactly in one of the contig.</p>
<p>SGA showed the highest value for long strings.</p>
<p>I thought one of the strength of overlap-based method would be its robustness to sequencing errors or polymorphism.
In my head two reads would overlap significantly even if a few bases are discordant.
However, here, the overlap definition is based on comparing suffixes/prefixes, meaning that exact match is necessary.</p>
<p>SGA was modified in <span class="citation">Malhotra et al. (<a href="#ref-Malhotra2013">2013</a>)</span> to report all paths instead of the consensus, and validate the breakpoints regions of their tumoral calls.</p>
<hr />
<p><strong>AbySS</strong></p>
</div>
<div id="sv-genotypers" class="section level2">
<h2>SV genotypers</h2>
<p><span class="citation">Eggertsson et al. (<a href="#ref-Eggertsson2019">2019</a>)</span> improved their variant GraphTyper to integrate structural variants.
They show that it can genotype SNVs, indels and SVs from short-read data in large-scale datasets.
The approach is very similar to BayesTyper: SVs are discovered with methods likes Manta, then genotyped.
Many of the input SVs might be errors but it’s not a big problem because the genotyper can figure out if the variant is supported or not by mapped reads.
One limitation of this breakpoint-centric approach is that some informative reads might be discarded.</p>
</div>
<div id="others" class="section level2">
<h2>Others</h2>
<p><strong>SVMerge</strong> <span class="citation">(Wong et al. <a href="#ref-Wong2010">2010</a>)</span> is a pipeline applying multiple detection methods and using local de novo assembly to refine breakpoints.
Written in Perl, SVMerge is supposedly flexible and modular to integrate new callers.</p>
<p>The pipeline is divided in four modules: calling, filtering, merging and de novo assembly.
After calling, calls are filtered according to their intrinsic quality as well as their proximity to gaps (less 600bp), highly repetitive regions or centromere/telomere (less than 1 Mb).
Calls are then separated by type and merged.
After extension of their boundaries, SV are locally assembled (by default ABySS assembler) and the consensus aligned to the reference genome using Exonerate.
These alignments are finally parsed to retrieve convincing SVs.</p>
<p>They mention attempting to detect complex SV that they defined as simple deletion/duplication with additional inversions.</p>
<p>In the special situation of heterozygous deletion, read depth is also analyzed to be sure that these regions are not discarded due to the presence of one normal copy.
Still the clear limitation of this approach is its restriction to homozygous events.
At this stage the assembly is not suited for diploid genomes.</p>
<p>As another demonstration of their assembly validation, only 11% of their insertion could be validated.
To conclude on the subject of limitations, they also filtered out inter-chromosomal translocation candidates because too numerous.</p>
<hr />
<p><strong>CREST</strong> <span class="citation">(Wang et al. <a href="#ref-Wang2011">2011</a>)</span> uses both discordant paired-mapping and split-reads to call SVs.</p>
<hr />
<p><strong>DELLY</strong> <span class="citation">(Rausch et al. <a href="#ref-Rausch2012">2012</a>)</span> DELLY integrates information from short insert paired-reads and long-range mate-pairs as well as split-reads alignments.
It can theoretically detect any type of structural variation.
Still I’m suspecting the detection of duplications to be challenging with this approach.
They stress the advantage of mixing two sets of reads with different insert sizes.</p>
<p>The PEM side of the algorithm follows previous approaches: from the distribution of insert size, uniquely mapped read pairs are flagged as discordant if farther away than 3 times the distribution SD, or with discordant orientations.
Depending on the orientations and insert size the type of event is estimated: deletion, inversion, tandem duplication or translocation.
After being grouped by chromosome and sorted, the discordant mapping are used to build a graph with each node being a read and an edge meaning that they support the same SV.
Identification of SV is then done by looking up for cliques or sub-cliques.
Each type of event is analyzed separately.
These putative SVs are then screened for supporting split-reads.
This step first identify one-anchored reads (i.e. with one unmapped read).
They transformed the reference sequence in non-deletion to fit a deletion situation and developed a k-mer approach to avoid expensive exploration of all possible splits.
The exact breakpoint of a SV is found by comparing diagonal segment when mapping the reads k-mers to the reference sequence.
Obviously this approach is penalized by insertion of new sequences at the breakpoint location, especially if longer than half the read length.
Two split-reads are usually required to call a SV.
Finally the supporting split-reads are used to create a consensus sequence which re-aligned to the reference genome and define the final SV boundaries.
Additionally DELLY checks that both PEM and SR derived event are concordant.</p>
<p>They noticed an accumulation of putative calls in centro- and telomeric regions.
Believing it could be actual inter-individual variability or due to incomplete reference they didn’t completely filter these calls out.
Still, they limited the number of variant in any particular genomic region in order to avoid countless alignments.</p>
<hr />
<p><strong>SVM<span class="math inline">\(^2\)</span></strong> <span class="citation">(Chiara, Pesole, and Horner <a href="#ref-Chiara2012">2012</a>)</span> uses Support Vector Machine integrating coverage and paired-end read information.</p>
<p><span class="math inline">\(+++\)</span></p>
<hr />
<p><strong>forestSV</strong> <span class="citation">(Michaelson and Sebat <a href="#ref-Michaelson2012">2012</a>)</span> trained a random forest machine learning model with known variant. Different signals such as coverage, read-pair signature and others are integrated in the model. After training the method scans a BAM file to find probable variants.</p>
<hr />
<p><strong>GASVPro</strong> <span class="citation">(Sindi et al. <a href="#ref-Sindi2012">2012</a>)</span> integrates read depth and paired-end signal in their model and account for multiple alignments.</p>
<p>Apparently it weights multiple mapping per read in proportion to their probability <span class="citation">(Halper-Stromberg et al. <a href="#ref-Halper-Stromberg2014">2014</a>)</span>.</p>
<p><span class="math inline">\(+++\)</span></p>
<hr />
<p><strong>LUMPY</strong> <span class="citation">(Layer et al. <a href="#ref-Layer2014">2014</a>)</span> integrates multiple SV signals such as coverage, PEM and split-reads. It can be extended to new custom SV signals and can analyze several samples at the same time.</p>
<hr />
<p><strong>ERDS</strong> <span class="citation">(Zhu et al. <a href="#ref-Zhu2012">2012</a>)</span> integrates both coverage and B allele frequency in a Hidden Markov Model.</p>
<hr />
<p><strong><span class="citation">Yang et al. (<a href="#ref-Yang2013">2013</a>)</span></strong> integrates both paired-end information and split-reads detect somatic SV in cancer genomes.</p>
<hr />
<p><strong>SMUFFIN</strong> <span class="citation">(Moncunill et al. <a href="#ref-Moncunill2014">2014</a>)</span> analyzes directly the reads to find differences between pattern in the normal and tumor samples. Once somatic patterns are identified they can be mapped back to the reference genome.</p>
<hr />
<p><strong>SMaSH</strong> <span class="citation">(Talwalkar et al. <a href="#ref-Talwalkar2014">2014</a>)</span> is a toolkit to benchmark human genome variant calling. I think it’s also giving calls, maybe using a consensus of its preferred methods ?</p>
<hr />
<p><strong>SlideSort-BPR</strong> <span class="citation">(Wijaya et al. <a href="#ref-Wijaya2014">2014</a>)</span> is based on a fast all-against-all read comparison to find patterns of breakpoints.</p>
<hr />
<p><strong>CLImAT</strong> <span class="citation">(Yu et al. <a href="#ref-Yu2014">2014</a>)</span> looks at both RD and B allele frequency. Quantile normalization is used on the B allele frequency.</p>
<hr />
<p><strong>Parliament</strong> <span class="citation">(English et al. <a href="#ref-English2015">2015</a>)</span> is a consensus SV-calling infrastructure that merges multiple data types and SV detection methods. For example it can merge results from short and long read sequencing, as well as aCGH.</p>
<hr />
<p>The gnomAD pipeline was used in a few studies, including the one exploring the transcriptomic effects of rare SVs in brain <span class="citation">(CommonMind Consortium et al. <a href="#ref-CommonMindConsortium2020">2020</a>)</span>.
It consists of: Delly, Lumpy, Manta, Wham, MELT, CNVnator, GenomeSTRiP, cn.MOPS.</p>
<hr />
<p>The Hall lab used <a href="https://github.com/hall-lab/svtools">svtools</a> (discovery with LUMPY, CNVnator, svtyper; then genotyping) to perform a GWAS on Finns.</p>
</div>
<div id="benchmark" class="section level2">
<h2>Benchmark</h2>
<p>IS:Insert-size; Or:Orientation</p>
<table>
<thead>
<tr class="header">
<th>Year</th>
<th>Method</th>
<th>Approach</th>
<th>Note</th>
<th>Mult. samples</th>
<th>Event size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2007</td>
<td><span class="citation">Korbel et al. (<a href="#ref-Korbel2007">2007</a>)</span></td>
<td>PEM</td>
<td>IS/Or</td>
<td>no</td>
<td>&gt;~3kb</td>
</tr>
<tr class="even">
<td>2008</td>
<td><span class="citation">Campbell et al. (<a href="#ref-Campbell2008">2008</a>)</span></td>
<td>PEM, RD</td>
<td>IS/Or, CBS</td>
<td>no</td>
<td>CNV&gt;~30kb</td>
</tr>
<tr class="odd">
<td>2008</td>
<td>SeqSeg <span class="citation">(Chiang et al. <a href="#ref-Chiang2009">2009</a>)</span></td>
<td>RD</td>
<td>2 bins around breakpoints</td>
<td>case-control</td>
<td>&gt;~29kb</td>
</tr>
<tr class="even">
<td>2009</td>
<td>EWT <span class="citation">(Yoon et al. <a href="#ref-Yoon2009">2009</a>)</span></td>
<td>RD</td>
<td>Event-Wise Testing</td>
<td>no</td>
<td>&gt;~1kb</td>
</tr>
<tr class="odd">
<td>2009</td>
<td>CNV-seq <span class="citation">(Xie and Tammi <a href="#ref-Xie2009">2009</a>)</span></td>
<td>RD</td>
<td>theoretical</td>
<td>case-control</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2009</td>
<td>BreakDancer <span class="citation">(Chen et al. <a href="#ref-Chen2009">2009</a>)</span></td>
<td>PEM</td>
<td>nbReads+IS+coverage, Poisson sig.</td>
<td>&gt;~10bp</td>
<td></td>
</tr>
<tr class="odd">
<td>2009</td>
<td>Pindel <span class="citation">(Ye et al. <a href="#ref-Ye2009">2009</a>)</span></td>
<td>SR</td>
<td>Pattern-Growth</td>
<td>no</td>
<td>D:1bp-10kb, I:1-20bp</td>
</tr>
<tr class="even">
<td>2010</td>
<td>CNAseg <span class="citation">(Ivakhno et al. <a href="#ref-Ivakhno2010">2010</a>)</span></td>
<td>RD</td>
<td>HMM, flowcell varibility, Skellam dist.</td>
<td>no</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2010</td>
<td>VariationHunter-CR <span class="citation">(Hormozdiari et al. <a href="#ref-Hormozdiari2010">2010</a>)</span></td>
<td>PEM</td>
<td>Transposon model</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2010</td>
<td>SVMerge <span class="citation">(Wong et al. <a href="#ref-Wong2010">2010</a>)</span></td>
<td>PL</td>
<td><em>de novo</em> local assembly</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2010</td>
<td>NovelSeq <span class="citation">(Hajirasouliha et al. <a href="#ref-Hajirasouliha2010">2010</a>)</span></td>
<td>PEM+Assembly</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2011</td>
<td>JointSLM <span class="citation">(Magi et al. <a href="#ref-Magi2011">2011</a>)</span></td>
<td>RD</td>
<td>HMM, Common SVs</td>
<td>Yes</td>
<td>&gt;~500bp</td>
</tr>
<tr class="odd">
<td>2011</td>
<td>Control-FREEC <span class="citation">(Boeva et al. <a href="#ref-Boeva2011">2011</a>)</span></td>
<td>RD+BAF</td>
<td>CNV and contamination</td>
<td>single, pair</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2011</td>
<td>BIC-seq <span class="citation">(Xi et al. <a href="#ref-Xi2011">2011</a>)</span></td>
<td>RD</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2011</td>
<td>Genome STRIP <span class="citation">(Handsaker et al. <a href="#ref-Handsaker2011">2011</a>)</span></td>
<td>PEM</td>
<td>Population-based</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2011</td>
<td>CNVnator <span class="citation">(Abyzov et al. <a href="#ref-Abyzov2011">2011</a>)</span></td>
<td>RD</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>VarScan 2 <span class="citation">(Koboldt et al. <a href="#ref-Koboldt2012">2012</a>)</span></td>
<td>RD</td>
<td>Control, exome</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>cn.MOPS <span class="citation">(Klambauer et al. <a href="#ref-Klambauer2012">2012</a>)</span></td>
<td>RD</td>
<td>MS, Bayesian</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>CLEVER <span class="citation">(Marschall et al. <a href="#ref-Marschall2012">2012</a>)</span></td>
<td>PEM</td>
<td>Graphs</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>Splitread <span class="citation">(Karakoc et al. <a href="#ref-Karakoc2012">2012</a>)</span></td>
<td>SR</td>
<td>Exome</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>YAHA <span class="citation">(Faust and Hall <a href="#ref-Faust2012">2012</a>)</span></td>
<td>SR</td>
<td>Long reads</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>SVM2 <span class="citation">(Chiara, Pesole, and Horner <a href="#ref-Chiara2012">2012</a>)</span></td>
<td>RD+PEM</td>
<td>SVM</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>GASVPro <span class="citation">(Sindi et al. <a href="#ref-Sindi2012">2012</a>)</span></td>
<td>RD+PEM</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>LUMPY <span class="citation">(Layer et al. <a href="#ref-Layer2014">2014</a>)</span></td>
<td>PEM+SR</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>DELLY <span class="citation">(Rausch et al. <a href="#ref-Rausch2012">2012</a>)</span></td>
<td>PEM+SR</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2013</td>
<td>WaveCNV <span class="citation">(Holt et al. <a href="#ref-Holt2013">2013</a>)</span></td>
<td>RD</td>
<td>Cancer</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2013</td>
<td>CNV-TV <span class="citation">(Duan et al. <a href="#ref-Duan2013a">2013</a>)</span></td>
<td>RD</td>
<td>Segmentation</td>
<td>single, pair</td>
<td>&gt; 1Kbp</td>
</tr>
<tr class="even">
<td>2013</td>
<td>PeSV-Fisher <span class="citation">(Escaraḿis et al. <a href="#ref-Escaramis2013">2013</a>)</span></td>
<td>PEM+RD</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2013</td>
<td>CNVeM <span class="citation">(Wang et al. <a href="#ref-Wang2013">2013</a>)</span></td>
<td>RD</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2013</td>
<td>CNVfinder <span class="citation">(McCallum and Wang <a href="#ref-McCallum2013">2013</a>)</span></td>
<td>RD</td>
<td>HMM, GC/mappability</td>
<td>no</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2013</td>
<td>GENSENG <span class="citation">(Szatkiewicz et al. <a href="#ref-Szatkiewicz2013">2013</a>)</span></td>
<td>RD</td>
<td>HMM, GC/mappability</td>
<td>no</td>
<td>&gt;800 bp</td>
</tr>
<tr class="even">
<td>2014</td>
<td>CLImAT <span class="citation">(Yu et al. <a href="#ref-Yu2014">2014</a>)</span></td>
<td>RD+BAF</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2014</td>
<td>Gustaf <span class="citation">(Trappe et al. <a href="#ref-Trappe2014">2014</a>)</span></td>
<td>SR</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2014</td>
<td>SMASH <span class="citation">(Talwalkar et al. <a href="#ref-Talwalkar2014">2014</a>)</span></td>
<td>PEM+SR</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2015</td>
<td>GROM-RD <span class="citation">(Smith, Kawash, and Grigoriev <a href="#ref-Smith2015">2015</a>)</span></td>
<td>RD</td>
<td>Aggressive normalization</td>
<td>no</td>
<td>500 bp++</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th>Year</th>
<th>Method</th>
<th>GC correction</th>
<th>Map correction</th>
<th>CentTelomere dist.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2007</td>
<td><span class="citation">Korbel et al. (<a href="#ref-Korbel2007">2007</a>)</span></td>
<td>NA</td>
<td>NA</td>
<td>0</td>
</tr>
<tr class="even">
<td>2008</td>
<td><span class="citation">Campbell et al. (<a href="#ref-Campbell2008">2008</a>)</span></td>
<td>none</td>
<td>binning</td>
<td>1Mb</td>
</tr>
<tr class="odd">
<td>2008</td>
<td>SeqSeg <span class="citation">(Chiang et al. <a href="#ref-Chiang2009">2009</a>)</span></td>
<td>unnecessary</td>
<td>unnecessary</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2009</td>
<td>EWT <span class="citation">(Yoon et al. <a href="#ref-Yoon2009">2009</a>)</span></td>
<td>median</td>
<td>random mapping</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2009</td>
<td>CNV-seq <span class="citation">(Xie and Tammi <a href="#ref-Xie2009">2009</a>)</span></td>
<td>unnecessary</td>
<td>unnecessary</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2009</td>
<td>BreakDancer <span class="citation">(Chen et al. <a href="#ref-Chen2009">2009</a>)</span></td>
<td>NA</td>
<td>NA</td>
<td>pooled possible</td>
</tr>
<tr class="odd">
<td>2009</td>
<td>Pindel <span class="citation">(Ye et al. <a href="#ref-Ye2009">2009</a>)</span></td>
<td>NA</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2010</td>
<td>CNAseg <span class="citation">(Ivakhno et al. <a href="#ref-Ivakhno2010">2010</a>)</span></td>
<td>LOESS</td>
<td>DWT smoothing</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2010</td>
<td>VariationHunter-CR <span class="citation">(Hormozdiari et al. <a href="#ref-Hormozdiari2010">2010</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2010</td>
<td>SVMerge <span class="citation">(Wong et al. <a href="#ref-Wong2010">2010</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2010</td>
<td>NovelSeq <span class="citation">(Hajirasouliha et al. <a href="#ref-Hajirasouliha2010">2010</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2011</td>
<td>JointSLM <span class="citation">(Magi et al. <a href="#ref-Magi2011">2011</a>)</span></td>
<td>median</td>
<td>NA</td>
<td>0</td>
</tr>
<tr class="odd">
<td>2011</td>
<td>Control-FREEC <span class="citation">(Boeva et al. <a href="#ref-Boeva2011">2011</a>)</span></td>
<td>poly model</td>
<td>track + exc</td>
<td>50 Kbp</td>
</tr>
<tr class="even">
<td>2011</td>
<td>BIC-seq <span class="citation">(Xi et al. <a href="#ref-Xi2011">2011</a>)</span></td>
<td></td>
<td></td>
<td><em>In theory</em>, so No</td>
</tr>
<tr class="odd">
<td>2011</td>
<td>Genome STRIP <span class="citation">(Handsaker et al. <a href="#ref-Handsaker2011">2011</a>)</span></td>
<td></td>
<td></td>
<td>Yes</td>
</tr>
<tr class="even">
<td>2011</td>
<td>CNVnator <span class="citation">(Abyzov et al. <a href="#ref-Abyzov2011">2011</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>VarScan 2 <span class="citation">(Koboldt et al. <a href="#ref-Koboldt2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>cn.MOPS <span class="citation">(Klambauer et al. <a href="#ref-Klambauer2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>CLEVER <span class="citation">(Marschall et al. <a href="#ref-Marschall2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>Splitread <span class="citation">(Karakoc et al. <a href="#ref-Karakoc2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>YAHA <span class="citation">(Faust and Hall <a href="#ref-Faust2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>SVM2 <span class="citation">(Chiara, Pesole, and Horner <a href="#ref-Chiara2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>GASVPro <span class="citation">(Sindi et al. <a href="#ref-Sindi2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2012</td>
<td>LUMPY <span class="citation">(Layer et al. <a href="#ref-Layer2014">2014</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2012</td>
<td>DELLY <span class="citation">(Rausch et al. <a href="#ref-Rausch2012">2012</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2013</td>
<td>WaveCNV <span class="citation">(Holt et al. <a href="#ref-Holt2013">2013</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2013</td>
<td>CNV-TV <span class="citation">(Duan et al. <a href="#ref-Duan2013a">2013</a>)</span></td>
<td>median</td>
<td>random mapping</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2013</td>
<td>PeSV-Fisher <span class="citation">(Escaraḿis et al. <a href="#ref-Escaramis2013">2013</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2013</td>
<td>CNVeM <span class="citation">(Wang et al. <a href="#ref-Wang2013">2013</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2013</td>
<td>CNVfinder <span class="citation">(McCallum and Wang <a href="#ref-McCallum2013">2013</a>)</span></td>
<td>model</td>
<td>model</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2013</td>
<td>GENSENG <span class="citation">(Szatkiewicz et al. <a href="#ref-Szatkiewicz2013">2013</a>)</span></td>
<td>model</td>
<td>model</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2014</td>
<td>CLImAT <span class="citation">(Yu et al. <a href="#ref-Yu2014">2014</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2014</td>
<td>Gustaf <span class="citation">(Trappe et al. <a href="#ref-Trappe2014">2014</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>2014</td>
<td>SMASH <span class="citation">(Talwalkar et al. <a href="#ref-Talwalkar2014">2014</a>)</span></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>2015</td>
<td>GROM-RD <span class="citation">(Smith, Kawash, and Grigoriev <a href="#ref-Smith2015">2015</a>)</span></td>
<td>quantile</td>
<td>quantile</td>
<td>Yes or no</td>
</tr>
</tbody>
</table>
</div>
<div id="misc" class="section level2">
<h2>Misc</h2>
<div id="mapping-algorithm-effect" class="section level3">
<h3>Mapping algorithm effect</h3>
<p>Except for assembly approaches, the choice of read mapping algorithm might affect the results.
Methods that manipulate discordant mapping in particular might gain the most at using a mapping strategy that fits best their approach.</p>
<p>For example in <span class="citation">Malhotra et al. (<a href="#ref-Malhotra2013">2013</a>)</span>, reads were realigned with <em>Novoalign</em> before investigating the discordant reads with <em>HYDRA-MULTI</em>.</p>
</div>
<div id="comparing-sv-calls" class="section level3">
<h3>Comparing SV calls</h3>
<p>GASV was developed to model uncertainty in the breakpoints <span class="citation">(Sindi et al. <a href="#ref-Sindi2009">2009</a>)</span>.
Each supporting read/probe/call is represented as a polygon defined by the start/end location and error (e.g. from insert size distribution).
Polygons are then intersected using an efficient plane sweep to identify subset of intersecting polygons, that are assumed to support the same variant.
Most of the paper describes GASV on raw data but they mention that it can be used to compare SV calls across studies.</p>
<p>VarSim first normalize VCFs to canonical forms before comparison <span class="citation">(Mu et al. <a href="#ref-Mu2015">2015</a>)</span>.
Each record is converted to a simple insertion or deletion followed by SNVs.
The insertion/deletion is either at the start/end of the variant, whatever results in the least number of mismatches.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/23369161">Wittler et al. (2013)</a> describe a clustering approach to better define overlapping deletions from paired-end reads.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25887570">Leung et al. (2015)</a> propose a benchmarking tool, SV-AUTOPILOT, that tests SV calling methods.
I should read how they overlap the SVs called and their truth sets.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5668921/">Sedlazeck et al. (2017)</a> describe SURVIVOR a tool to annotate and compare different SV callsets.
It builds on <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5286201/">SURVIVOR</a> to fine-tune breakpoints.</p>
<p>The Simpson lab has a <a href="http://simpsonlab.github.io/2015/06/15/merging-sv-calls/">blog post</a> on the topic of merging SV calls from different callers.
They used a window-based approach were calls at less than 300 bp from each other were merged.</p>
<p>A few methods use normalization approaches to transform variants in VCF into a simpler or more comparable variant.
Variant normalization is described in <a href="https://genome.sph.umich.edu/wiki/Variant_Normalization">this wiki</a>.
<a href="https://github.com/vcflib/vcflib#vcfallelicprimitives">vcflib</a> has a <code>vcfallelicprimitives</code> function.</p>
<p><a href="https://github.com/mkirsche/Jasmine">JASMINE</a> (Jointly Accurate Sv Merging with Intersample Network Edges) can merge SVs from different samples (or methods?) using a network approach.
In the network representation, SVs are merged using a minimum spanning forest algorithm (<em>not sure yet how, must watch the RECOMB talk</em>).
It offers multiple ways to specify distance thresholds such as absolute, relative to variant size, per variant in the VCF INFO field.
Insertions can also be considered based on their sequence identity.</p>
</div>
<div id="in-silico-validation" class="section level3">
<h3>In Silico validation</h3>
<p><strong>Reads realignment</strong> <em>targetSeqView</em> <span class="citation">(Halper-Stromberg et al. <a href="#ref-Halper-Stromberg2014">2014</a>)</span> was developed to visualize and assess the quality of structural variants, especially in regions with repeats.
The goal is to distinguish true SV from alignment artifacts.</p>
<p>The reads are re-aligned to the SV sequence and the two contiguous (reference) sequences.
The score represents how better the reads map to the SV sequence compared to the reference sequences.
It relies on a binomial distribution that takes into account the base position in the read.</p>
<p>Reads mapping to each sides of the SV junction are realigned to the 3 sequences.
The final score is the log-likelihood of the probability in the rearranged reference versus the ones in the contiguous sequences.</p>
<p>They focused their validated on the Ig and TCR loci in lymphoid cancer.
These regions have a segmentally duplicated architecture that are difficult to assay using NGS.</p>
<p>It’s a R package, and the re-alignment is performed using Smith-Waterman algorithm from the Biostrings package.
For their approach they recommend first using a NGS aligner that report multiple alignments per reads.
Other strategies might be better for RD analysis but reduce the pool of potential supporting reads mapping locally to the region of interest.</p>
<p>Some visualization tools described below can also assist in validating SVs.</p>
</div>
<div id="visualization" class="section level3">
<h3>Visualization</h3>
<p><a href="https://github.com/brwnj/covviz">covviz</a> visualizes coverage across multiple samples highlighting region where it diverges significantly from the other samples.
It’s an interactive app that seems to use Plotly.
It slick!
Additional QC features are offered like sex prediction, PCA and low/variable coverage metrics per sample.
Brent Pedersen contributed to it which might explain why it can make use of <em>indexcov</em> (fast coverage computation from indexes; part of <a href="https://github.com/brentp/goleft">goleft</a>.
Apparently distributed/implemented by <a href="https://base2genomics.com/">Base2Genomics</a>, a company which emphasizes looking at all variation in a genome including SVs and STRs.</p>
<p><a href="https://github.com/VCCRI/SVPV">SVPV</a> (Structural Variant Prediction Viewer) visualizes alignments of paired-end reads around input SVs.
Useful to inspect calls and compare multiple callsets with automated PDF creation.
It creates a multi-panel figure showing depth, insert-size clustered pairs, soft clip presence.</p>
</div>
</div>
</div>
<div id="long-read-sequencing" class="section level1">
<h1>Long-read sequencing</h1>
<p>pbmm2 + pbsv was used in <span class="citation">Hiatt et al. (<a href="#ref-Hiatt2020">2020</a>)</span>.
They also made <em>de novo</em> assemblies with hifiasm and “called” SVs from the minimap2 alignments.</p>
</div>
<div id="single-cell-sequencing" class="section level1">
<h1>Single-cell sequencing</h1>
<div id="from-scrna-seq" class="section level2">
<h2>From scRNA-seq</h2>
<p><a href="https://github.com/broadinstitute/inferCNV/wiki">InferCNV</a> compares expression of tumor cells across the genome to that of normal cells.
The algorithm is as follow:</p>
<ol style="list-style-type: decimal">
<li>genes expressed only in a few cells are removed</li>
<li>depth normalization per cell, adjusting the total to match the median total.</li>
<li>log(x+1) transformation</li>
<li>for each gene, subtract the average across normal cells</li>
<li>winsorize (e.g. force to be in [-3,3] range)</li>
<li>smoothing using a running average weighted using a pyramidal scheme (default 101 window).</li>
<li>center cell expression around 0</li>
<li>for each gene, subtract the average across normal cells (again)</li>
<li>revert log-transformation</li>
</ol>
<p>The CNVs are called at the cell level using HMM approaches.
Either a 3 states version (loss, neutral, gain) or a more granular 6 states one.
It’s interesting how they use synthetic data (from the normal cells) as a baseline, to calibrate the HMM results a posteriori.
Finally, the can annotate the CNV calls using a Bayesian Network (and MCMC), and remove calls with high posterior probability of being normal.</p>
<p>Additionally, the modified expression profiles can be further de-noised, for example to clean up the visualization.
One approach forced to 0 values matching the average + 3 SD of the normal cells.
The values can be further cleaned up using median filtering, i.e. a rolling median approach.</p>
<p>It looks like the subcluster identification is performed using hierarchical clustering.
They also mention that other part of the analysis can benefit from subcluster partition, e.g. running the HMM per cluster.
I thought the CNV calling was done at the cell level, I might be missing something.</p>
<hr />
<p><a href="https://github.com/JEFworks/HoneyBADGER">HoneyBadger</a> <strong>+++</strong></p>
</div>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Abyzov2011">
<p>Abyzov, Alexej, Alexander E. Urban, Michael Snyder, and Mark Gerstein. 2011. “CNVnator: An Approach to Discover, Genotype, and Characterize Typical and Atypical CNVs from Family and Population Genome Sequencing.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.114876.110">https://doi.org/10.1101/gr.114876.110</a>.</p>
</div>
<div id="ref-Alkan2009">
<p>Alkan, Can, Jeffrey M Kidd, Tomas Marques-Bonet, Gozde Aksay,..., and Evan E Eichler. 2009. “Personalized Copy Number and Segmental Duplication Maps Using Next-Generation Sequencing.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.437">https://doi.org/10.1038/ng.437</a>.</p>
</div>
<div id="ref-Amarasinghe2013">
<p>Amarasinghe, Kaushalya C, Jason Li, and Saman K Halgamuge. 2013. “CoNVEX: Copy Number Variation Estimation in Exome Sequencing Data Using HMM.” <em>BMC Bioinformatics</em>. <a href="https://doi.org/10.1186/1471-2105-14-S2-S2">https://doi.org/10.1186/1471-2105-14-S2-S2</a>.</p>
</div>
<div id="ref-Bao2014">
<p>Bao, Lei, Minya Pu, and Karen Messer. 2014. “AbsCN-Seq: A Statistical Method to Estimate Tumor Purity, Ploidy and Absolute Copy Numbers from Next Generation Sequencing Data.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btt759">https://doi.org/10.1093/bioinformatics/btt759</a>.</p>
</div>
<div id="ref-Boeva2012">
<p>Boeva, Valentina, Tatiana Popova, Kevin Bleakley, Pierre Chiche,..., and Emmanuel Barillot. 2012. “Control-FREEC: A Tool for Assessing Copy Number and Allelic Content Using Next-Generation Sequencing Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btr670">https://doi.org/10.1093/bioinformatics/btr670</a>.</p>
</div>
<div id="ref-Boeva2014">
<p>Boeva, Valentina, Tatiana Popova, Maxime Lienard, Sebastien Toffoli,..., and Jean-François Laes. 2014. “Multi-Factor Data Normalization Enables the Detection of Copy Number Aberrations in Amplicon Sequencing Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btu436">https://doi.org/10.1093/bioinformatics/btu436</a>.</p>
</div>
<div id="ref-Boeva2011">
<p>Boeva, Valentina, Andrei Zinovyev, Kevin Bleakley, Jean Philippe Vert,..., and Emmanuel Barillot. 2011. “Control-Free Calling of Copy Number Alterations in Deep-Sequencing Data Using GC-Content Normalization.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btq635">https://doi.org/10.1093/bioinformatics/btq635</a>.</p>
</div>
<div id="ref-Campbell2008">
<p>Campbell, Peter J, Philip J Stephens, Erin D Pleasance, Sarah O’Meara,..., and Andrew A Futreal. 2008. “Identification of Somatically Acquired Rearrangements in Cancer Using Genome-Wide Massively Parallel Paired-End Sequencing.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.128">https://doi.org/10.1038/ng.128</a>.</p>
</div>
<div id="ref-Chen2013">
<p>Chen, Gary K, Xiao Chang, Christina Curtis, and Kai Wang. 2013. “Precise Inference of Copy Number Alterations in Tumor Samples from SNP Arrays.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btt521">https://doi.org/10.1093/bioinformatics/btt521</a>.</p>
</div>
<div id="ref-Chen2014">
<p>Chen, Ken, Lei Chen, Xian Fan, John Wallis,..., and George Weinstock. 2014. “TIGRA: A Targeted Iterative Graph Routing Assembler for Breakpoint Assembly.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.162883.113">https://doi.org/10.1101/gr.162883.113</a>.</p>
</div>
<div id="ref-Chen2009">
<p>Chen, Ken, John W Wallis, Michael D McLellan, David E Larson,..., and Elaine R Mardis. 2009. “BreakDancer: An Algorithm for High-Resolution Mapping of Genomic Structural Variation.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1363">https://doi.org/10.1038/nmeth.1363</a>.</p>
</div>
<div id="ref-Chen2015">
<p>Chen, Xiang, Pankaj Gupta, Jianmin Wang, Joy Nakitandwe,..., and Jinghui Zhang. 2015. “CONSERTING: Integrating Copy-Number Analysis with Structural-Variation Detection.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.3394">https://doi.org/10.1038/nmeth.3394</a>.</p>
</div>
<div id="ref-Chiang2009">
<p>Chiang, Derek Y, Gad Getz, David B Jaffe, Michael J T O’Kelly,..., and Eric S Lander. 2009. “High-Resolution Mapping of Copy-Number Alterations with Massively Parallel Sequencing.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1276">https://doi.org/10.1038/nmeth.1276</a>.</p>
</div>
<div id="ref-Chiara2012">
<p>Chiara, Matteo, Graziano Pesole, and David S. Horner. 2012. “SVM2: An Improved Paired-End-Based Tool for the Detection of Small Genomic Structural Variations Using High-Throughput Single-Genome Resequencing Data.” <em>Nucleic Acids Research</em>. <a href="https://doi.org/10.1093/nar/gks606">https://doi.org/10.1093/nar/gks606</a>.</p>
</div>
<div id="ref-CommonMindConsortium2020">
<p>CommonMind Consortium, Lide Han, Xuefang Zhao, Mary Lauren Benton,..., and Douglas M. Ruderfer. 2020. “Functional Annotation of Rare Structural Variation in the Human Brain.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-020-16736-1">https://doi.org/10.1038/s41467-020-16736-1</a>.</p>
</div>
<div id="ref-Duan2013a">
<p>Duan, Junbo, Ji-Gang Zhang, Hong-Wen Deng, and Yu-Ping Wang. 2013. “Comparative Studies of Copy Number Variation Detection Methods for Next-Generation Sequencing Technologies.” <em>PloS One</em>. <a href="https://doi.org/10.1371/journal.pone.0059128">https://doi.org/10.1371/journal.pone.0059128</a>.</p>
</div>
<div id="ref-Eggertsson2019">
<p>Eggertsson, Hannes P., Snaedis Kristmundsdottir, Doruk Beyter, Hakon Jonsson,..., and Pall Melsted. 2019. “GraphTyper2 Enables Population-Scale Genotyping of Structural Variation Using Pangenome Graphs.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-019-13341-9">https://doi.org/10.1038/s41467-019-13341-9</a>.</p>
</div>
<div id="ref-Emde2012">
<p>Emde, Anne-Katrin, Marcel H. Schulz, David Weese, Ruping Sun,..., and Knut Reinert. 2012. “Detecting Genomic Indel Variants with Exact Breakpoints in Single- and Paired-End Sequencing Data Using SplazerS.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/bts019">https://doi.org/10.1093/bioinformatics/bts019</a>.</p>
</div>
<div id="ref-English2015">
<p>English, Adam C, William J Salerno, Oliver a Hampton, Claudia Gonzaga-Jauregui,..., and Richard a Gibbs. 2015. “Assessing Structural Variation in a Personal Genometowards a Human Reference Diploid Genome.” <em>BMC Genomics</em>. <a href="https://doi.org/10.1186/s12864-015-1479-3">https://doi.org/10.1186/s12864-015-1479-3</a>.</p>
</div>
<div id="ref-Escaramis2013">
<p>Escaraḿis, Geòrgia, Cristian Tornador, Laia Bassaganyas, Raquel Rabionet,..., and Xavier Estivill. 2013. “PeSV-Fisher: Identification of Somatic and Non-Somatic Structural Variants Using Next Generation Sequencing Data.” <em>PLoS ONE</em>. <a href="https://doi.org/10.1371/journal.pone.0063377">https://doi.org/10.1371/journal.pone.0063377</a>.</p>
</div>
<div id="ref-Faust2012">
<p>Faust, Gregory G., and Ira M. Hall. 2012. “YAHA: Fast and Flexible Long-Read Alignment with Optimal Breakpoint Detection.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/bts456">https://doi.org/10.1093/bioinformatics/bts456</a>.</p>
</div>
<div id="ref-Fromer2012">
<p>Fromer, Menachem, Jennifer L Moran, Kimberly Chambert, Eric Banks,..., and Shaun M Purcell. 2012. “Discovery and Statistical Genotyping of Copy-Number Variation from Whole-Exome Sequencing Depth.” <em>American Journal of Human Genetics</em>. <a href="https://doi.org/10.1016/j.ajhg.2012.08.005">https://doi.org/10.1016/j.ajhg.2012.08.005</a>.</p>
</div>
<div id="ref-Glusman2015">
<p>Glusman, Gustavo, Alissa Severson, Varsha Dhankani, Max Robinson,..., and Leroy Hood. 2015. “Identification of Copy Number Variants in Whole-Genome Data Using Reference Coverage Profiles.” <em>Frontiers in Genetics</em>. <a href="https://doi.org/10.3389/fgene.2015.00045">https://doi.org/10.3389/fgene.2015.00045</a>.</p>
</div>
<div id="ref-Gusnanto2012">
<p>Gusnanto, Arief, Henry M Wood, Yudi Pawitan, Pamela Rabbitts, and Stefano Berri. 2012. “Correcting for Cancer Genome Size and Tumour Cell Content Enables Better Estimation of Copy Number Alterations from Next-Generation Sequence Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btr593">https://doi.org/10.1093/bioinformatics/btr593</a>.</p>
</div>
<div id="ref-Hajirasouliha2010">
<p>Hajirasouliha, Iman, Fereydoun Hormozdiari, Can Alkan, Jeffrey M Kidd,..., and S Cenk Sahinalp. 2010. “Detection and Characterization of Novel Sequence Insertions Using Paired-End Next-Generation Sequencing.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btq152">https://doi.org/10.1093/bioinformatics/btq152</a>.</p>
</div>
<div id="ref-Halper-Stromberg2014">
<p>Halper-Stromberg, Eitan, Jared Steranka, Kathleen H. Burns, Sarven Sabunciyan, and Rafael A. Irizarry. 2014. “Visualization and Probability-Based Scoring of Structural Variants Within Repetitive Sequences.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btu054">https://doi.org/10.1093/bioinformatics/btu054</a>.</p>
</div>
<div id="ref-Handsaker2011">
<p>Handsaker, Robert E, Joshua M Korn, James Nemesh, and Steven A McCarroll. 2011. “Discovery and Genotyping of Genome Structural Polymorphism by Sequencing on a Population Scale.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.768">https://doi.org/10.1038/ng.768</a>.</p>
</div>
<div id="ref-Handsaker2015">
<p>Handsaker, Robert E, Vanessa Van Doren, Jennifer R Berman, Giulio Genovese,..., and Steven a McCarroll. 2015. “Large Multiallelic Copy Number Variations in Humans.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3200">https://doi.org/10.1038/ng.3200</a>.</p>
</div>
<div id="ref-He2011">
<p>He, Dan, Farhad Hormozdiari, Nicholas Furlotte, and Eleazar Eskin. 2011. “Efficient Algorithms for Tandem Copy Number Variation Reconstruction in Repeat-Rich Regions.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btr169">https://doi.org/10.1093/bioinformatics/btr169</a>.</p>
</div>
<div id="ref-Hiatt2020">
<p>Hiatt, Susan M., James M.J. Lawlor, Lori H. Handley, Ryne C. Ramaker,..., and Gregory M. Cooper. 2020. “Long-Read Genome Sequencing for the Diagnosis of Neurodevelopmental Disorders.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.07.02.185447">https://doi.org/10.1101/2020.07.02.185447</a>.</p>
</div>
<div id="ref-Holt2013">
<p>Holt, Carson, Bojan Losic, Deepa Pai, Zhen Zhao,..., and Lakshmi B Muthuswamy. 2013. “WaveCNV: Allele Specific Copy Number Alterations in Primary Tumors and Xenograft Models from Next-Generation Sequencing.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btt611">https://doi.org/10.1093/bioinformatics/btt611</a>.</p>
</div>
<div id="ref-Hormozdiari2010">
<p>Hormozdiari, Fereydoun, Iman Hajirasouliha, Phuong Dao, Faraz Hach,..., and S Cenk Sahinalp. 2010. “Next-Generation VariationHunter: Combinatorial Algorithms for Transposon Insertion Discovery.” <em>Bioinformatics</em>.</p>
</div>
<div id="ref-Hormozdiari2011a">
<p>Hormozdiari, Fereydoun, Iman Hajirasouliha, Andrew McPherson, Evan E. Eichler, and S. Cenk Sahinalp. 2011. “Simultaneous Structural Variation Discovery Among Multiple Paired-End Sequenced Genomes.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.120501.111">https://doi.org/10.1101/gr.120501.111</a>.</p>
</div>
<div id="ref-Iqbal2012">
<p>Iqbal, Zamin, Mario Caccamo, Isaac Turner, Paul Flicek, and Gil McVean. 2012. “De Novo Assembly and Genotyping of Variants Using Colored de Bruijn Graphs.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.1028">https://doi.org/10.1038/ng.1028</a>.</p>
</div>
<div id="ref-Ivakhno2010">
<p>Ivakhno, Sergii, Tom Royce, Anthony J. Cox, Dirk J. Evers,..., and Simon Tavaré. 2010. “CNAseg–a Novel Framework for Identification of Copy Number Changes in Cancer from Second-Generation Sequencing Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btq587">https://doi.org/10.1093/bioinformatics/btq587</a>.</p>
</div>
<div id="ref-Karakoc2012">
<p>Karakoc, Emre, Can Alkan, Brian J O’Roak, Megan Y Dennis,..., and Evan E Eichler. 2012. “Detection of Structural Variants and Indels Within Exome Data.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1810">https://doi.org/10.1038/nmeth.1810</a>.</p>
</div>
<div id="ref-Klambauer2012">
<p>Klambauer, Günter, Karin Schwarzbauer, Andreas Mayr, Djork Arné Clevert,..., and Sepp Hochreiter. 2012. “Cn.MOPS: Mixture of Poissons for Discovering Copy Number Variations in Next-Generation Sequencing Data with a Low False Discovery Rate.” <em>Nucleic Acids Research</em>. <a href="https://doi.org/10.1093/nar/gks003">https://doi.org/10.1093/nar/gks003</a>.</p>
</div>
<div id="ref-Koboldt2012">
<p>Koboldt, Daniel C., Qunyuan Zhang, David E. Larson, Dong Shen,..., and Richard K. Wilson. 2012. “VarScan 2: Somatic Mutation and Copy Number Alteration Discovery in Cancer by Exome Sequencing.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.129684.111">https://doi.org/10.1101/gr.129684.111</a>.</p>
</div>
<div id="ref-Korbel2007">
<p>Korbel, Jan O., Alexander Eckehart Urban, Jason P. Affourtit, Brian Godwin,..., and Michael Snyder. 2007. “Paired-End Mapping Reveals Extensive Structural Variation in the Human Genome.” <em>Science</em>. <a href="https://doi.org/10.1126/science.1149504">https://doi.org/10.1126/science.1149504</a>.</p>
</div>
<div id="ref-Krumm2012">
<p>Krumm, Niklas, Peter H. Sudmant, Arthur Ko, Brian J. O’Roak,..., and Evan E. Eichler. 2012. “Copy Number Variation Detection and Genotyping from Exome Sequence Data.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.138115.112">https://doi.org/10.1101/gr.138115.112</a>.</p>
</div>
<div id="ref-Layer2014">
<p>Layer, Ryan M, Colby Chiang, Aaron R Quinlan, and Ira M Hall. 2014. “LUMPY: A Probabilistic Framework for Structural Variant Discovery.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2014-15-6-r84">https://doi.org/10.1186/gb-2014-15-6-r84</a>.</p>
</div>
<div id="ref-Li2012">
<p>Li, Jason, Richard Lupat, Kaushalya C Amarasinghe, Ella R Thompson,..., and Kylie L Gorringe. 2012. “CONTRA: Copy Number Analysis for Targeted Resequencing.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/bts146">https://doi.org/10.1093/bioinformatics/bts146</a>.</p>
</div>
<div id="ref-Lindberg2014">
<p>Lindberg, Michael R, Ira M Hall, and Aaron R Quinlan. 2014. “Population-Based Structural Variation Discovery with Hydra-Multi.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btu771">https://doi.org/10.1093/bioinformatics/btu771</a>.</p>
</div>
<div id="ref-Magi2010">
<p>Magi, Alberto, Matteo Benelli, Giuseppina Marseglia, Genni Nannetti,..., and Francesca Torricelli. 2010. “A Shifting Level Model Algorithm That Identifies Aberrations in Array-CGH Data.” <em>Biostat</em>. <a href="https://doi.org/10.1093/biostatistics/kxp051">https://doi.org/10.1093/biostatistics/kxp051</a>.</p>
</div>
<div id="ref-Magi2011">
<p>Magi, Alberto, Matteo Benelli, Seungtai Yoon, Franco Roviello, and Francesca Torricelli. 2011. “Detecting Common Copy Number Variants in High-Throughput Sequencing Data by Using JointSLM Algorithm.” <em>Nucleic Acids Research</em>. <a href="https://doi.org/10.1093/nar/gkr068">https://doi.org/10.1093/nar/gkr068</a>.</p>
</div>
<div id="ref-Magi2013">
<p>Magi, Alberto, Lorenzo Tattini, Ingrid Cifola, Romina D’Aurizio,..., and Gian Franco Gensini. 2013. “EXCAVATOR: Detecting Copy Number Variants from Whole-Exome Sequencing Data.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2013-14-10-r120">https://doi.org/10.1186/gb-2013-14-10-r120</a>.</p>
</div>
<div id="ref-Malhotra2013">
<p>Malhotra, Ankit, Michael Lindberg, Gregory G Faust, Mitchell L Leibowitz,..., and Ira M Hall. 2013. “Breakpoint Profiling of 64 Cancer Genomes Reveals Numerous Complex Rearrangements Spawned by Homology-Independent Mechanisms.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.143677.112">https://doi.org/10.1101/gr.143677.112</a>.</p>
</div>
<div id="ref-Marschall2012">
<p>Marschall, Tobias, Ivan G Costa, Stefan Canzar, Markus Bauer,..., and Alexander Schönhuth. 2012. “CLEVER: Clique-Enumerating Variant Finder.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/bts566">https://doi.org/10.1093/bioinformatics/bts566</a>.</p>
</div>
<div id="ref-McCallum2013">
<p>McCallum, Kenneth Jordan, and Ji-Ping Wang. 2013. “Quantifying Copy Number Variations Using a Hidden Markov Model with Inhomogeneous Emission Distributions.” <em>Biostatistics (Oxford, England)</em>. <a href="https://doi.org/10.1093/biostatistics/kxt003">https://doi.org/10.1093/biostatistics/kxt003</a>.</p>
</div>
<div id="ref-Michaelson2012">
<p>Michaelson, Jacob J, and Jonathan Sebat. 2012. “forestSV: Structural Variant Discovery Through Statistical Learning.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.2085">https://doi.org/10.1038/nmeth.2085</a>.</p>
</div>
<div id="ref-Moncunill2014">
<p>Moncunill, Valent́i, Santi Gonzalez, Śilvia Beà, Lise O Andrieux,..., and David Torrents. 2014. “Comprehensive Characterization of Complex Structural Variations in Cancer by Directly Comparing Genome Sequence Reads.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/nbt.3027">https://doi.org/10.1038/nbt.3027</a>.</p>
</div>
<div id="ref-Mu2015">
<p>Mu, John C., Marghoob Mohiyuddin, Jian Li, Narges Bani Asadi,..., and Hugo Y.K. Lam. 2015. “VarSim: A High-Fidelity Simulation and Validation Framework for High-Throughput Genome Sequencing with Cancer Applications.” <a href="https://doi.org/10.1093/bioinformatics/btu828">https://doi.org/10.1093/bioinformatics/btu828</a>.</p>
</div>
<div id="ref-Nilsen2012">
<p>Nilsen, Gro, Knut Liestøl, Peter Van Loo, Hans Kristian Moen Vollan,..., and Ole Christian Lingjærde. 2012. “Copynumber: Efficient Algorithms for Single- and Multi-Track Copy Number Segmentation.” <em>BMC Genomics</em>. <a href="https://doi.org/10.1186/1471-2164-13-591">https://doi.org/10.1186/1471-2164-13-591</a>.</p>
</div>
<div id="ref-Popova2009">
<p>Popova, Tatiana, Elodie Manié, Dominique Stoppa-Lyonnet, Guillem Rigaill,..., and Marc Henri Stern. 2009. “Genome Alteration Print (GAP): A Tool to Visualize and Mine Complex Cancer Genomic Profiles Obtained by SNP Arrays.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2009-10-11-r128">https://doi.org/10.1186/gb-2009-10-11-r128</a>.</p>
</div>
<div id="ref-Pyon2011">
<p>Pyon, Yoon Soo, Matthew Hayes, and Jing Li. 2011. “Model Based Clustering Approach for Identifying Structural Variation Using Next Generation Sequencing Data.” In. <a href="https://doi.org/10.1109/ICCABS.2011.5729871">https://doi.org/10.1109/ICCABS.2011.5729871</a>.</p>
</div>
<div id="ref-Rausch2012">
<p>Rausch, Tobias, Thomas Zichner, Andreas Schlattl, Adrian M Stütz,..., and Jan O Korbel. 2012. “DELLY: Structural Variant Discovery by Integrated Paired-End and Split-Read Analysis.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/bts378">https://doi.org/10.1093/bioinformatics/bts378</a>.</p>
</div>
<div id="ref-Rimmer2014">
<p>Rimmer, Andy, Hang Phan, Iain Mathieson, Zamin Iqbal,..., and Gerton Lunter. 2014. “Integrating Mapping-, Assembly- and Haplotype-Based Approaches for Calling Variants in Clinical Sequencing Applications.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3036">https://doi.org/10.1038/ng.3036</a>.</p>
</div>
<div id="ref-Sathirapongsasuti2011">
<p>Sathirapongsasuti, Jarupon Fah, Hane Lee, Basil a J Horst, Georg Brunner,..., and Stanley F Nelson. 2011. “Exome Sequencing-Based Copy-Number Variation and Loss of Heterozygosity Detection: ExomeCNV.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btr462">https://doi.org/10.1093/bioinformatics/btr462</a>.</p>
</div>
<div id="ref-Scott2016">
<p>Scott, Emma C., Eugene J. Gardner, Ashiq Masood, Nelson T. Chuang,..., and Scott E. Devine. 2016. “A Hot L1 Retrotransposon Evades Somatic Repression and Initiates Human Colorectal Cancer.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.201814.115">https://doi.org/10.1101/gr.201814.115</a>.</p>
</div>
<div id="ref-Sepulveda2013">
<p>Sepúlveda, Nuno, Susana G. Campino, Samuel A. Assefa, Colin J. Sutherland,..., and Taane G. Clark. 2013. “A Poisson Hierarchical Modelling Approach to Detecting Copy Number Variation in Sequence Coverage Data.” <em>BMC Genomics</em>. <a href="https://doi.org/10.1186/1471-2164-14-128">https://doi.org/10.1186/1471-2164-14-128</a>.</p>
</div>
<div id="ref-Shi2013">
<p>Shi, Yuhao, and Jacek Majewski. 2013. “FishingCNV: A Graphical Software Package for Detecting Rare Copy Number Variations in Exome-Sequencing Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btt151">https://doi.org/10.1093/bioinformatics/btt151</a>.</p>
</div>
<div id="ref-Simpson2012">
<p>Simpson, Jared T., and Richard Durbin. 2012. “Efficient de Novo Assembly of Large Genomes Using Compressed Data Structures.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.126953.111">https://doi.org/10.1101/gr.126953.111</a>.</p>
</div>
<div id="ref-Sindi2009">
<p>Sindi, Suzanne, Elena Helman, Ali Bashir, and Benjamin J. Raphael. 2009. “A Geometric Approach for Classification and Comparison of Structural Variants.” <a href="https://doi.org/10.1093/bioinformatics/btp208">https://doi.org/10.1093/bioinformatics/btp208</a>.</p>
</div>
<div id="ref-Sindi2012">
<p>Sindi, Suzanne S, Selim Onal, Luke C Peng, Hsin-Ta Wu, and Benjamin J Raphael. 2012. “An Integrative Probabilistic Model for Identification of Structural Variation in Sequencing Data.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2012-13-3-r22">https://doi.org/10.1186/gb-2012-13-3-r22</a>.</p>
</div>
<div id="ref-Smith2015">
<p>Smith, Sean D., Joseph K. Kawash, and Andrey Grigoriev. 2015. “GROM-RD: Resolving Genomic Biases to Improve Read Depth Detection of Copy Number Variants.” <em>PeerJ</em>. <a href="https://doi.org/10.7717/peerj.836">https://doi.org/10.7717/peerj.836</a>.</p>
</div>
<div id="ref-Sudmant2015a">
<p>Sudmant, P. H., S. Mallick, B. J. Nelson, F. Hormozdiari,..., and E. E. Eichler. 2015. “Global Diversity, Population Stratification, and Selection of Human Copy Number Variation.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aab3761">https://doi.org/10.1126/science.aab3761</a>.</p>
</div>
<div id="ref-Szatkiewicz2013">
<p>Szatkiewicz, Jin P., Wei Waibo Wang, Patrick F. Sullivan, Wei Waibo Wang, and Wei Sun. 2013. “Improving Detection of Copy-Number Variation by Simultaneous Bias Correction and Read-Depth Segmentation.” <em>Nucleic Acids Research</em>. <a href="https://doi.org/10.1093/nar/gks1363">https://doi.org/10.1093/nar/gks1363</a>.</p>
</div>
<div id="ref-Talevich2014">
<p>Talevich, Eric, A Hunter Shain, Thomas Botton, and Boris C Bastian. 2014. “CNVkit: Copy Number Detection and Visualization for Targeted Sequencing Using Off-Target Reads.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/010876">https://doi.org/10.1101/010876</a>.</p>
</div>
<div id="ref-Talwalkar2014">
<p>Talwalkar, Ameet, Jesse Liptrap, Julie Newcomb, Christopher Hartl,..., and David Patterson. 2014. “SMaSH: A Benchmarking Toolkit for Human Genome Variant Calling.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btu345">https://doi.org/10.1093/bioinformatics/btu345</a>.</p>
</div>
<div id="ref-Trappe2014">
<p>Trappe, Kathrin, Anne-Katrin Emde, Hans-Christian Ehrlich, and Knut Reinert. 2014. “Gustaf: Detecting and Correctly Classifying SVs in the NGS Twilight Zone.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btu431">https://doi.org/10.1093/bioinformatics/btu431</a>.</p>
</div>
<div id="ref-Tubio2014">
<p>Tubio, Jose M.C. C., Yilong Li, Young Seok Ju, Inigo Martincorena,..., and Peter J. Campbell. 2014. “Extensive Transduction of Nonrepetitive DNA Mediated by L1 Retrotransposition in Cancer Genomes.” <em>Science</em>. <a href="https://doi.org/10.1126/science.1251343">https://doi.org/10.1126/science.1251343</a>.</p>
</div>
<div id="ref-Tuzun2005">
<p>Tuzun, Eray, Andrew J Sharp, Jeffrey a Bailey, Rajinder Kaul,..., and Evan E Eichler. 2005. “Fine-Scale Structural Variation of the Human Genome.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng1562">https://doi.org/10.1038/ng1562</a>.</p>
</div>
<div id="ref-VanLoo2010">
<p>Van Loo, P., S. H. Nordgard, O. C. Lingjaerde, H. G. Russnes,..., and V. N. Kristensen. 2010. “Allele-Specific Copy Number Analysis of Tumors.” <em>Proceedings of the National Academy of Sciences</em>. <a href="https://doi.org/10.1073/pnas.1009843107">https://doi.org/10.1073/pnas.1009843107</a>.</p>
</div>
<div id="ref-Venkatraman2007">
<p>Venkatraman, E. S., and Adam B. Olshen. 2007. “A Faster Circular Binary Segmentation Algorithm for the Analysis of Array CGH Data.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btl646">https://doi.org/10.1093/bioinformatics/btl646</a>.</p>
</div>
<div id="ref-Walker2013">
<p>Walker, Susan, and Stephen W Scherer. 2013. “Identification of Candidate Intergenic Risk Loci in Autism Spectrum Disorder.” <em>BMC Genomics</em>. <a href="https://doi.org/10.1186/1471-2164-14-499">https://doi.org/10.1186/1471-2164-14-499</a>.</p>
</div>
<div id="ref-Wang2014">
<p>Wang, Chen, Jared M Evans, Aditya V Bhagwate, Naresh Prodduturi,..., and Yan W Asmann. 2014. “PatternCNV: A Versatile Tool for Detecting Copy Number Changes from Exome Sequencing Data.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btu363">https://doi.org/10.1093/bioinformatics/btu363</a>.</p>
</div>
<div id="ref-Wang2011">
<p>Wang, Jianmin, Charles G Mullighan, John Easton, Stefan Roberts,..., and Jinghui Zhang. 2011. “CREST Maps Somatic Structural Variation in Cancer Genomes with Base-Pair Resolution.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1628">https://doi.org/10.1038/nmeth.1628</a>.</p>
</div>
<div id="ref-Wang2013">
<p>Wang, Zhanyong, Farhad Hormozdiari, Wen-Yun Yang, Eran Halperin, and Eleazar Eskin. 2013. “CNVeM: Copy Number Variation Detection Using Uncertainty of Read Mapping.” <em>Journal of Computational Biology : A Journal of Computational Molecular Cell Biology</em>. <a href="https://doi.org/10.1089/cmb.2012.0258">https://doi.org/10.1089/cmb.2012.0258</a>.</p>
</div>
<div id="ref-Wijaya2014">
<p>Wijaya, Edward, Kana Shimizu, Kiyoshi Asai, and Michiaki Hamada. 2014. “Reference-Free Prediction of Rearrangement Breakpoint Reads.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btu360">https://doi.org/10.1093/bioinformatics/btu360</a>.</p>
</div>
<div id="ref-Wong2010">
<p>Wong, Kim, Thomas M Keane, James Stalker, and David J Adams. 2010. “Enhanced Structural Variant and Breakpoint Detection Using SVMerge by Integration of Multiple Detection Methods and Local Assembly.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2010-11-12-r128">https://doi.org/10.1186/gb-2010-11-12-r128</a>.</p>
</div>
<div id="ref-Xi2011">
<p>Xi, Ruibin, Angela G Hadjipanayis, Lovelace J Luquette, Tae-Min Kim,..., and Peter J Park. 2011. “Copy Number Variation Detection in Whole-Genome Sequencing Data Using the Bayesian Information Criterion.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1110574108">https://doi.org/10.1073/pnas.1110574108</a>.</p>
</div>
<div id="ref-Xie2009">
<p>Xie, Chao, and Martti T Tammi. 2009. “CNV-Seq, a New Method to Detect Copy Number Variation Using High-Throughput Sequencing.” <em>BMC Bioinformatics</em>. <a href="https://doi.org/10.1186/1471-2105-10-80">https://doi.org/10.1186/1471-2105-10-80</a>.</p>
</div>
<div id="ref-Yang2013">
<p>Yang, Lixing, Lovelace J Luquette, Nils Gehlenborg, Ruibin Xi,..., and Peter J Park. 2013. “Diverse Mechanisms of Somatic Structural Variations in Human Cancer Genomes.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2013.04.010">https://doi.org/10.1016/j.cell.2013.04.010</a>.</p>
</div>
<div id="ref-Ye2009">
<p>Ye, Kai, Marcel H. Schulz, Quan Long, Rolf Apweiler, and Zemin Ning. 2009. “Pindel: A Pattern Growth Approach to Detect Break Points of Large Deletions and Medium Sized Insertions from Paired-End Short Reads.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btp394">https://doi.org/10.1093/bioinformatics/btp394</a>.</p>
</div>
<div id="ref-Yoon2009">
<p>Yoon, Seungtai, Zhenyu Xuan, Vladimir Makarov, Kenny Ye, and Jonathan Sebat. 2009. “Sensitive and Accurate Detection of Copy Number Variants Using Read Depth of Coverage.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.092981.109">https://doi.org/10.1101/gr.092981.109</a>.</p>
</div>
<div id="ref-Yu2014">
<p>Yu, Zhenhua, Yuanning Liu, Yi Shen, Minghui Wang, and Ao Li. 2014. “CLImAT: Accurate Detection of Copy Number Alteration and Loss of Heterozygosity in Impure and Aneuploid Tumor Samples Using Whole-Genome Sequencing Data.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btu346">https://doi.org/10.1093/bioinformatics/btu346</a>.</p>
</div>
<div id="ref-Zhu2012">
<p>Zhu, Mingfu, Anna C. Need, Yujun Han, Dongliang Ge,..., and David B. Goldstein. 2012. “Using ERDS to Infer Copy-Number Variants in High-Coverage Genomes.” <em>American Journal of Human Genetics</em>. <a href="https://doi.org/10.1016/j.ajhg.2012.07.004">https://doi.org/10.1016/j.ajhg.2012.07.004</a>.</p>
</div>
</div>
</div>
