---
sidebar: true
title: Structural Variation Impact
bibliography: [../../library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---

# Gene expression

Coding regions of **800-1800 known genes** are subject to **variation in copy number** [@Sharp2006].
On average, *394 genes* are predicted to differ in term of copy number *between two individuals* [@Alkan2009].
Different types of genes tend to be affected by common CNVs.
Genes involved in *immune response* or *environmental interaction* in general tend to be located in region of active CNV.
CNV seem to have played an important role in evolutionary adaptation [@Sharp2006].
@Lee2010 note that CNVs are particularly frequent in genes with a role at our interface with environment.
Moreover, these CNVs tend to be multi-allelic and in general more complex.

**SV can influence gene expression in several ways**.
Variation in the copy number of the entire gene can lead to **gene dosage** effect although genes involved in common CNVs are assumed to be tolerant to it.
Most of the segmental duplications containing genes are tandem changes [@Alkan2009].
SV on part of a gene may create **new transcript variant** through *exon shuffling*, or even *gene fusion*.
Variation in non-coding regions can affect **regulatory regions or chromatin structure**.
Copy number variants can affect the expression of genes **up to 100kb** away from the variant [@Raphael2012].
Finally, deletions can activate recessive allele through loss of heterozygosity.

Variation in non-coding regions can also affect regulatory regions or chromatin structure around genes.
For examples, non-coding CNVs have been associated with gene expression [@Stranger2007a] and can sometimes affect the expression of genes up to 100kb away from the variant [@Lower2009].
In @Stranger2007a, *53%* of the CNVs influencing gene expression were located outside of the gene.
When @Sudmant2015a described the SVs in the 1000GP, they called eQTLs in the lymphoblastoid cell lines from the Geuvadis project and found 54 eQTLs with a SV association (0.56%) and 166 SVs in LD with SNV/indels eQTLs.
Most of these SV-eQTLs overlapped coding sequence but some were located in upstream regions.
A more recent study using deep WGS investigated SV-eQTLs in multiple tissues from the GTEx dataset [@Chiang2017] (see [more detailed review post](/2018/03/31/SVeQTLinGTEx/)).
They found that 3.5-6.8% of eQTLs could be attributed to SVs and that SVs tend to have a larger effect size.
Most causal SVs were **non-coding** (89%) and were **enriched** in or close to **regulatory regions** (e.g. exons, TSS, TFBS, enhancers, gene 3' end).
They also detected an enrichment of rare SVs close to genes with outlier expressions, suggesting that rare SVs could have a high-impact on gene expression.
Indeed, rare SVs were enriched close to gene expression outliers, especially large CNVs, duplications, and variants altering gene dosage.

Multi-allelic CNVs are more likely involved with gene dosage changes than one-copy CNVs [@Handsaker2015].
There, 88% of gene dosage changes were caused by common multi-allelic CNVs.
Gene dosage changes from these CNVs correlates with gene expression.

In @CommonMindConsortium2020, WGS and RNA-seq was leveraged to explore the effect of SVs on gene expression and devise a prediction approach to annotate rare CNVs.
They observed mostly consistent effect on the gene expression for genic deletions and duplications or their promoters.
They noted that many complete gene deletions/duplications are not expression outliers though.
They built a disruption score that is better at ranking pathogenic CNVs than the traditional size+frequency considerations.
Also pathogenic deletions and duplications associated with NDDs have higher "disruption score".


## Micro RNAs
CNVs can also affect gene expression by **disrupting miRNAs dosage**.
These non-coding small RNAs often regulate genes and sometimes entire pathways expression.
There doesn't seem to be convincing studies showing the direct effect of CNVs on miRNAs on gene expression (on disease).
One review just cited distinct studies showing miRNAs importance, CNVs importance, potential impact on brain disorders (also broad unsurprising evolutionary facts) [@Persengiev2013].

One reference from Volinia 2010 apparently confirms the role of CNV-causing dysregulation of miRNAs in disease occurrence.
**To read**

## Genes with recurrent CNV

Many genes are recurrently affected by copy number variation in normal genomes.
@Lee2010 note that CNVs are particularly frequent in genes with a role at our interface with environment.
Moreover, these CNVs tend to be multi-allelic and in general more complex.
These genes must be haplosufficient (right?).
For example, @Alkan2009 genes with full CNV :  complement factor H-related complex (CFHR1-4), defensin gene family (DEFB103B), amylase gene family (AMY1A), as well as CCL3L1 and TBC1D3.

CNVs in SD are enriched for genes  involved in **sensory response, immune and inflammatory response, cell signaling and cell adhesion and structural proteins** [@Cooper2007].

4867 genes had an exon impacted by SVs, when comparing the two human genome assemblies [@Pang2010].
Similarly, 5320 protein-coding genes overlapped a SV from the 1KGP overlapped *(manually computed from DGV variants and Gencode v19)* [@Mills2011].
Only 2229 protein-coding genes had an exon overlapping one of Genome STRiP 2.0 calls in more than 800 samples *(manually computed from their calls and Gencode v19)* [@Handsaker2015].

## Enhancer hijacking

**Enhancer hijacking** happens when a regulatory region induces the **ectopic expression** of a gene it normally doesn't regulate because of a **CNV-mediated re-positioning**.
An elegant study described this mechanism in *human limb malformations* using CRISPR/Cas genome editing and conformation experiments [@Lupianez2015].
In brief, three different phenotypes were caused by **deletions**, **inversion**/tandem duplication, or **tandem duplications**, that **spanned the boundary of a topologically associated domains (TADs)**.
After showing that the TAD architecture was conserved in mice, these SVs were recreated in mice using genome editing. 
The ectopic expression of genes during limb development in mice was assessed by RNA-seq and in situ hybridization.
Using 4C-seq again, the *ectopic interaction* were also visible in patient-derived fibroblasts.
Investigating deeper the minimal region, genomic annotations and additional mice experiments helped identify 5 candidate enhancers potentially responsible for the normal and ectopic regulation. 
Finally, further genome editing confirmed that the **breaking of the TAD boundary** (and insulation region) **was the crucial factor** rather than the distance between enhancers and genes.

@CommonMindConsortium2020 showed that SV overlapping enhancers was selected against.
Same for SVs overlapping CTCF sites.
The enhancer annotation was also significantly associated with their gene expression z-scores, although mostly for duplications.

## Methylation trigerring

**Transposon insertion can trigger silencing through methylation**.
One documented case was found by [Lippman et al. 2004](https://pubmed.ncbi.nlm.nih.gov/15269773/) who showed in Arabidopsis that the gene for flowering time (FWA) was regulated to some extent by the methylation of nearby SINE-like TEs.
Another older observation in mice found that de novo retroviral insertion inactivates genes through de novo methylation of surrounding regions ([Jahner & Jaenisch 1985](https://pubmed.ncbi.nlm.nih.gov/2989695/).

Focusing on evolutionary patterns, a study in Arabidopsis showed that fixed/old insertions tend to "lose" their methylation but polymorphic insertions don't ([Hollister & Gaut 2009](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2720190/)).
Their results suggest negative selection against methylated insertions, potentially because of their effect on nearby genes.

@Rebollo2011 compared H3K9me3 profiles of embryonic cells in two strains of mice, focusing around common or polymorphic copies of threes TE families.
In contrast to LINE or ETn/MusD, IAP copies tended to be more hypomethylated. 
For polymorphic IAP copies, only the strain with the copy was hypomethylated. 
**The methylation decreased almost linearly up to 3 kbp from the insertion.**
They found that it **rarely spread into genes**, hinting at some mechanisms that could stop methylation spreading.
Although the genome-wide analysis was performed on H3K9me3, they zoom in one insertion close to the B3galtl gene.
This time looking at DNA methylation, they show **a clear example** where the empty sites is completely unmethylated while the strains with the inserted IAP show methylation spreading to the CpG island about half the time.
**The expression of this gene was indeed reduced** in the strain with the IAP insertion.

Studying this in mice, @Rebollo2012 showed **active genes tend to be protected and that methylation spreading from a nearby TE is rare**.
They measured methylation at fixed endogenous retrovirus (ERV) copies as well as a few insertionally polymorphic copies.
Overall they found that ERV copies near the TSS are more likely hypomethylated.
They further interrogated the methylation at the gene promoter near methylated ERV copies to test if the methylation had spread.
Only 1 out of 29 cases had a sign of methylation spreading (and they knew about it from [a previous study](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002301)).
They observe a similar pattern of "methylation vs TSS distance" between fixed and polymorphic copies.
Part of the explanation lies in the action of H3K4me3 and CTCF, blocking the methylation spreading.

In a recent preprint, [Wyler et al 2020](https://www.biorxiv.org/content/10.1101/2020.06.16.154047v2) tested this model in *Brachypodium distachyon* ([grass](https://en.wikipedia.org/wiki/Brachypodium_distachyon)).
They used methylation and transcriptome profiling across 11 accessions.
They found that methylation in only a few TE families leads to lower gene expression.
However, they did find a few polymorphic insertions associated with gene expression changes across multiple accessions.
Take-home message seems to be: **not a large effect globally but some families/insertions can silence nearby genes**.


# Large chromosomal aberrations

Early milestones listed in @Lee2010:

- 1959 Down syndrome/trisomy 21, Turner syndrome (45,X), Kleinfelter syndrome (47, XXY)
- 1961 Partial trisomy Down syndrome
- 1963 First inherited deletion syndrome: Cri-du-chat (5p-)

Large CNVs that span multiple genes can cause 'contiguous gene syndromes' or genomic disorders ("old" ref in @Lee2010)

# Rare SVs

One of the first duplication associated with a disease was found by Lupski and colleagues [@Baker2012].
They found that hereditary neuropathy Charcot-Marie-Tooth disease was associated with the presence of an extra copy of a particular genetic region.
Their paper was rejected from *Nature* and *Science* right away.
The deletion of the same locus causes hereditary neuropathy with liability to pressure palsies (HNPP).
Both are mediated by NAHR.

Williams-Beuren Syndrome is a mendelian disorder characterized by deletion at 7q11.23.
The syndrome is remarkably consistent with the presence of these micro-deletions [@Lee2010].
Segmental duplications are mediating the deletion, as well as a relatively common inversion (see inversion section).

DiGeorge/velocardiofacial syndrom is characterized by deletion of 22q11.2.

Pinto et al. (Nature 2010) found a higher burden of rare genic CNVs (1.19x more >30kb CNV with AF<1%) in 996 ASD vs 1,287 controls using dense genotyping arrays.
The enrichment was stronger for in loci implicated with ASD in the past (1.69x).
Novel ASD genes were identified, e.g. SHANK2 (SHANK3 was a known ASD gene already), by zooming in to the *de novo* CNVs.
Similar enrichmnent has been observed in schizophrenia and bipolar disorder.
 
| Disease                       | Gene    | SV  | Size           | Description           | Ref               |
|-------------------------------|---------|-----|----------------|-----------------------|-------------------|
| ASD                           | SHANK2  | DEL | 66 kbp         | de novo exonic        | Pinto Nature 2010 |
| Early-onset Alzheimer disease | APP     | DUP | 400 kb, 3.5 Mb | novel large rare CNVs |                   |
| Breast cancer                 | BRCA1/2 | MEI | 300 bp         | Alu                   |                   |
| Parkinson disease             | SNCA    | DUP |                |                       |                   |

# Association studies and LD

| Disease                                              | Gene        | SV                  | Size   | Description                                              | Ref                         |
|------------------------------------------------------|-------------|---------------------|--------|----------------------------------------------------------|-----------------------------|
| Crohn's Disease                                      | IRGM        | DEL                 | 20.1kb | upstream, known because perfect LD with associated SNP   | Wellcome Trust Nature 2010  |
| Crohn's Disease                                      | IRGM        | DEL                 | 3.9kb  | upstream, not in LD with associated SNP                  | Wellcome Trust Nature 2010  |
| Crohn's Disease                                      | DEFB4       |                     |        |                                                          |                             |
| Type 2 Diabetes                                      | TSPAN8      | inverted DUP + DEL? | 1kb    | partly exonic, gene known but CNV not in LD with SNP hit | Wellcome Trust Nature 2010  |
| Psoriasis                                            | LCE3B LCE3C | DEL                 | 32.2kb | full gene(s) loss                                        | de Cid Nature Genetics 2009 |
| lower LDL cholesterol levels                         | PCSK9       | DEL                 | 14kb   | exon 1                                                   | @Beyter2019                 |
| decreased serum albumin level, increased cholesterol | ALB         | DEL                 | 4kb    | promoter                                                 | @Chen2020a                  |
| Alzheimer, parkinson                                 | MAPT        | INV                 | 900kb  | 17q21.31                                                 |                             |
| obesity, autism,                                     |             | INV                 | 4.5Mb  | 8p23.1                                                   |                             |

+ How is association tested ? How many such studies have been performed ?

One approach to test potential effect of common SVs with phenotypes uses **SNP in strong LD as markers**.
Although it seemed at first glance that genotyped SNPs were in sufficient LD with many SVs, these analysis were **biased toward a specific type of SVs**.
Unfortunately, these SVs are **not representative** of the SV landscape and actually represent the least frequent class of SVs [@Sharp2006].
For practical and interpretation purpose, these studies *focused on simple deletions*, avoiding sites of complex, multi-copy or tandem rearrangements.
Conversely, some SVs are found to be recurrent by being present in different haplotype regions, or resulting in both duplication and deletions, or displaying different breakpoints.

As reviewed in @Ionita-Laza2009, challenges in CNV association studies range from the **signal-to-noise ratio**, the difficulty to derive **precise genotype**.
Similar to other variants, other challenges comprise the sample size and power computation, multiple test correction, reproducibility, population stratification.

Two strategies have been used:

1. Inferring the **genotype** and performing **standard tests**.
1. Directly **analyzing raw measurements**.

The former can result in a loss of information and power when the genotyping process is not exact.
The latter has more power but its results will need to be validated and investigated through genotyping.

@Ionita-Laza2009 cites associations between **common CNVs and risk for HIV infections, autoimmune disease and asthma**. 

**Disease-related CNVs are not restricted to rare variants** and the *genetic risk tend to be higher than for SNPs* [@Ionita-Laza2009].

**Genetic pleiotropy** might exists [@Ionita-Laza2009].
For example, gains and losses in *CCL3L1* are associated with HIV or rheumatoid arthritis. 
Hence, what is protective for a trait (*CCR5* deletion/*CCL3L1* duplication for HIV/AIDS) can predispose for other (rheumatoid arthritis).
Similar for *DEFB4* and Crohn disease and Psoriasis.

CNVs have been associated with autism, schizophrenia, Crohn's disease, rheumatoid arthritis, type 1 diabetes, obesity and numerous developmental diseases [@Zarrei2015].

Deletion of haplo-insufficient genes can lead to disease.
For example, deletion of *LCE3B* and *LCE3C* is a risk factor for psoriasis [@Lee2010].

@Beyter2019 discovered ~70K SVs using long-read sequencing in almost 2K Icelanders and imputed the presence of ~50K in 166K Icelanders from SNP arrays.
They performed association tests with the phenotype available and notably found a 14 Kbp deletion that overlaps the first exon of PCSK9 and correlates with lower LDL cholesterol levels.

@Chen2020a performed a GWAS on 65K SVs detected with 20x WGS vs 116 traits in 4-5K Finns.
They identified 31 hits in 15 loci, including 2 novel loci (a promoter deletion, and a multi-allelic CNV).
5 out of the 6 replicated hits have stronger SNV association though.

## Inversions

From @Gonzalez2019 introduction:

- A common inversion in 8p23.1 has been linked to obesity, autism, neuroticism.
- Inversion at 17q21.31 was associated with Alzheimer, Parkinson diseases, heart failure.

@Gonzalez2019 imputed the presence of inversions in >400K Europeans individuals and performed association tests agains 25 obesity-related traits.
Seven inversions were associated with multiple diseases.
Some inversions could be disrupting TADs.

An inversion in a parental chromosome may predispose to de novo CNVs [@Lee2010].
For example the deletion in 17q21.31 results from an inversion common in Europeans.
Other examples include Sotos syndrome or Williams-Beuren syndrome.
These results suggest that parents are sometimes needed to fully understand the origin of CNVs.




# Cancer

+ How are SV/CNVs used to estimate cell heterogeneity ? How is this validated ?
+ Can we infer tumor evolution ? Or pooling samples, recurrent path in tumor evolution ?
+ How clear are the recurrence patterns usually ? Is it consistent across types ? Are the different breakpoints a real issue ?
+ How is recurrence defined in samples/cancer with many large abberations ?
+ How is DNA satellite expansion associated with different types of cancer ?
+ What is the importance of centromeric/telomeric regions in cancer ?
+ What types of transposable elements get activated in cancer ?
+ How is SV involved in epigenetics changes in the tumoral genome ?

SVs can affect gene expression through genetic and epigenetics mechanisms [@Albertson2003].

Genomic gains are particularly important to identify driver genes [@Dear2009].
Moreover, duplication have been less studied than deletions.

@De2011a found an enrichment of somatic CNV breakpoints in simple repeats, Alu and CR1.

Replication timing has also been associated with sCNVs formation [@De2011].

**Satellite DNA "Breaking of satellites' silence" is a paradigm in cancerogenesis** [@Rich2014].
In cancer satellites are hypomethylated and are thought to induce peri-centromeric instability.
This might explain the observed over-expression of satellite repeats in tumors.

For example, @Ting2011 observed a 40-fold increase in pericentromeric satellites expression in pancreatic ductal adenocarcinomas.
Alpha satellites were the most abundant, HSATII highly specific to cancer.
The satellite expression also correlated with L1 expression.
The hypothesis is an alteration of the silencing mechanisms of these repeats.


## Pan-Cancer patterns

### Arm-level aberrations

Copy number aberration is common in tumors from almost all cancer types. 
Tumors sometimes harbor whole-genome doubling or, more frequently, chromosome arm-level gains or losses.
Aneuploidy, i.e. chromosome gain or loss, is seen in almost all cancer types although at varying frequencies [@Baudis2007;@Kim2013b;@Zack2013].

Aggregating CGH data across 5,918 epithelial tumors, @Baudis2007 identified frequent arm-level gains in 8q, 20q, 1q, 3q, 5p, 7q and 17q; and frequent losses in 3p, 4q, 13q, 17p and 18q.
The median number of arm-level aberration per sample ranged from 0 in squamous skin neoplasias to 12 in small cell lung carcinomas [@Baudis2007].

To further interrogate the distribution of sCNVs in different types of cancer, @Beroukhim2010 analyzed 3,131 cancer genomes from 26 different cancer types.
Although the frequency of sCNV decreases with their size, arm-level aberrations stood out as recurrent aberrations for both losses and gains and across all 26 cancer types studied
Arm-level aberrations were ~30 times more frequent than expected from the frequency-size relationship of other sCNVs.
In this pan-cancer survey, 25\% of a typical cancer genome was affected by arm-level sCNVs.
In contrast to some focal sCNVs, arm-level sCNVs resulted in low-amplitude changes, most of the time a single copy loss/gain.
Interestingly, it appeared that chromosome arms had either somatic losses or gains, but rarely both.

Similar observations were made in a larger meta-analysis of 8,227 cancer CNV profiles from 107 CGH studies [@Kim2013b].
Chromosome arms preferentially showed losses or gains, with 1q, 5p, 7p, 3q and 20q most frequently gained and 4q, 6q, 8p, 13q and 17p most frequently lost.
Cancer types were clustered based on their arm-level aberration frequencies and formed three groups with similar developmental origins.
Five cancer of hematologic-origin clustered together; most of the  epithelial cancers formed the second cluster; the third cluster contained four neuroepithelial cancers as well as sarcoma and RCC.
Arm-level aberrations in gene-rich arms tend to co-occur when concordant (gain-gain, loss-loss) [@Kim2013b].

Using a single detection platform across 4,934 cancers and 11 cancer types, The Cancer Genome Atlas described attempted to time sCNVs [@Zack2013].
For example, whole-genome doubling was observed in 37\% of the tumors and tended to occur prior to other types of sCNVs.
The rate of both arm-level and focal sCNVs were higher in tumor that experienced whole-genome doubling.x
In term of arm-level aberrations, @Zack2013 found a median of 3 gains and 5 losses per tumors.

The pan-cancer studies described above didn't include or comment on sex chromosomes in their analysis [@Baudis2007;@Beroukhim2010;@Kim2013b;@Zack2013].

### Focal sCNVs

@Li2012a investigated the association of **19 genomic features with the frequency of somatic CNV breakpoints** (SNP-array, 8 cancer types).
**Evolutionary indel** and **substitution rates** and **conserved elements** showed the strongest correlation.
Moreover they see *two classes* of breakpoint hotspots: the *cancer-type-specific* weakly correlated with genomic features; the *common* hotspots with higher correlation.
They suggest that they follow **different evolutionary models**: positive selection driven by cancer genes and non-adaptive evolution related to genomic context.
So the cancer-specific hotspots are driven by cancer genes (sometimes specific to the cancer type); but common hotspots are more often due to the region being more fragile and susceptible to sCNVs.
They used sequence features, DNA secondary structure motif, evolutionary features and functional annotation.
Cancer-type specific hotspots were enriched in cancer genes.
In their figure 2A we see interesting patterns.
First, some strong association are type-specific, e.g. SINE, exon density, PG4, CpG islands.
Then, *some cancers show opposite patterns*: SINE are enriched in sCNVs in ovarian cancer but depleted in kidney cancer.

Several studies observed **shared sCNV hotspot** location across **different cancer types** [@Li2012a;@Zack2013].
There are still hotspots specific to the different cancer types.

**Recurrent focal CNVs** have been used previously to **identify tumor-suppressor genes and oncogenes** [@Zack2013].
The large sample size and diverse origin of the samples in the TCGA/ICGC consortium lead to the identification of common or cancer-specific hotspots of sCNVs [@Beroukhim2010;@Zack2013].
For example, @Zack2013 identified *140 regions with recurrent sCNVs* when pooling cancers from 11 different types.
sCNVs in those regions tended to be smaller and with higher amplitude.
They further identified *229 regions with cancer-type-specific recurrence* that were not significant in the pan-cancer analysis.

Large-sample size also allows researchers to look for **anti-correlation patterns between pairs of regions**.
*Positive* correlations with other genetic events may indicate functional *synergies*, whereas *anticorrelations* may indicate functional *redundancies* [@Zack2013].
For example, @Zack2013 found 7% of regions pairs to be anti-correlated.
In practice, they tested pairs of hotspot regions.
These pairs tend to physically interact and are suspected to have related functions.

**Ploidy and purity varies within and across cancer lineages** [@Zack2013].
Cancers with whole-genome duplication have higher rate of every other types of sCNVs, and these sCNVs are predicted to occur after the WGD.

@Zhang2016 used the same data and further characterized the genomic features explaining the location of sCNV breakpoints.
Briefly they counted the number of sCNA breakpoints in 1 Mb bins and regressed it against genomic features.
Their models completed previous studies and found new associations, the strongest being the **distance to telomere and centromere**.
Frequency of sCNAs decreased with the distance to telomere/centromere.
Among the positive significant associations: **L1 coverage, SINE count, low-complexity repeat coverage, CpG island**.
A more surprising one is the **number of conserved elements** (*they don't discuss it so I'm not sure what to think of it*).
**Direct repeats**, which are a class of non-B conformation, were also positively associated and mentioned several times.
They decided to **separate telomere-bound and interstitial sCNVs** and show different breakpoint locations.


## Gene fusion

A **translocation** is causing a gene fusion between BCR and ABL genes and is seen in a important fraction of patients with chronic myelogenous leukemia.
BCR-ABL is directly implicated in the development of the cancer [@Raphael2012].
**Fusion genes are also recurrent in some solid tumors**, such as prostate and lung cancers [@Raphael2012].
Another translocation in Burkitt's lymphoma, activates MYC oncogene by **fusing a promoter** [@Raphael2012].

## Enhancer hijacking

There have been several examples of **somatic CNV** that change the structure of the **regulatory region** and induce **ectopic activity of oncogenes**.

@Weischenfeldt2016 developed a method to detect **associations between sCNA breakpoints in a TAD and gene over-expression**.
TADs (Topologically Associated Domains) are 3D domains (mean size 830 kb) that are supposed to be conserved across cell types and confine regulatory elements and targets.
They analyzed the 7K cancer genomes (26 subtypes) with SNP array and RNA-seq data in TCGA.
They first described a known cancer gene, TERT, which had been already found to be upregulated by such mechanisms.
Interestingly **both deletion and duplication** were involved in over-expression.
Then they describe and validate in much details to genes, IRS4 and IGF2.
In the first, **deletions cluster right next to a regulatory region of IRS4**.
In tumors with these deletions, the region was now *active* (H3K27ac).
They also showed that IRS4 is *likely a cancer gene* by over-expressing it and observing rapid tumor growth in mice.
In the second, **single tandem duplications of IGF2 and a super-enhancer** creates a **novel chromatin domain** with both IGF2 and the super-enhancer.
Again, they nicely validated that the region was active and touching (4C-seq) in tumor with the duplication.
Here they didn't look at enhancer amplification or balanced translocation (soon with PCAWG).

These enhancer hijacking events are important because **a single copy change usually can lead to large over-expression**.
In contrast dosage effect due to full-gene CNV are (supposedly) as strong as the amplification.

## Relation with epigenetic marks

Somatic SVs can be associated with epigenetic changes and vice versa.
On one hand, we would like to know if the epigenetic state of a genome affects the formation of somatic SVs.

@De2011a found that **G-quaduplex(G4) sequences are enriched in somatic CNV breakpoints** (SNP-array, 26 cancer types).
Moreover, these many sCNVs breakpoints shows abnormal hypomethylation of the G4 sequence.
They found evidence of G4-mediated SV and proposed that G4 hypomethylation could drive tissue-specific mutational patterns in cancer.
G4 structures obstruct DNA polymerase and *increase the risk of DNA damage and NAHR*.

In another study, @Eden2003 design a specific experiment to show that overall **hypomethylation of tumor-prone mice increased the rate of LOH**.
Several other studies detected significant association between genome instability and hypomethylation [@Wong2001;@Nakagawa2005], but it's **difficult to know** if there is any real link or just co-occurrence as the tumor develops.

Three-dimensional genome structure has also been linked to the location of somatic SVs, using chromosome conformation capture [@Fudenberg2011].

On the other hand, **somatic SV can affect important genes of the epigenetic machinery.**
Several genes (EXAMPLES) have been associated with cancer and are involved in histone modification(REF) or methylation(REF).
It's important to understand how SV affect those genes as they seem to have high functional impact.



## Mobile element insertion

L1 retro-transposition is a **common phenomenon** in some cancer, e.g. **epithelial tumors**.
However the assumption is that they become active once the cancer is initiated, thanks to derepression in the tumor tissue.
Hence they are not believed to initiate tumorigenesis but could contribute to the tumor progression and metastasis.

@Tubio2014 performed a survey of **somatic L1 activity across 12 cancer types**, focusing on the **transduction** of flanking sequences.
Around 50% of the tumors had at least one somatic L1 insertion.
Somatic retro-transposition was **most frequent in colon cancer (93%) and lung cancer (75%)**.
They found that 24% of events presented 3' transduction, **half** of which were **orphan transduction** (no L1 sequence inserted).
The size of the transduction was typically less than 1 kb but some were up to 12 kb.
They identified 72 of **'hot' L1 sources** that contributed to more than 95% of the somatic insertions.
Most of these were not known to be hot and are likely intact L1 with **low frequency in human populations**.
Four hot L1 produced 50% of all the transducted insertions.
In the tumors, these hot L1 tended to be **hypomethylated**.
They even observed **sequential retro-transposition** where a L1 jumps somewhere, then jumps again (then jumps again), taking with it part of the second (and third) region as transducted sequence.
This opens up new ways for a L1 insertion to bring regulation changes in a new insertion: it could jump once, grab a regulatory region, jump again and activate another region.
Still, they found no insertion with functional potential. For example, none of the genes with insertion were disregulated, nor expressing aberrant RNA.
Most of the insertions located in gene-poor regions.
Hence they concluded that the vast majority of somatic L1 insertion are **passenger variants**.

In more recent study by @Scott2016, a **driver somatic L1 insertion** in the APC gene contributed to the formation of **colorectal cancer** as the first of the classic two-hits mechanism.
The patient had a subtype of colorectal cancer (MSS) which is **known to start with the disruption of the APC gene**.
The insertion is located in the known mutation hotspot in exon 16 and there is a second hit (premature stop codon) in the other allele.
Both variants were validated and confirmed to be somatic and on different alleles.
They managed to identify the **sources** of this and other insertions and it ended up being **heterozygous L1s absent from the reference genome**.
Using the 1000 Genomes project MEI dataset they show that these hot L1s are present in **african-descent** genomes.
This suggest that the distribution of germline L1 across population might affect the risk to develop cancer (to a certain extent).
Finally they show the the promoter regions was hypomethylated in both the normal and tumor, maybe because of a SNP in one CpG.
In contrast to these hot L1s, other germline L1s were not expressed.

Another example from Shukla et al (2013) shows a **somatic L1 insertion associated with the upregulation of the ST18 gene in hepatocellular carcinoma**.
Apparently the insertion disrupted a repressor region in one of the gene's intron.

# Neurological disorders

De novo and inherited may contribute to the genetic vulnerability for conditions such as schizophrenia or autism [@Lee2010]

Alzheimer and Parkinson's disease have been associated with changes in gene dosage resulting from alterations in copy number [@Magi2010].

**Autism Spectrum Disorder** (ASD) is a neuro-developmental disorder characterized by social impairment, communication difficulties and restricted behavioral patterns.
This largely genetic condition has been linked to DNA alterations (including CNVs) in neuronal genes, but these accounts for only 20% of the cases.
The incidence of *de novo* CNVs in ASD family is significantly higher [@Ionita-Laza2009].

Novel deletions/duplications >100 kb were present in **15% of schizophrenia cases**, 3 times mores than in controls [@Ionita-Laza2009] (FIND ORIGINAL REF).

Reduced penetrance has been observed in some families, for example for inherited CNVs and autism [@Lee2010].

In @Hiatt2020, PacBio CCS found potentially pathogenic SVs in 2 out of 6 proband-parent trios with unexplained neurodevelopmental disorders (NDDs).
The families had negative short-read tests.
One of the SV is a *de novo* L1-mediated insertion which result in the duplication of exon 3 of *CDK5* and creates a frameshift.
Mosaicism is suspected because they also observed paternal reads without the insertion.
The other proband has some complex *de novo* translocations and inversions affecting 7 genes, including one (*MLLT3*) whose expression seems disrupted (~40% decrease).

## Epilepsy

Epilepsy has a **prevalence of ~1%** in the population, and a **lifetime incidence** up to **3%**.

One of the challenge seems to be the **heterogeneity of the phenotype**.
Even in locus that are clearly associated with epilepsy risk, there are examples of *unaffected carrier* parents, *incomplete penetrance* and *variable expressivity*.
The phenotypes of epilepsy can be complex, and there are **several different types of epilepsy**, e.g. generalized epilepsy, focal epilepsy, epileptic encephalopathy.
Apparently, the genetic basis of refractory and childhood-onset epilepsies remains elusive [@Helbig2014].
Also the **same single-gene mutation** can cause a **wide range of seizure types** [@Mefford2010].

**Recurrent microdeletions** were identified in **up to 3% of patients** with idiopathic generalized epilepsies and in 1% of focal epilepsies [@Helbig2014].
*15q13.3* and *16p13.11* are the *most frequent microdeletions*.
These variants are often transmitted from a healthy parent.
In addition, **up to 10% of patients** with various epilepsies **carry a unique and possibly pathogenic CNV**.
The presence of rare genic, sometimes *de novo*, CNVs was observed in several studies and different epilepsy types:

+ 8.9% of patients with generalized and focal epilepsies [@Mefford2010].
+ 7.9% of patients with epileptic encephalopathies [@Mefford2011].
+ 32% of patients with childhood epilepsy [@Helbig2014].
+ 4.8% of patients with *de novo* CNVs [@Mefford2015].
+ 11% of patients with absence epilepsies [@Addis2016].

**Rare CNVs**, sometimes *de novo*, were found in patients with **unclassified epilepsies and complex phenotypes** [@Helbig2014].
In this study, 222 patients were screened using aCGH/SNP-array for CNVs larger than 100 kb.
88 rare CNVs in 71 patients (31.9%) were discovered.
18 of these were considered **pathogenic**, and 15 **likely pathogenic**.
Out of the 55 CNVs of unknown significance the **deletion/duplication ratio was ~1:2**.
11 were *de novo* out the 42 that could be assessed, including all four variants larger than 3 Mb.
No novel recurrent CNVs larger than 100 kb were detected.

In other neurological disease such as **mental retardation**, the classic pattern is also micro-*deletions*, often flanked by *segmental duplications*, *de novo* in affected patients, *never seen* in controls.
There is also one example where a common inversion favors later deletion by NAHR [@Mefford2009].

# Congenital Heart Disease

Congenital Heart Disease (**CHD**) is the **most common survivable birth defect**.
Its incidence is estimated around *4-10 in 1,000 live births*.
The heart is the first organ to start developing in the human embryo. Or so CHD researchers claim. At least one of the first.
Individuals with CHD have increased risks for extra-cardiac congenital anomalies and neurodevelopmental problems.
Although there is a genetic side to the disease, most of the times there is no visible Mendelian inheritance.
Other factors exist, for example alcohol, infections or type 1 diabetes in the mother.

**Large chromosomal abnormalities** are historically associated with CHD.
Many **aneuploidy syndromes** have a some CHD synptoms: Down syndome (3x21), Edward syndrome (3x18), Patau syndrome (3x13), Turner syndrome (1xX), Klinefelter sndrome (XXY).

Tetralogy of Fallot (TOF) is a subtype of CHD with complex phenotype.

A molecular diagnosis can be made in up to one in eight CHD patients [@Costain2016].

One of the most established region is **22q11.2** where inherited and *de novo* CNVs have been associated with CHD.
It is flanked by segmental duplications which increases the risk of NAHR.
**1q21.1** is also emerging as an important CNV-associated region [@Costain2016].
Mostly duplications and in present in up to 1% of TOF patients.
Non-recurrent deletions of varying sizes in 8p23.1 are also found more and more.
In this region, *GATA4* gene haploinsufficiency has been associated with CHD for a long time.

In many studies, an **excess of rare CNVs** was found in non-syndromic CHD patients compared to control populations [@Costain2016].
Most of these studies focused on **large and/or genic CNVs**.
Some loci, e.g. 22q11.2, involve *both deletions and duplications*.
The same excess is observed when looking at *de novo* CNVs only.
However, most CNVs know to be pathogenic are inherited highlighting their **variable expression and reduced penetrance**. 

Why are CNVs important in CHD ?
One explanation is that large CNVs can simultaneously disrupt multiple genes, which might counter the redundancy of cardiac development pathways.

One of the desired clinical outcome is to predict potential neurodevelopmental issues.

Cardiomyopathies, vasculopathies and cardiac arrhythmias are not CHD ?

# Asthma

In @Ionita-Laza2009, Table 1 mention common deletions in *GSTM1* associated with increased risk (OR 1.59-1.89). *GSTM1* is a potent antioxydant whose deletion relates to asthma-related outcomes.

# Other diseases

1q21.1 CNVs can lead to another contiguous gene syndrome which present a large range of clinical presentations [@Lee2010].

ICF syndrome (Immunodeficiency, Centromeric instability and Facial anomalies) is characterized by an extension of pericentromeric satellites.
Hypomethylation of these satellites is thought to be the cause for this instability on chromosome 1, 9 and 16 [@Jeanpierre1993].

Hereditary nonpolyposis colorectal cancer (HNPCC), also known as Lynch syndrome, is a common autosomal dominant syndrome characterized by early age at onset, neoplastic lesions, and microsatellite instability (MSI).


@Lee2010 has a table listing examples of diseases associated with a SV, grouped by configuration. 
Here is a subset:

- Single gene
    - recessive: Duchenne/Becker muscular distrophy (*DMD*)
    - dominant: Sotos syndrome (*NSD1*)
    - dosage: Early-onset Alzheimer disease (*APP*), 22q autism (*SHANK3*), Psoriasis (*LCE3C*/*LCE3B*), 
    - position effect: Crohn disease (*IRGM*)
    - multiallelic: Crohn disease predisposition (*DEFB4*), Parkinson disease (*SNCA*), HIV/AIDS susceptibility/Kawasaki disease susceptibility/rheumatoid arthritis predisposition (*CCL3L1*)
- Multiple genes
    - recurrent: Velo-cardio-facial syndrome/DiGeorge syndrome (22q11.2 deletion)
    - non-recurrent: Potocki– Lupski syndrome [dup(17)(p11.2p11.2)]
    - heterogeneous: Autism, Bipolar disorder, Schizophrenia, Age-related macular degeneration, Tetralogy of Fallot
    - buffering or modifier effects: Spinal muscular atrophy (*SMN1*/*SMN2*)
    - epigenetic effects: Silver–Russell syndrome (11p15 duplication)
    - somatic mosaicism: Tuberous sclerosis (*TSC1*, *TSC2*)
    - somatic instability: Li-Fraumeni syndrome

# Loss of chromosome Y and diseases

Mosaic loss of chromosome Y (LOY) in blood in aging men has been known for several decades already.
It is estimated that **15% of males aged 70 or more** have a **LOY** affecting **10% or more of their blood cells** [@Forsberg2014].
Recent studies by the Dumanski Lab, found that LOY in blood has been **associated with risk for all-cause mortality, non-hematological cancers and Alzheimer Disease**.
They first used a SNP-arrays and a prospective study of 982 elderly men (aged 71-84) to investigate all-cause mortality and non-hematological cancers and found significantly higher risk for individuals with LOY.
Their models corrected for age, hypertension, exercise, smoking, BMI, diabetes, cholesterol levels and ancestry.
The median median survival time for men with LOY was *5.5 years shorter*.
These observations were replicated in another cohort of 488 men aged 70.
Their next study focused on Alzheimer disease and found a significantly higher risk for men with LOY.
They used the same prospective cohorts plus a AD case-control cohort also using SNP-arrays.
In this study they define LOY using the 99% quantile of the null distribution.
All three datasets support the increased risk for LOY carriers, controlling for age, APOE genotypes, smoking, BMI, diabetes, LDL/HDL, hypertension, exercise, education and alcohol consumption.
APOE genotypes didn't show any differences in term of LOY but were included in the case-control model.

One proposed mechanism explaining these association is that LOY is a marker of *biological age*, hence more accurate than the chronological age at estimating these risks.
They also propose that LOY could result in *defective immunosurveillance* which would be consistent with the AD and cancer phenotypes.
Anyway they propose that LOY become one of the markers used for personalized medicine to better assess one's risk of cancer or AD.


# Somatic mosaic CNVs

Mosaic CNVs have been observed in some diseases: Rubinstein-Taybi syndrome, tuberous sclerosis, aniridia [@Lee2010].

# Incomplete penetrance

Some syndromes have strong genotype-phenotype relationship (e.g. Williams-Beuren syndrome), while others some others result from very different CNVs (e.g. Down syndrome caused by parts of chr 21 to full trisomy) [@Lee2010].

Recurrent deletions and duplications at 15q13.3 have been associated with schizophrenia and epilepsy but are inherited in the majority of cases [@Lee2010].

Sometimes the effect of CNVs are so variable that they "evade syndromic classification", for example for 1q21.1 deletions [@Lee2010].

Some CNVs have a modifier (or buffer) effect: they are necessary but nor sufficient for a disease. 
For example, 1q21.1 deletion and thrombocytopaenia absent radius (TAR) [@Lee2010].

Epigenetic effects can also explain variable penetrance sometimes, like in Silver-Russell syndrome and (possibly) developmental verbal dyspraxia [@Lee2010].

# Mutational load

The concept that the aggregation of multiple CNVs can model diseases like ASD, bipolar disorder, schizophrenia, macular degeneration, tetralogy of fallot [@Lee2010].

In their review @Lee2010 mention Li-Fraumeni syndrome as a *striking prototype*.
This familial cancer syndrome is caused by mutations in *TP53* but evidence suggests the presence of modifier loci.
The amount and size of CNVs seem to correlate with carrier of TP53 mutations or to those developping cancer.

# Importance of understanding SV's impact

SVs are difficult to interpret but they can be informative and for example provide an explanation to patients and families about the challenges they faced [@Lee2010].
Monitoring SVs could also provide earlier diagnosis for conditions which can benefit from beneficial interventions, like autism.

In addition to including SVs in the interpretation, a *shift in mindset* is also necessary to look at the genomic level rather than genetic level [@Lee2010].
For example, in addition to looking for the one causative gene in a large CNV, we should also think at the gene/protein pathways, and the mutational load and modifier effects, at the epigenetic impact.

@Lee2010 mentions that only 5% of GWAS hits might be explained by CNVs and that it would mean that CNVs shouldn't account for much of the *missing heritability*.
`Do they mean _large_ CNVs here?`

# Bibliography

