---
sidebar: true
title: Structural Variation Impact
bibliography: [../../library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---


<div id="TOC">
<ul>
<li><a href="#gene-expression">Gene expression</a><ul>
<li><a href="#micro-rnas">Micro RNAs</a></li>
<li><a href="#genes-with-recurrent-cnv">Genes with recurrent CNV</a></li>
<li><a href="#enhancer-hijacking">Enhancer hijacking</a></li>
<li><a href="#methylation-trigerring">Methylation trigerring</a></li>
</ul></li>
<li><a href="#large-chromosomal-aberrations">Large chromosomal aberrations</a></li>
<li><a href="#association-studies-and-ld">Association studies and LD</a><ul>
<li><a href="#inversions">Inversions</a></li>
</ul></li>
<li><a href="#cancer">Cancer</a><ul>
<li><a href="#pan-cancer-patterns">Pan-Cancer patterns</a><ul>
<li><a href="#arm-level-aberrations">Arm-level aberrations</a></li>
<li><a href="#focal-scnvs">Focal sCNVs</a></li>
</ul></li>
<li><a href="#gene-fusion">Gene fusion</a></li>
<li><a href="#enhancer-hijacking-1">Enhancer hijacking</a></li>
<li><a href="#relation-with-epigenetic-marks">Relation with epigenetic marks</a></li>
<li><a href="#mobile-element-insertion">Mobile element insertion</a></li>
</ul></li>
<li><a href="#neurological-disorders">Neurological disorders</a><ul>
<li><a href="#epilepsy">Epilepsy</a></li>
</ul></li>
<li><a href="#congenital-heart-disease">Congenital Heart Disease</a></li>
<li><a href="#asthma">Asthma</a></li>
<li><a href="#other-diseases">Other diseases</a></li>
<li><a href="#loss-of-chromosome-y-and-diseases">Loss of chromosome Y and diseases</a></li>
<li><a href="#bibliography">Bibliography</a></li>
</ul>
</div>

<div id="gene-expression" class="section level1">
<h1>Gene expression</h1>
<p>Coding regions of <strong>800-1800 known genes</strong> are subject to <strong>variation in copy number</strong> <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006">2006</a>)</span>.
On average, <em>394 genes</em> are predicted to differ in term of copy number <em>between two individuals</em> <span class="citation">(Alkan et al. <a href="#ref-Alkan2009">2009</a>)</span>.
Different types of genes tend to be affected by common CNVs.
Genes involved in <em>immune response</em> or <em>environmental interaction</em> in general tend to be located in region of active CNV.
CNV seem to have played an important role in evolutionary adaptation <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006">2006</a>)</span>.</p>
<p><strong>SV can influence gene expression in several ways</strong>.
Variation in the copy number of the entire gene can lead to <strong>gene dosage</strong> effect although genes involved in common CNVs are assumed to be tolerant to it.
Most of the segmental duplications containing genes are tandem changes <span class="citation">(Alkan et al. <a href="#ref-Alkan2009">2009</a>)</span>.
SV on part of a gene may create <strong>new transcript variant</strong> through <em>exon shuffling</em>, or even <em>gene fusion</em>.
Variation in non-coding regions can affect <strong>regulatory regions or chromatin structure</strong>.
Copy number variants can affect the expression of genes <strong>up to 100kb</strong> away from the variant <span class="citation">(Raphael <a href="#ref-Raphael2012">2012</a>)</span>.
Finally, deletions can activate recessive allele through loss of heterozygosity.</p>
<p>Variation in non-coding regions can also affect regulatory regions or chromatin structure around genes.
For examples, non-coding CNVs have been associated with gene expression <span class="citation">(Stranger et al. <a href="#ref-Stranger2007b">2007</a>)</span> and can sometimes affect the expression of genes up to 100kb away from the variant <span class="citation">(Lower et al. <a href="#ref-Lower2009">2009</a>)</span>.
In <span class="citation">Stranger et al. (<a href="#ref-Stranger2007b">2007</a>)</span>, <em>53%</em> of the CNVs influencing gene expression were located outside of the gene.
When <span class="citation">Sudmant et al. (<a href="#ref-Sudmant2015a">2015</a>)</span> described the SVs in the 1000GP, they called eQTLs in the lymphoblastoid cell lines from the Geuvadis project and found 54 eQTLs with a SV association (0.56%) and 166 SVs in LD with SNV/indels eQTLs.
Most of these SV-eQTLs overlapped coding sequence but some were located in upstream regions.
A more recent study using deep WGS investigated SV-eQTLs in multiple tissues from the GTEx dataset <span class="citation">(Chiang et al. <a href="#ref-Chiang2017">2017</a>)</span> (see <a href="/2018/03/31/SVeQTLinGTEx/">more detailed review post</a>).
They found that 3.5-6.8% of eQTLs could be attributed to SVs and that SVs tend to have a larger effect size.
Most causal SVs were <strong>non-coding</strong> (89%) and were <strong>enriched</strong> in or close to <strong>regulatory regions</strong> (e.g. exons, TSS, TFBS, enhancers, gene 3’ end).
They also detected an enrichment of rare SVs close to genes with outlier expressions, suggesting that rare SVs could have a high-impact on gene expression.
Indeed, rare SVs were enriched close to gene expression outliers, especially large CNVs, duplications, and variants altering gene dosage.</p>
<p>Multi-allelic CNVs are more likely involved with gene dosage changes than one-copy CNVs <span class="citation">(Handsaker et al. <a href="#ref-Handsaker2015">2015</a>)</span>.
There, 88% of gene dosage changes were caused by common multi-allelic CNVs.
Gene dosage changes from these CNVs correlates with gene expression.</p>
<div id="micro-rnas" class="section level2">
<h2>Micro RNAs</h2>
<p>CNVs can also affect gene expression by <strong>disrupting miRNAs dosage</strong>.
These non-coding small RNAs often regulate genes and sometimes entire pathways expression.
There doesn’t seem to be convincing studies showing the direct effect of CNVs on miRNAs on gene expression (on disease).
One review just cited distinct studies showing miRNAs importance, CNVs importance, potential impact on brain disorders (also broad unsurprising evolutionary facts) <span class="citation">(Persengiev, Kondova, and Bontrop <a href="#ref-Persengiev2013">2013</a>)</span>.</p>
<p>One reference from Volinia 2010 apparently confirms the role of CNV-causing dysregulation of miRNAs in disease occurrence.
<strong>To read</strong></p>
</div>
<div id="genes-with-recurrent-cnv" class="section level2">
<h2>Genes with recurrent CNV</h2>
<p>Many genes are recurrently affected by copy number variation in normal genomes.
These genes must be haplosufficient (right?).
For example, <span class="citation">Alkan et al. (<a href="#ref-Alkan2009">2009</a>)</span> genes with full CNV : complement factor H-related complex (CFHR1-4), defensin gene family (DEFB103B), amylase gene family (AMY1A), as well as CCL3L1 and TBC1D3.</p>
<p>CNVs in SD are enriched for genes involved in <strong>sensory response, immune and inflammatory response, cell signaling and cell adhesion and structural proteins</strong> <span class="citation">(Cooper, Nickerson, and Eichler <a href="#ref-Cooper2007">2007</a>)</span>.</p>
<p>4867 genes had an exon impacted by SVs, when comparing the two human genome assemblies <span class="citation">(Pang et al. <a href="#ref-Pang2010">2010</a>)</span>.
Similarly, 5320 protein-coding genes overlapped a SV from the 1KGP overlapped <em>(manually computed from DGV variants and Gencode v19)</em> <span class="citation">(Mills et al. <a href="#ref-Mills2011">2011</a>)</span>.
Only 2229 protein-coding genes had an exon overlapping one of Genome STRiP 2.0 calls in more than 800 samples <em>(manually computed from their calls and Gencode v19)</em> <span class="citation">(Handsaker et al. <a href="#ref-Handsaker2015">2015</a>)</span>.</p>
</div>
<div id="enhancer-hijacking" class="section level2">
<h2>Enhancer hijacking</h2>
<p><strong>Enhancer hijacking</strong> happens when a regulatory region induces the <strong>ectopic expression</strong> of a gene it normally doesn’t regulate because of a <strong>CNV-mediated re-positioning</strong>.
An elegant study described this mechanism in <em>human limb malformations</em> using CRISPR/Cas genome editing and conformation experiments <span class="citation">(Lupiáñez et al. <a href="#ref-Lupianez2015">2015</a>)</span>.
In brief, three different phenotypes were caused by <strong>deletions</strong>, <strong>inversion</strong>/tandem duplication, or <strong>tandem duplications</strong>, that <strong>spanned the boundary of a topologically associated domains (TADs)</strong>.
After showing that the TAD architecture was conserved in mice, these SVs were recreated in mice using genome editing.
The ectopic expression of genes during limb development in mice was assessed by RNA-seq and in situ hybridization.
Using 4C-seq again, the <em>ectopic interaction</em> were also visible in patient-derived fibroblasts.
Investigating deeper the minimal region, genomic annotations and additional mice experiments helped identify 5 candidate enhancers potentially responsible for the normal and ectopic regulation.
Finally, further genome editing confirmed that the <strong>breaking of the TAD boundary</strong> (and insulation region) <strong>was the crucial factor</strong> rather than the distance between enhancers and genes.</p>
</div>
<div id="methylation-trigerring" class="section level2">
<h2>Methylation trigerring</h2>
<p><strong>Transposon insertion can trigger silencing through methylation</strong>.
One documented case was found by <a href="https://pubmed.ncbi.nlm.nih.gov/15269773/">Lippman et al. 2004</a> who showed in Arabidopsis that the gene for flowering time (FWA) was regulated to some extent by the methylation of nearby SINE-like TEs.
Another older observation in mice found that de novo retroviral insertion inactivates genes through de novo methylation of surrounding regions (<a href="https://pubmed.ncbi.nlm.nih.gov/2989695/">Jahner &amp; Jaenisch 1985</a>.</p>
<p>Focusing on evolutionary patterns, a study in Arabidopsis showed that fixed/old insertions tend to “lose” their methylation but polymorphic insertions don’t (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2720190/">Hollister &amp; Gaut 2009</a>).
Their results suggest negative selection against methylated insertions, potentially because of their effect on nearby genes.</p>
<p><span class="citation">Rebollo et al. (<a href="#ref-Rebollo2011">2011</a>)</span> compared H3K9me3 profiles of embryonic cells in two strains of mice, focusing around common or polymorphic copies of threes TE families.
In contrast to LINE or ETn/MusD, IAP copies tended to be more hypomethylated.
For polymorphic IAP copies, only the strain with the copy was hypomethylated.
<strong>The methylation decreased almost linearly up to 3 kbp from the insertion.</strong>
They found that it <strong>rarely spread into genes</strong>, hinting at some mechanisms that could stop methylation spreading.
Although the genome-wide analysis was performed on H3K9me3, they zoom in one insertion close to the B3galtl gene.
This time looking at DNA methylation, they show <strong>a clear example</strong> where the empty sites is completely unmethylated while the strains with the inserted IAP show methylation spreading to the CpG island about half the time.
<strong>The expression of this gene was indeed reduced</strong> in the strain with the IAP insertion.</p>
<p>Studying this in mice, <span class="citation">Rebollo et al. (<a href="#ref-Rebollo2012">2012</a>)</span> showed <strong>active genes tend to be protected and that methylation spreading from a nearby TE is rare</strong>.
They measured methylation at fixed endogenous retrovirus (ERV) copies as well as a few insertionally polymorphic copies.
Overall they found that ERV copies near the TSS are more likely hypomethylated.
They further interrogated the methylation at the gene promoter near methylated ERV copies to test if the methylation had spread.
Only 1 out of 29 cases had a sign of methylation spreading (and they knew about it from <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002301">a previous study</a>).
They observe a similar pattern of “methylation vs TSS distance” between fixed and polymorphic copies.
Part of the explanation lies in the action of H3K4me3 and CTCF, blocking the methylation spreading.</p>
<p>In a recent preprint, <a href="https://www.biorxiv.org/content/10.1101/2020.06.16.154047v2">Wyler et al 2020</a> tested this model in <em>Brachypodium distachyon</em> (<a href="https://en.wikipedia.org/wiki/Brachypodium_distachyon">grass</a>).
They used methylation and transcriptome profiling across 11 accessions.
They found that methylation in only a few TE families leads to lower gene expression.
However, they did find a few polymorphic insertions associated with gene expression changes across multiple accessions.
Take-home message seems to be: <strong>not a large effect globally but some families/insertions can silence nearby genes</strong>.</p>
</div>
</div>
<div id="large-chromosomal-aberrations" class="section level1">
<h1>Large chromosomal aberrations</h1>
<p>Early milestones listed in <span class="citation">Lee and Scherer (<a href="#ref-Lee2010">2010</a>)</span>:</p>
<ul>
<li>1959 Down syndrome/trisomy 21, Turner syndrome (45,X), Kleinfelter syndrome (47, XXY)</li>
<li>1961 Partial trisomy Down syndrome</li>
<li>1963 First inherited deletion syndrome: Cri-du-chat (5p-)</li>
</ul>
<p>Large CNVs that span multiple genes can cause ‘contiguous gene syndromes’ or genomic disorders (“old” ref in <span class="citation">Lee and Scherer (<a href="#ref-Lee2010">2010</a>)</span>)</p>
</div>
<div id="association-studies-and-ld" class="section level1">
<h1>Association studies and LD</h1>
<p>One approach to test potential effect of common SVs with phenotypes uses <strong>SNP in strong LD as markers</strong>.
Although it seemed at first glance that genotyped SNPs were in sufficient LD with many SVs, these analysis were <strong>biased toward a specific type of SVs</strong>.
Unfortunately, these SVs are <strong>not representative</strong> of the SV landscape and actually represent the least frequent class of SVs <span class="citation">(Sharp, Cheng, and Eichler <a href="#ref-Sharp2006">2006</a>)</span>.
For practical and interpretation purpose, these studies <em>focused on simple deletions</em>, avoiding sites of complex, multi-copy or tandem rearrangements.
Conversely, some SVs are found to be recurrent by being present in different haplotype regions, or resulting in both duplication and deletions, or displaying different breakpoints.</p>
<p>As reviewed in <span class="citation">Ionita-Laza et al. (<a href="#ref-Ionita-Laza2009">2009</a>)</span>, challenges in CNV association studies range from the <strong>signal-to-noise ratio</strong>, the difficulty to derive <strong>precise genotype</strong>.
Similar to other variants, other challenges comprise the sample size and power computation, multiple test correction, reproducibility, population stratification.</p>
<p>Two strategies have been used:</p>
<ol style="list-style-type: decimal">
<li>Inferring the <strong>genotype</strong> and performing <strong>standard tests</strong>.</li>
<li>Directly <strong>analyzing raw measurements</strong>.</li>
</ol>
<p>The former can result in a loss of information and power when the genotyping process is not exact.
The latter has more power but its results will need to be validated and investigated through genotyping.</p>
<p><span class="citation">Beyter et al. (<a href="#ref-Beyter2019">2019</a>)</span> discovered SVs using long-read sequencing in almost 2K Icelanders and imputed their presence in 166K Icelanders from SNP arrays.
They performed association tests with the phenotype available and notably found a 14 Kbp deletion that overlaps the first exon of PCSK9 and correlates with lower LDL cholesterol levels.</p>
<div id="inversions" class="section level2">
<h2>Inversions</h2>
<p>From <span class="citation">González et al. (<a href="#ref-Gonzalez2019">2019</a>)</span> introduction:</p>
<ul>
<li>A common inversion in 8p23.1 has been linked to obesity, autism, neuroticism.</li>
<li>Inversion at 17q21.31 was associated with Alzheimer, Parkinson diseases, heart failure.</li>
</ul>
<p><span class="citation">González et al. (<a href="#ref-Gonzalez2019">2019</a>)</span> imputed the presence of inversions in &gt;400K Europeans individuals and performed association tests agains 25 obesity-related traits.
Seven inversions were associated with multiple diseases.
Some inversions could be disrupting TADs.</p>
<p>An inversion in a parental chromosome may predispose to de novo CNVs <span class="citation">(Lee and Scherer <a href="#ref-Lee2010">2010</a>)</span>.
For example the deletion in 17q21.31 results from an inversion common in Europeans.
Other examples include Sotos syndrome or Williams-Beuren syndrome.
These results suggest that parents are sometimes needed to fully understand the origin of CNVs.</p>
</div>
</div>
<div id="cancer" class="section level1">
<h1>Cancer</h1>
<ul>
<li>How are SV/CNVs used to estimate cell heterogeneity ? How is this validated ?</li>
<li>Can we infer tumor evolution ? Or pooling samples, recurrent path in tumor evolution ?</li>
<li>How clear are the recurrence patterns usually ? Is it consistent across types ? Are the different breakpoints a real issue ?</li>
<li>How is recurrence defined in samples/cancer with many large abberations ?</li>
<li>How is DNA satellite expansion associated with different types of cancer ?</li>
<li>What is the importance of centromeric/telomeric regions in cancer ?</li>
<li>What types of transposable elements get activated in cancer ?</li>
<li>How is SV involved in epigenetics changes in the tumoral genome ?</li>
</ul>
<p>SVs can affect gene expression through genetic and epigenetics mechanisms <span class="citation">(Albertson et al. <a href="#ref-Albertson2003">2003</a>)</span>.</p>
<p>Genomic gains are particularly important to identify driver genes <span class="citation">(Dear <a href="#ref-Dear2009">2009</a>)</span>.
Moreover, duplication have been less studied than deletions.</p>
<p><span class="citation">De and Michor (<a href="#ref-De2011a">2011</a><a href="#ref-De2011a">a</a>)</span> found an enrichment of somatic CNV breakpoints in simple repeats, Alu and CR1.</p>
<p>Replication timing has also been associated with sCNVs formation <span class="citation">(De and Michor <a href="#ref-De2011">2011</a><a href="#ref-De2011">b</a>)</span>.</p>
<p><strong>Satellite DNA “Breaking of satellites’ silence” is a paradigm in cancerogenesis</strong> <span class="citation">(Rich, Ogryzko, and Pirozhkova <a href="#ref-Rich2014">2014</a>)</span>.
In cancer satellites are hypomethylated and are thought to induce peri-centromeric instability.
This might explain the observed over-expression of satellite repeats in tumors.</p>
<p>For example, <span class="citation">Ting et al. (<a href="#ref-Ting2011">2011</a>)</span> observed a 40-fold increase in pericentromeric satellites expression in pancreatic ductal adenocarcinomas.
Alpha satellites were the most abundant, HSATII highly specific to cancer.
The satellite expression also correlated with L1 expression.
The hypothesis is an alteration of the silencing mechanisms of these repeats.</p>
<div id="pan-cancer-patterns" class="section level2">
<h2>Pan-Cancer patterns</h2>
<div id="arm-level-aberrations" class="section level3">
<h3>Arm-level aberrations</h3>
<p>Copy number aberration is common in tumors from almost all cancer types.
Tumors sometimes harbor whole-genome doubling or, more frequently, chromosome arm-level gains or losses.
Aneuploidy, i.e. chromosome gain or loss, is seen in almost all cancer types although at varying frequencies <span class="citation">(Baudis <a href="#ref-Baudis2007">2007</a>; Kim and Marioni <a href="#ref-Kim2013b">2013</a>; Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.</p>
<p>Aggregating CGH data across 5,918 epithelial tumors, <span class="citation">Baudis (<a href="#ref-Baudis2007">2007</a>)</span> identified frequent arm-level gains in 8q, 20q, 1q, 3q, 5p, 7q and 17q; and frequent losses in 3p, 4q, 13q, 17p and 18q.
The median number of arm-level aberration per sample ranged from 0 in squamous skin neoplasias to 12 in small cell lung carcinomas <span class="citation">(Baudis <a href="#ref-Baudis2007">2007</a>)</span>.</p>
<p>To further interrogate the distribution of sCNVs in different types of cancer, <span class="citation">Beroukhim et al. (<a href="#ref-Beroukhim2010">2010</a>)</span> analyzed 3,131 cancer genomes from 26 different cancer types.
Although the frequency of sCNV decreases with their size, arm-level aberrations stood out as recurrent aberrations for both losses and gains and across all 26 cancer types studied
Arm-level aberrations were ~30 times more frequent than expected from the frequency-size relationship of other sCNVs.
In this pan-cancer survey, 25% of a typical cancer genome was affected by arm-level sCNVs.
In contrast to some focal sCNVs, arm-level sCNVs resulted in low-amplitude changes, most of the time a single copy loss/gain.
Interestingly, it appeared that chromosome arms had either somatic losses or gains, but rarely both.</p>
<p>Similar observations were made in a larger meta-analysis of 8,227 cancer CNV profiles from 107 CGH studies <span class="citation">(Kim and Marioni <a href="#ref-Kim2013b">2013</a>)</span>.
Chromosome arms preferentially showed losses or gains, with 1q, 5p, 7p, 3q and 20q most frequently gained and 4q, 6q, 8p, 13q and 17p most frequently lost.
Cancer types were clustered based on their arm-level aberration frequencies and formed three groups with similar developmental origins.
Five cancer of hematologic-origin clustered together; most of the epithelial cancers formed the second cluster; the third cluster contained four neuroepithelial cancers as well as sarcoma and RCC.
Arm-level aberrations in gene-rich arms tend to co-occur when concordant (gain-gain, loss-loss) <span class="citation">(Kim and Marioni <a href="#ref-Kim2013b">2013</a>)</span>.</p>
<p>Using a single detection platform across 4,934 cancers and 11 cancer types, The Cancer Genome Atlas described attempted to time sCNVs <span class="citation">(Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
For example, whole-genome doubling was observed in 37% of the tumors and tended to occur prior to other types of sCNVs.
The rate of both arm-level and focal sCNVs were higher in tumor that experienced whole-genome doubling.x
In term of arm-level aberrations, <span class="citation">Zack et al. (<a href="#ref-Zack2013">2013</a>)</span> found a median of 3 gains and 5 losses per tumors.</p>
<p>The pan-cancer studies described above didn’t include or comment on sex chromosomes in their analysis <span class="citation">(Baudis <a href="#ref-Baudis2007">2007</a>; Beroukhim et al. <a href="#ref-Beroukhim2010">2010</a>; Kim and Marioni <a href="#ref-Kim2013b">2013</a>; Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.</p>
</div>
<div id="focal-scnvs" class="section level3">
<h3>Focal sCNVs</h3>
<p><span class="citation">Li et al. (<a href="#ref-Li2012a">2012</a>)</span> investigated the association of <strong>19 genomic features with the frequency of somatic CNV breakpoints</strong> (SNP-array, 8 cancer types).
<strong>Evolutionary indel</strong> and <strong>substitution rates</strong> and <strong>conserved elements</strong> showed the strongest correlation.
Moreover they see <em>two classes</em> of breakpoint hotspots: the <em>cancer-type-specific</em> weakly correlated with genomic features; the <em>common</em> hotspots with higher correlation.
They suggest that they follow <strong>different evolutionary models</strong>: positive selection driven by cancer genes and non-adaptive evolution related to genomic context.
So the cancer-specific hotspots are driven by cancer genes (sometimes specific to the cancer type); but common hotspots are more often due to the region being more fragile and susceptible to sCNVs.
They used sequence features, DNA secondary structure motif, evolutionary features and functional annotation.
Cancer-type specific hotspots were enriched in cancer genes.
In their figure 2A we see interesting patterns.
First, some strong association are type-specific, e.g. SINE, exon density, PG4, CpG islands.
Then, <em>some cancers show opposite patterns</em>: SINE are enriched in sCNVs in ovarian cancer but depleted in kidney cancer.</p>
<p>Several studies observed <strong>shared sCNV hotspot</strong> location across <strong>different cancer types</strong> <span class="citation">(Li et al. <a href="#ref-Li2012a">2012</a>; Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
There are still hotspots specific to the different cancer types.</p>
<p><strong>Recurrent focal CNVs</strong> have been used previously to <strong>identify tumor-suppressor genes and oncogenes</strong> <span class="citation">(Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
The large sample size and diverse origin of the samples in the TCGA/ICGC consortium lead to the identification of common or cancer-specific hotspots of sCNVs <span class="citation">(Beroukhim et al. <a href="#ref-Beroukhim2010">2010</a>; Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
For example, <span class="citation">Zack et al. (<a href="#ref-Zack2013">2013</a>)</span> identified <em>140 regions with recurrent sCNVs</em> when pooling cancers from 11 different types.
sCNVs in those regions tended to be smaller and with higher amplitude.
They further identified <em>229 regions with cancer-type-specific recurrence</em> that were not significant in the pan-cancer analysis.</p>
<p>Large-sample size also allows researchers to look for <strong>anti-correlation patterns between pairs of regions</strong>.
<em>Positive</em> correlations with other genetic events may indicate functional <em>synergies</em>, whereas <em>anticorrelations</em> may indicate functional <em>redundancies</em> <span class="citation">(Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
For example, <span class="citation">Zack et al. (<a href="#ref-Zack2013">2013</a>)</span> found 7% of regions pairs to be anti-correlated.
In practice, they tested pairs of hotspot regions.
These pairs tend to physically interact and are suspected to have related functions.</p>
<p><strong>Ploidy and purity varies within and across cancer lineages</strong> <span class="citation">(Zack et al. <a href="#ref-Zack2013">2013</a>)</span>.
Cancers with whole-genome duplication have higher rate of every other types of sCNVs, and these sCNVs are predicted to occur after the WGD.</p>
<p><span class="citation">Zhang, Xu, and Frishman (<a href="#ref-Zhang2016">2016</a>)</span> used the same data and further characterized the genomic features explaining the location of sCNV breakpoints.
Briefly they counted the number of sCNA breakpoints in 1 Mb bins and regressed it against genomic features.
Their models completed previous studies and found new associations, the strongest being the <strong>distance to telomere and centromere</strong>.
Frequency of sCNAs decreased with the distance to telomere/centromere.
Among the positive significant associations: <strong>L1 coverage, SINE count, low-complexity repeat coverage, CpG island</strong>.
A more surprising one is the <strong>number of conserved elements</strong> (<em>they don’t discuss it so I’m not sure what to think of it</em>).
<strong>Direct repeats</strong>, which are a class of non-B conformation, were also positively associated and mentioned several times.
They decided to <strong>separate telomere-bound and interstitial sCNVs</strong> and show different breakpoint locations.</p>
</div>
</div>
<div id="gene-fusion" class="section level2">
<h2>Gene fusion</h2>
<p>A <strong>translocation</strong> is causing a gene fusion between BCR and ABL genes and is seen in a important fraction of patients with chronic myelogenous leukemia.
BCR-ABL is directly implicated in the development of the cancer <span class="citation">(Raphael <a href="#ref-Raphael2012">2012</a>)</span>.
<strong>Fusion genes are also recurrent in some solid tumors</strong>, such as prostate and lung cancers <span class="citation">(Raphael <a href="#ref-Raphael2012">2012</a>)</span>.
Another translocation in Burkitt’s lymphoma, activates MYC oncogene by <strong>fusing a promoter</strong> <span class="citation">(Raphael <a href="#ref-Raphael2012">2012</a>)</span>.</p>
</div>
<div id="enhancer-hijacking-1" class="section level2">
<h2>Enhancer hijacking</h2>
<p>There have been several examples of <strong>somatic CNV</strong> that change the structure of the <strong>regulatory region</strong> and induce <strong>ectopic activity of oncogenes</strong>.</p>
<p><span class="citation">Weischenfeldt et al. (<a href="#ref-Weischenfeldt2016">2016</a>)</span> developed a method to detect <strong>associations between sCNA breakpoints in a TAD and gene over-expression</strong>.
TADs (Topologically Associated Domains) are 3D domains (mean size 830 kb) that are supposed to be conserved across cell types and confine regulatory elements and targets.
They analyzed the 7K cancer genomes (26 subtypes) with SNP array and RNA-seq data in TCGA.
They first described a known cancer gene, TERT, which had been already found to be upregulated by such mechanisms.
Interestingly <strong>both deletion and duplication</strong> were involved in over-expression.
Then they describe and validate in much details to genes, IRS4 and IGF2.
In the first, <strong>deletions cluster right next to a regulatory region of IRS4</strong>.
In tumors with these deletions, the region was now <em>active</em> (H3K27ac).
They also showed that IRS4 is <em>likely a cancer gene</em> by over-expressing it and observing rapid tumor growth in mice.
In the second, <strong>single tandem duplications of IGF2 and a super-enhancer</strong> creates a <strong>novel chromatin domain</strong> with both IGF2 and the super-enhancer.
Again, they nicely validated that the region was active and touching (4C-seq) in tumor with the duplication.
Here they didn’t look at enhancer amplification or balanced translocation (soon with PCAWG).</p>
<p>These enhancer hijacking events are important because <strong>a single copy change usually can lead to large over-expression</strong>.
In contrast dosage effect due to full-gene CNV are (supposedly) as strong as the amplification.</p>
</div>
<div id="relation-with-epigenetic-marks" class="section level2">
<h2>Relation with epigenetic marks</h2>
<p>Somatic SVs can be associated with epigenetic changes and vice versa.
On one hand, we would like to know if the epigenetic state of a genome affects the formation of somatic SVs.</p>
<p><span class="citation">De and Michor (<a href="#ref-De2011a">2011</a><a href="#ref-De2011a">a</a>)</span> found that <strong>G-quaduplex(G4) sequences are enriched in somatic CNV breakpoints</strong> (SNP-array, 26 cancer types).
Moreover, these many sCNVs breakpoints shows abnormal hypomethylation of the G4 sequence.
They found evidence of G4-mediated SV and proposed that G4 hypomethylation could drive tissue-specific mutational patterns in cancer.
G4 structures obstruct DNA polymerase and <em>increase the risk of DNA damage and NAHR</em>.</p>
<p>In another study, <span class="citation">Eden et al. (<a href="#ref-Eden2003">2003</a>)</span> design a specific experiment to show that overall <strong>hypomethylation of tumor-prone mice increased the rate of LOH</strong>.
Several other studies detected significant association between genome instability and hypomethylation <span class="citation">(Wong et al. <a href="#ref-Wong2001">2001</a>; Nakagawa et al. <a href="#ref-Nakagawa2005">2005</a>)</span>, but it’s <strong>difficult to know</strong> if there is any real link or just co-occurrence as the tumor develops.</p>
<p>Three-dimensional genome structure has also been linked to the location of somatic SVs, using chromosome conformation capture <span class="citation">(Fudenberg et al. <a href="#ref-Fudenberg2011">2011</a>)</span>.</p>
<p>On the other hand, <strong>somatic SV can affect important genes of the epigenetic machinery.</strong>
Several genes (EXAMPLES) have been associated with cancer and are involved in histone modification(REF) or methylation(REF).
It’s important to understand how SV affect those genes as they seem to have high functional impact.</p>
</div>
<div id="mobile-element-insertion" class="section level2">
<h2>Mobile element insertion</h2>
<p>L1 retro-transposition is a <strong>common phenomenon</strong> in some cancer, e.g. <strong>epithelial tumors</strong>.
However the assumption is that they become active once the cancer is initiated, thanks to derepression in the tumor tissue.
Hence they are not believed to initiate tumorigenesis but could contribute to the tumor progression and metastasis.</p>
<p><span class="citation">Tubio et al. (<a href="#ref-Tubio2014">2014</a>)</span> performed a survey of <strong>somatic L1 activity across 12 cancer types</strong>, focusing on the <strong>transduction</strong> of flanking sequences.
Around 50% of the tumors had at least one somatic L1 insertion.
Somatic retro-transposition was <strong>most frequent in colon cancer (93%) and lung cancer (75%)</strong>.
They found that 24% of events presented 3’ transduction, <strong>half</strong> of which were <strong>orphan transduction</strong> (no L1 sequence inserted).
The size of the transduction was typically less than 1 kb but some were up to 12 kb.
They identified 72 of <strong>‘hot’ L1 sources</strong> that contributed to more than 95% of the somatic insertions.
Most of these were not known to be hot and are likely intact L1 with <strong>low frequency in human populations</strong>.
Four hot L1 produced 50% of all the transducted insertions.
In the tumors, these hot L1 tended to be <strong>hypomethylated</strong>.
They even observed <strong>sequential retro-transposition</strong> where a L1 jumps somewhere, then jumps again (then jumps again), taking with it part of the second (and third) region as transducted sequence.
This opens up new ways for a L1 insertion to bring regulation changes in a new insertion: it could jump once, grab a regulatory region, jump again and activate another region.
Still, they found no insertion with functional potential. For example, none of the genes with insertion were disregulated, nor expressing aberrant RNA.
Most of the insertions located in gene-poor regions.
Hence they concluded that the vast majority of somatic L1 insertion are <strong>passenger variants</strong>.</p>
<p>In more recent study by <span class="citation">Scott et al. (<a href="#ref-Scott2016">2016</a>)</span>, a <strong>driver somatic L1 insertion</strong> in the APC gene contributed to the formation of <strong>colorectal cancer</strong> as the first of the classic two-hits mechanism.
The patient had a subtype of colorectal cancer (MSS) which is <strong>known to start with the disruption of the APC gene</strong>.
The insertion is located in the known mutation hotspot in exon 16 and there is a second hit (premature stop codon) in the other allele.
Both variants were validated and confirmed to be somatic and on different alleles.
They managed to identify the <strong>sources</strong> of this and other insertions and it ended up being <strong>heterozygous L1s absent from the reference genome</strong>.
Using the 1000 Genomes project MEI dataset they show that these hot L1s are present in <strong>african-descent</strong> genomes.
This suggest that the distribution of germline L1 across population might affect the risk to develop cancer (to a certain extent).
Finally they show the the promoter regions was hypomethylated in both the normal and tumor, maybe because of a SNP in one CpG.
In contrast to these hot L1s, other germline L1s were not expressed.</p>
<p>Another example from Shukla et al (2013) shows a <strong>somatic L1 insertion associated with the upregulation of the ST18 gene in hepatocellular carcinoma</strong>.
Apparently the insertion disrupted a repressor region in one of the gene’s intron.</p>
</div>
</div>
<div id="neurological-disorders" class="section level1">
<h1>Neurological disorders</h1>
<p>De novo and inherited may contribute to the genetic vulnerability for conditions such as schizophrenia or autism <span class="citation">(Lee and Scherer <a href="#ref-Lee2010">2010</a>)</span></p>
<p>Alzheimer and Parkinson’s disease have been associated with changes in gene dosage resulting from alterations in copy number <span class="citation">(Magi et al. <a href="#ref-Magi2010">2010</a>)</span>.</p>
<p><strong>Autism Spectrum Disorder</strong> (ASD) is a neuro-developmental disorder characterized by social impairment, communication difficulties and restricted behavioral patterns.
This largely genetic condition has been linked to DNA alterations (including CNVs) in neuronal genes, but these accounts for only 20% of the cases.
The incidence of <em>de novo</em> CNVs in ASD family is significantly higher <span class="citation">(Ionita-Laza et al. <a href="#ref-Ionita-Laza2009">2009</a>)</span>.</p>
<p>Novel deletions/duplications &gt;100 kb were present in <strong>15% of schizophrenia cases</strong>, 3 times mores than in controls <span class="citation">(Ionita-Laza et al. <a href="#ref-Ionita-Laza2009">2009</a>)</span> (FIND ORIGINAL REF).</p>
<div id="epilepsy" class="section level2">
<h2>Epilepsy</h2>
<p>Epilepsy has a <strong>prevalence of ~1%</strong> in the population, and a <strong>lifetime incidence</strong> up to <strong>3%</strong>.</p>
<p>One of the challenge seems to be the <strong>heterogeneity of the phenotype</strong>.
Even in locus that are clearly associated with epilepsy risk, there are examples of <em>unaffected carrier</em> parents, <em>incomplete penetrance</em> and <em>variable expressivity</em>.
The phenotypes of epilepsy can be complex, and there are <strong>several different types of epilepsy</strong>, e.g. generalized epilepsy, focal epilepsy, epileptic encephalopathy.
Apparently, the genetic basis of refractory and childhood-onset epilepsies remains elusive <span class="citation">(Helbig et al. <a href="#ref-Helbig2014">2014</a>)</span>.
Also the <strong>same single-gene mutation</strong> can cause a <strong>wide range of seizure types</strong> <span class="citation">(Mefford et al. <a href="#ref-Mefford2010">2010</a>)</span>.</p>
<p><strong>Recurrent microdeletions</strong> were identified in <strong>up to 3% of patients</strong> with idiopathic generalized epilepsies and in 1% of focal epilepsies <span class="citation">(Helbig et al. <a href="#ref-Helbig2014">2014</a>)</span>.
<em>15q13.3</em> and <em>16p13.11</em> are the <em>most frequent microdeletions</em>.
These variants are often transmitted from a healthy parent.
In addition, <strong>up to 10% of patients</strong> with various epilepsies <strong>carry a unique and possibly pathogenic CNV</strong>.
The presence of rare genic, sometimes <em>de novo</em>, CNVs was observed in several studies and different epilepsy types:</p>
<ul>
<li>8.9% of patients with generalized and focal epilepsies <span class="citation">(Mefford et al. <a href="#ref-Mefford2010">2010</a>)</span>.</li>
<li>7.9% of patients with epileptic encephalopathies <span class="citation">(Mefford et al. <a href="#ref-Mefford2011">2011</a>)</span>.</li>
<li>32% of patients with childhood epilepsy <span class="citation">(Helbig et al. <a href="#ref-Helbig2014">2014</a>)</span>.</li>
<li>4.8% of patients with <em>de novo</em> CNVs <span class="citation">(Mefford <a href="#ref-Mefford2015">2015</a>)</span>.</li>
<li>11% of patients with absence epilepsies <span class="citation">(Addis et al. <a href="#ref-Addis2016">2016</a>)</span>.</li>
</ul>
<p><strong>Rare CNVs</strong>, sometimes <em>de novo</em>, were found in patients with <strong>unclassified epilepsies and complex phenotypes</strong> <span class="citation">(Helbig et al. <a href="#ref-Helbig2014">2014</a>)</span>.
In this study, 222 patients were screened using aCGH/SNP-array for CNVs larger than 100 kb.
88 rare CNVs in 71 patients (31.9%) were discovered.
18 of these were considered <strong>pathogenic</strong>, and 15 <strong>likely pathogenic</strong>.
Out of the 55 CNVs of unknown significance the <strong>deletion/duplication ratio was ~1:2</strong>.
11 were <em>de novo</em> out the 42 that could be assessed, including all four variants larger than 3 Mb.
No novel recurrent CNVs larger than 100 kb were detected.</p>
<p>In other neurological disease such as <strong>mental retardation</strong>, the classic pattern is also micro-<em>deletions</em>, often flanked by <em>segmental duplications</em>, <em>de novo</em> in affected patients, <em>never seen</em> in controls.
There is also one example where a common inversion favors later deletion by NAHR <span class="citation">(Mefford and Eichler <a href="#ref-Mefford2009">2009</a>)</span>.</p>
</div>
</div>
<div id="congenital-heart-disease" class="section level1">
<h1>Congenital Heart Disease</h1>
<p>Congenital Heart Disease (<strong>CHD</strong>) is the <strong>most common survivable birth defect</strong>.
Its incidence is estimated around <em>4-10 in 1,000 live births</em>.
The heart is the first organ to start developing in the human embryo. Or so CHD researchers claim. At least one of the first.
Individuals with CHD have increased risks for extra-cardiac congenital anomalies and neurodevelopmental problems.
Although there is a genetic side to the disease, most of the times there is no visible Mendelian inheritance.
Other factors exist, for example alcohol, infections or type 1 diabetes in the mother.</p>
<p><strong>Large chromosomal abnormalities</strong> are historically associated with CHD.
Many <strong>aneuploidy syndromes</strong> have a some CHD synptoms: Down syndome (3x21), Edward syndrome (3x18), Patau syndrome (3x13), Turner syndrome (1xX), Klinefelter sndrome (XXY).</p>
<p>Tetralogy of Fallot (TOF) is a subtype of CHD with complex phenotype.</p>
<p>A molecular diagnosis can be made in up to one in eight CHD patients <span class="citation">(Costain, Silversides, and Bassett <a href="#ref-Costain2016">2016</a>)</span>.</p>
<p>One of the most established region is <strong>22q11.2</strong> where inherited and <em>de novo</em> CNVs have been associated with CHD.
It is flanked by segmental duplications which increases the risk of NAHR.
<strong>1q21.1</strong> is also emerging as an important CNV-associated region <span class="citation">(Costain, Silversides, and Bassett <a href="#ref-Costain2016">2016</a>)</span>.
Mostly duplications and in present in up to 1% of TOF patients.
Non-recurrent deletions of varying sizes in 8p23.1 are also found more and more.
In this region, <em>GATA4</em> gene haploinsufficiency has been associated with CHD for a long time.</p>
<p>In many studies, an <strong>excess of rare CNVs</strong> was found in non-syndromic CHD patients compared to control populations <span class="citation">(Costain, Silversides, and Bassett <a href="#ref-Costain2016">2016</a>)</span>.
Most of these studies focused on <strong>large and/or genic CNVs</strong>.
Some loci, e.g. 22q11.2, involve <em>both deletions and duplications</em>.
The same excess is observed when looking at <em>de novo</em> CNVs only.
However, most CNVs know to be pathogenic are inherited highlighting their <strong>variable expression and reduced penetrance</strong>.</p>
<p>Why are CNVs important in CHD ?
One explanation is that large CNVs can simultaneously disrupt multiple genes, which might counter the redundancy of cardiac development pathways.</p>
<p>One of the desired clinical outcome is to predict potential neurodevelopmental issues.</p>
<p>Cardiomyopathies, vasculopathies and cardiac arrhythmias are not CHD ?</p>
</div>
<div id="asthma" class="section level1">
<h1>Asthma</h1>
<p>In <span class="citation">Ionita-Laza et al. (<a href="#ref-Ionita-Laza2009">2009</a>)</span>, Table 1 mention common deletions in <em>GSTM1</em> associated with increased risk (OR 1.59-1.89). <em>GSTM1</em> is a potent antioxydant whose deletion relates to asthma-related outcomes.</p>
</div>
<div id="other-diseases" class="section level1">
<h1>Other diseases</h1>
<ul>
<li>How is association tested ? How many such studies have been performed ?</li>
</ul>
<p>CNVs have been associated with autism, schizophrenia, Crohn’s disease, rheumatoid arthritis, type 1 diabetes, obesity and numerous developmental diseases <span class="citation">(Zarrei et al. <a href="#ref-Zarrei2015">2015</a>)</span>.</p>
<p>One of the first duplication associated with a disease was found by Lupski and colleagues <span class="citation">(Baker <a href="#ref-Baker2012">2012</a>)</span>.
They found that hereditary neuropathy Charcot-Marie-Tooth disease was associated with the presence of an extra copy of a particular genetic region.
Their paper was rejected from <em>Nature</em> and <em>Science</em> right away.
The deletion of the same locus causes hereditary neuropathy with liability to pressure palsies (HNPP).
Both are mediated by NAHR.</p>
<p>Williams-Beuren Syndrome is a mendelian disorder characterized by deletion at 7q11.23.
DiGeorge/velocardiofacial syndrom is characterized by deletion of 22q11.2.</p>
<p>ICF syndrome (Immunodeficiency, Centromeric instability and Facial anomalies) is characterized by an extension of pericentromeric satellites.
Hypomethylation of these satellites is thought to be the cause for this instability on chromosome 1, 9 and 16 <span class="citation">(Jeanpierre et al. <a href="#ref-Jeanpierre1993">1993</a>)</span>.</p>
<p>Hereditary nonpolyposis colorectal cancer (HNPCC), also known as Lynch syndrome, is a common autosomal dominant syndrome characterized by early age at onset, neoplastic lesions, and microsatellite instability (MSI).</p>
<p><span class="citation">Ionita-Laza et al. (<a href="#ref-Ionita-Laza2009">2009</a>)</span> cites associations between <strong>common CNVs and risk for HIV infections, autoimmune disease and asthma</strong>.</p>
<p><strong>Disease-related CNVs are not restricted to rare variants</strong> and the <em>genetic risk tend to be higher than for SNPs</em> <span class="citation">(Ionita-Laza et al. <a href="#ref-Ionita-Laza2009">2009</a>)</span>.</p>
<p><strong>Genetic pleiotropy</strong> might exists <span class="citation">(Ionita-Laza et al. <a href="#ref-Ionita-Laza2009">2009</a>)</span>.
For example, gains and losses in <em>CLL3L1</em> are associated with HIV or Rheumatoid Arthritis. Similar for <em>DEFB4</em> and Crohn disease and Psoriasis.</p>
<p>Deletion of haplo-insufficient genes can lead to disease.
For example, deletion of <em>LCE3B</em> and <em>LCE3C</em> is a risk factor for psoriasis <span class="citation">(Lee and Scherer <a href="#ref-Lee2010">2010</a>)</span>.</p>
<p><span class="citation">Lee and Scherer (<a href="#ref-Lee2010">2010</a>)</span> has a table listing examples of diseases associated with a SV, grouped by configuration.
Here is a subset:</p>
<ul>
<li>Single gene
<ul>
<li>recessive: Duchenne/Becker muscular distrophy (<em>DMD</em>)</li>
<li>dominant: Sotos syndrome (<em>NSD1</em>)</li>
<li>dosage: Early-onset Alzheimer disease (<em>APP</em>), 22q autism (<em>SHANK3</em>), Psoriasis (<em>LCE3C</em>/<em>LCE3B</em>),</li>
<li>position effect: Crohn disease (<em>IRGM</em>)</li>
<li>multiallelic: Crohn disease predisposition (<em>DEFB4</em>), Parkinson disease (<em>SNCA</em>), HIV/AIDS susceptibility/Kawasaki disease susceptibility/rheumatoid arthritis predisposition (<em>CCL3L1</em>)</li>
</ul></li>
<li>Multiple genes
<ul>
<li>recurrent: Velo-cardio-facial syndrome/DiGeorge syndrome (22q11.2 deletion)</li>
<li>non-recurrent: Potocki– Lupski syndrome [dup(17)(p11.2p11.2)]</li>
<li>heterogeneous: Autism, Bipolar disorder, Schizophrenia, Age-related macular degeneration, Tetralogy of Fallot</li>
<li>buffering or modifier effects: Spinal muscular atrophy (<em>SMN1</em>/<em>SMN2</em>)</li>
<li>epigenetic effects: Silver–Russell syndrome (11p15 duplication)</li>
<li>somatic mosaicism: Tuberous sclerosis (<em>TSC1</em>, <em>TSC2</em>)</li>
<li>somatic instability: Li-Fraumeni syndrome</li>
</ul></li>
</ul>
</div>
<div id="loss-of-chromosome-y-and-diseases" class="section level1">
<h1>Loss of chromosome Y and diseases</h1>
<p>Mosaic loss of chromosome Y (LOY) in blood in aging men has been known for several decades already.
It is estimated that <strong>15% of males aged 70 or more</strong> have a <strong>LOY</strong> affecting <strong>10% or more of their blood cells</strong> <span class="citation">(Forsberg et al. <a href="#ref-Forsberg2014">2014</a>)</span>.
Recent studies by the Dumanski Lab, found that LOY in blood has been <strong>associated with risk for all-cause mortality, non-hematological cancers and Alzheimer Disease</strong>.
They first used a SNP-arrays and a prospective study of 982 elderly men (aged 71-84) to investigate all-cause mortality and non-hematological cancers and found significantly higher risk for individuals with LOY.
Their models corrected for age, hypertension, exercise, smoking, BMI, diabetes, cholesterol levels and ancestry.
The median median survival time for men with LOY was <em>5.5 years shorter</em>.
These observations were replicated in another cohort of 488 men aged 70.
Their next study focused on Alzheimer disease and found a significantly higher risk for men with LOY.
They used the same prospective cohorts plus a AD case-control cohort also using SNP-arrays.
In this study they define LOY using the 99% quantile of the null distribution.
All three datasets support the increased risk for LOY carriers, controlling for age, APOE genotypes, smoking, BMI, diabetes, LDL/HDL, hypertension, exercise, education and alcohol consumption.
APOE genotypes didn’t show any differences in term of LOY but were included in the case-control model.</p>
<p>One proposed mechanism explaining these association is that LOY is a marker of <em>biological age</em>, hence more accurate than the chronological age at estimating these risks.
They also propose that LOY could result in <em>defective immunosurveillance</em> which would be consistent with the AD and cancer phenotypes.
Anyway they propose that LOY become one of the markers used for personalized medicine to better assess one’s risk of cancer or AD.</p>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Addis2016">
<p>Addis, Laura, Richard E. Rosch, Antonio Valentin, Andrew Makoff,..., and Deb K. Pal. 2016. “Analysis of Rare Copy Number Variation in Absence Epilepsies.” <em>Neurology. Genetics</em>. <a href="https://doi.org/10.1212/NXG.0000000000000056">https://doi.org/10.1212/NXG.0000000000000056</a>.</p>
</div>
<div id="ref-Albertson2003">
<p>Albertson, Donna G, Colin Collins, Frank McCormick, and Joe W Gray. 2003. “Chromosome Aberrations in Solid Tumors.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng1215">https://doi.org/10.1038/ng1215</a>.</p>
</div>
<div id="ref-Alkan2009">
<p>Alkan, Can, Jeffrey M Kidd, Tomas Marques-Bonet, Gozde Aksay,..., and Evan E Eichler. 2009. “Personalized Copy Number and Segmental Duplication Maps Using Next-Generation Sequencing.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.437">https://doi.org/10.1038/ng.437</a>.</p>
</div>
<div id="ref-Baker2012">
<p>Baker, Monya. 2012. “Structural Variation: The Genome’s Hidden Architecture.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/nmeth.1858">https://doi.org/10.1038/nmeth.1858</a>.</p>
</div>
<div id="ref-Baudis2007">
<p>Baudis, Michael. 2007. “Genomic Imbalances in 5918 Malignant Epithelial Tumors: An Explorative Meta-Analysis of Chromosomal CGH Data.” <em>BMC Cancer</em>. <a href="https://doi.org/10.1186/1471-2407-7-226">https://doi.org/10.1186/1471-2407-7-226</a>.</p>
</div>
<div id="ref-Beroukhim2010">
<p>Beroukhim, Rameen, Craig H Mermel, Dale Porter, Guo Wei,..., and Matthew Meyerson. 2010. “The Landscape of Somatic Copy-Number Alteration Across Human Cancers.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature08822">https://doi.org/10.1038/nature08822</a>.</p>
</div>
<div id="ref-Beyter2019">
<p>Beyter, Doruk, Helga Ingimundardottir, Hannes Eggertsson, Eythor Bjornsson,..., and Kari Stefansson. 2019. “Long Read Sequencing of 1,817 Icelanders Provides Insight into the Role of Structural Variants in Human Disease.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/848366">https://doi.org/10.1101/848366</a>.</p>
</div>
<div id="ref-Chiang2017">
<p>Chiang, Colby, Alexandra J Scott, Joe R Davis, Emily K Tsang,..., and Ira M Hall. 2017. “The Impact of Structural Variation on Human Gene Expression.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3834">https://doi.org/10.1038/ng.3834</a>.</p>
</div>
<div id="ref-Cooper2007">
<p>Cooper, Gregory M., Deborah A. Nickerson, and Evan E. Eichler. 2007. “Mutational and Selective Effects on Copy-Number Variants in the Human Genome.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng2054">https://doi.org/10.1038/ng2054</a>.</p>
</div>
<div id="ref-Costain2016">
<p>Costain, Gregory, Candice K Silversides, and Anne S Bassett. 2016. “The Importance of Copy Number Variation in Congenital Heart Disease.” <em>Npj Genomic Medicine</em>. <a href="https://doi.org/10.1038/npjgenmed.2016.31">https://doi.org/10.1038/npjgenmed.2016.31</a>.</p>
</div>
<div id="ref-De2011a">
<p>De, Subhajyoti, and Franziska Michor. 2011a. “DNA Replication Timing and Long-Range DNA Interactions Predict Mutational Landscapes of Cancer Genomes.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/nbt.2030">https://doi.org/10.1038/nbt.2030</a>.</p>
</div>
<div id="ref-De2011">
<p>———. 2011b. “DNA Secondary Structures and Epigenetic Determinants of Cancer Genome Evolution.” <em>Nature Structural &amp; Molecular Biology</em>. <a href="https://doi.org/10.1038/nsmb.2089">https://doi.org/10.1038/nsmb.2089</a>.</p>
</div>
<div id="ref-Dear2009">
<p>Dear, Paul H. 2009. “Copy-Number Variation: The End of the Human Genome?” <em>Trends in Biotechnology</em>. <a href="https://doi.org/10.1016/j.tibtech.2009.05.003">https://doi.org/10.1016/j.tibtech.2009.05.003</a>.</p>
</div>
<div id="ref-Eden2003">
<p>Eden, Amir, François Gaudet, Alpana Waghmare, and Rudolf Jaenisch. 2003. “Chromosomal Instability and Tumors Promoted by DNA Hypomethylation.” <em>Science (New York, N.Y.)</em>. <a href="https://doi.org/10.1126/science.1083557">https://doi.org/10.1126/science.1083557</a>.</p>
</div>
<div id="ref-Forsberg2014">
<p>Forsberg, Lars A., Chiara Rasi, Niklas Malmqvist, Hanna Davies,..., and Jan P. Dumanski. 2014. “Mosaic Loss of Chromosome Y in Peripheral Blood Is Associated with Shorter Survival and Higher Risk of Cancer.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.2966">https://doi.org/10.1038/ng.2966</a>.</p>
</div>
<div id="ref-Fudenberg2011">
<p>Fudenberg, Geoff, Gad Getz, Matthew Meyerson, and Leonid A. Mirny. 2011. “High Order Chromatin Architecture Shapes the Landscape of Chromosomal Alterations in Cancer.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/nbt.2049">https://doi.org/10.1038/nbt.2049</a>.</p>
</div>
<div id="ref-Gonzalez2019">
<p>González, Juan R, Carlos Ruiz-Arenas, Alejandro Cáceres, Ignasi Morán,..., and Luis A Pérez-Jurado. 2019. “Polymorphic Inversions Underlie the Shared Genetic Susceptibility to Prevalent Common Diseases.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/859280">https://doi.org/10.1101/859280</a>.</p>
</div>
<div id="ref-Handsaker2015">
<p>Handsaker, Robert E, Van DorenVanessa, Jennifer R Berman, Giulio Genovese,..., and Steven a McCarroll. 2015. “Large Multiallelic Copy Number Variations in Humans.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3200">https://doi.org/10.1038/ng.3200</a>.</p>
</div>
<div id="ref-Helbig2014">
<p>Helbig, Ingo, Marielle E.M. Swinkels, Emmelien Aten, Almuth Caliebe,..., and Bobby P.C. Koeleman. 2014. “Structural Genomic Variation in Childhood Epilepsies with Complex Phenotypes.” <em>European Journal of Human Genetics</em>. <a href="https://doi.org/10.1038/ejhg.2013.262">https://doi.org/10.1038/ejhg.2013.262</a>.</p>
</div>
<div id="ref-Ionita-Laza2009">
<p>Ionita-Laza, Iuliana, Angela J. Rogers, Christoph Lange, Benjamin A. Raby, and Charles Lee. 2009. “Genetic Association Analysis of Copy-Number Variation (CNV) in Human Disease Pathogenesis.” <em>Genomics</em>. <a href="https://doi.org/10.1016/j.ygeno.2008.08.012">https://doi.org/10.1016/j.ygeno.2008.08.012</a>.</p>
</div>
<div id="ref-Jeanpierre1993">
<p>Jeanpierre, M, C Turleau, A Aurias, M Prieur,..., and E Viegas-Pequignot. 1993. “An Embryonic-Like Methylation Pattern of Classical Satellite DNA Is Observed in ICF Syndrome.” <em>Human Molecular Genetics</em>.</p>
</div>
<div id="ref-Kim2013b">
<p>Kim, Jong Kyoung, and John C Marioni. 2013. “Inferring the Kinetics of Stochastic Gene Expression from Single-Cell RNA-Sequencing Data.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2013-14-1-r7">https://doi.org/10.1186/gb-2013-14-1-r7</a>.</p>
</div>
<div id="ref-Lee2010">
<p>Lee, Charles, and Stephen W. Scherer. 2010. “The Clinical Context of Copy Number Variation in the Human Genome.” <em>Expert Reviews in Molecular Medicine</em>. <a href="https://doi.org/10.1017/S1462399410001390">https://doi.org/10.1017/S1462399410001390</a>.</p>
</div>
<div id="ref-Li2012a">
<p>Li, Yudong, Li Zhang, Robyn L. Ball, Xinle Liang,..., and Han Liang. 2012. “Comparative Analysis of Somatic Copy-Number Alterations Across Different Human Cancer Types Reveals Two Distinct Classes of Breakpoint Hotspots.” <em>Human Molecular Genetics</em>. <a href="https://doi.org/10.1093/hmg/dds340">https://doi.org/10.1093/hmg/dds340</a>.</p>
</div>
<div id="ref-Lower2009">
<p>Lower, Karen M, Jim R Hughes, De GobbiMarco, Shirley Henderson,..., and Douglas R Higgs. 2009. “Adventitious Changes in Long-Range Gene Expression Caused by Polymorphic Structural Variation and Promoter Competition.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.0909331106">https://doi.org/10.1073/pnas.0909331106</a>.</p>
</div>
<div id="ref-Lupianez2015">
<p>Lupiáñez, Daŕio G., Katerina Kraft, Verena Heinrich, Peter Krawitz,..., and Stefan Mundlos. 2015. “Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2015.04.004">https://doi.org/10.1016/j.cell.2015.04.004</a>.</p>
</div>
<div id="ref-Magi2010">
<p>Magi, Alberto, Matteo Benelli, Giuseppina Marseglia, Genni Nannetti,..., and Francesca Torricelli. 2010. “A Shifting Level Model Algorithm That Identifies Aberrations in Array-CGH Data.” <em>Biostat</em>. <a href="https://doi.org/10.1093/biostatistics/kxp051">https://doi.org/10.1093/biostatistics/kxp051</a>.</p>
</div>
<div id="ref-Mefford2015">
<p>Mefford, Heather. 2015. “Copy Number Variant Analysis from Exome Data in 349 Patients with Epileptic Encephalopathy.” <em>Annals of Neurology</em>. <a href="https://doi.org/10.1002/ana.24457">https://doi.org/10.1002/ana.24457</a>.</p>
</div>
<div id="ref-Mefford2009">
<p>Mefford, Heather C., and Evan E. Eichler. 2009. “Duplication Hotspots, Rare Genomic Disorders, and Common Disease.” <em>Current Opinion in Genetics &amp; Development</em>. <a href="https://doi.org/10.1016/j.gde.2009.04.003">https://doi.org/10.1016/j.gde.2009.04.003</a>.</p>
</div>
<div id="ref-Mefford2010">
<p>Mefford, Heather C., Hiltrud Muhle, Philipp Ostertag, von SpiczakSarah,..., and Evan E. Eichler. 2010. “Genome-Wide Copy Number Variation in Epilepsy: Novel Susceptibility Loci in Idiopathic Generalized and Focal Epilepsies.” <em>PLoS Genetics</em>. <a href="https://doi.org/10.1371/journal.pgen.1000962">https://doi.org/10.1371/journal.pgen.1000962</a>.</p>
</div>
<div id="ref-Mefford2011">
<p>Mefford, Heather C., Simone C. Yendle, Cynthia Hsu, Joseph Cook,..., and Ingrid E. Scheffer. 2011. “Rare Copy Number Variants Are an Important Cause of Epileptic Encephalopathies.” <em>Annals of Neurology</em>. <a href="https://doi.org/10.1002/ana.22645">https://doi.org/10.1002/ana.22645</a>.</p>
</div>
<div id="ref-Mills2011">
<p>Mills, Ryan E., Klaudia Walter, Chip Stewart, Robert E. Handsaker,..., and Jan O. Korbel. 2011. “Mapping Copy Number Variation by Population-Scale Genome Sequencing.” <em>Nature</em>. <a href="https://doi.org/10.1038/nature09708">https://doi.org/10.1038/nature09708</a>.</p>
</div>
<div id="ref-Nakagawa2005">
<p>Nakagawa, Tohru, Yae Kanai, Saori Ushijima, Tadaichi Kitamura,..., and Setsuo Hirohashi. 2005. “DNA Hypomethylation on Pericentromeric Satellite Regions Significantly Correlates with Loss of Heterozygosity on Chromosome 9 in Urothelial Carcinomas.” <em>The Journal of Urology</em>. <a href="https://doi.org/10.1097/01.ju.0000141577.98902.49">https://doi.org/10.1097/01.ju.0000141577.98902.49</a>.</p>
</div>
<div id="ref-Pang2010">
<p>Pang, Andy W, Jeffrey R MacDonald, Dalila Pinto, John Wei,..., and Stephen W Scherer. 2010. “Towards a Comprehensive Structural Variation Map of an Individual Human Genome.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2010-11-5-r52">https://doi.org/10.1186/gb-2010-11-5-r52</a>.</p>
</div>
<div id="ref-Persengiev2013">
<p>Persengiev, Stephan, Ivanela Kondova, and Ronald Bontrop. 2013. “Insights on the Functional Interactions Between miRNAs and Copy Number Variations in the Aging Brain.” <em>Frontiers in Molecular Neuroscience</em>. <a href="https://doi.org/10.3389/fnmol.2013.00032">https://doi.org/10.3389/fnmol.2013.00032</a>.</p>
</div>
<div id="ref-Raphael2012">
<p>Raphael, Benjamin J. 2012. “Chapter 6: Structural Variation and Medical Genomics.” <em>PLoS Computational Biology</em>. <a href="https://doi.org/10.1371/journal.pcbi.1002821">https://doi.org/10.1371/journal.pcbi.1002821</a>.</p>
</div>
<div id="ref-Rebollo2011">
<p>Rebollo, Rita, Mohammad M. Karimi, Misha Bilenky, Liane Gagnier,..., and Dixie L. Mager. 2011. “Retrotransposon-Induced Heterochromatin Spreading in the Mouse Revealed by Insertional Polymorphisms.” <em>PLoS Genetics</em>. <a href="https://doi.org/10.1371/journal.pgen.1002301">https://doi.org/10.1371/journal.pgen.1002301</a>.</p>
</div>
<div id="ref-Rebollo2012">
<p>Rebollo, Rita, Katharine Miceli-Royer, Ying Zhang, Sharareh Farivar,..., and Dixie L Mager. 2012. “Epigenetic Interplay Between Mouse Endogenous Retroviruses and Host Genes.” <em>Genome Biology</em>. <a href="https://doi.org/10.1186/gb-2012-13-10-r89">https://doi.org/10.1186/gb-2012-13-10-r89</a>.</p>
</div>
<div id="ref-Rich2014">
<p>Rich, J, V V Ogryzko, and I V Pirozhkova. 2014. “Satellite DNA and Related Diseases.” <em>Biopolymers and Cell</em>. <a href="https://doi.org/10.7124/bc.00089E">https://doi.org/10.7124/bc.00089E</a>.</p>
</div>
<div id="ref-Scott2016">
<p>Scott, Emma C., Eugene J. Gardner, Ashiq Masood, Nelson T. Chuang,..., and Scott E. Devine. 2016. “A Hot L1 Retrotransposon Evades Somatic Repression and Initiates Human Colorectal Cancer.” <em>Genome Research</em>. <a href="https://doi.org/10.1101/gr.201814.115">https://doi.org/10.1101/gr.201814.115</a>.</p>
</div>
<div id="ref-Sharp2006">
<p>Sharp, Andrew J., Ze Cheng, and Evan E. Eichler. 2006. “Structural Variation of the Human Genome.” <em>Annual Review of Genomics and Human Genetics</em>. <a href="https://doi.org/10.1146/annurev.genom.7.080505.115618">https://doi.org/10.1146/annurev.genom.7.080505.115618</a>.</p>
</div>
<div id="ref-Stranger2007b">
<p>Stranger, Barbara E, Matthew S Forrest, Mark Dunning, Catherine E Ingle,..., and Emmanouil T Dermitzakis. 2007. “Expression Phenotypes.” <em>Recherche</em>. <a href="https://doi.org/DOI: 10.1126/science.1137223">https://doi.org/DOI: 10.1126/science.1137223</a>.</p>
</div>
<div id="ref-Sudmant2015a">
<p>Sudmant, P. H., S. Mallick, B. J. Nelson, F. Hormozdiari,..., and E. E. Eichler. 2015. “Global Diversity, Population Stratification, and Selection of Human Copy Number Variation.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aab3761">https://doi.org/10.1126/science.aab3761</a>.</p>
</div>
<div id="ref-Ting2011">
<p>Ting, David T, Doron Lipson, Suchismita Paul, Brian W Brannigan,..., and Daniel A Haber. 2011. “Aberrant Overexpression of Satellite Repeats in Pancreatic and Other Epithelial Cancers.” <em>Science (New York, N.Y.)</em>. <a href="https://doi.org/10.1126/science.1200801">https://doi.org/10.1126/science.1200801</a>.</p>
</div>
<div id="ref-Tubio2014">
<p>Tubio, Jose M.C. C., Yilong Li, Young Seok Ju, Inigo Martincorena,..., and Peter J. Campbell. 2014. “Extensive Transduction of Nonrepetitive DNA Mediated by L1 Retrotransposition in Cancer Genomes.” <em>Science</em>. <a href="https://doi.org/10.1126/science.1251343">https://doi.org/10.1126/science.1251343</a>.</p>
</div>
<div id="ref-Weischenfeldt2016">
<p>Weischenfeldt, Joachim, Taronish Dubash, Alexandros P. Drainas, Balca R. Mardin,..., and Jan O. Korbel. 2016. “Pan-Cancer Analysis of Somatic Copy-Number Alterations Implicates IRS4 and IGF2 in Enhancer Hijacking.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.3722">https://doi.org/10.1038/ng.3722</a>.</p>
</div>
<div id="ref-Wong2001">
<p>Wong, Nathalie, Wai Chun Lam, Paul Bo San Lai, Elizabeth Pang,..., and Philip James Johnson. 2001. “Hypomethylation of Chromosome 1 Heterochromatin DNA Correlates with Q-Arm Copy Gain in Human Hepatocellular Carcinoma.” <em>American Journal of Pathology</em>. <a href="https://doi.org/10.1016/S0002-9440(10)61718-X">https://doi.org/10.1016/S0002-9440(10)61718-X</a>.</p>
</div>
<div id="ref-Zack2013">
<p>Zack, Travis I., Steven E. Schumacher, Scott L. Carter, Andrew D. Cherniack,..., and Rameen Beroukhim. 2013. “Pan-Cancer Patterns of Somatic Copy Number Alteration.” <em>Nature Genetics</em>. <a href="https://doi.org/10.1038/ng.2760">https://doi.org/10.1038/ng.2760</a>.</p>
</div>
<div id="ref-Zarrei2015">
<p>Zarrei, Mehdi, Jeffrey R. Macdonald, Daniele Merico, and Stephen W. Scherer. 2015. “A Copy Number Variation Map of the Human Genome.” <em>Nature Reviews Genetics</em>. <a href="https://doi.org/10.1038/nrg3871">https://doi.org/10.1038/nrg3871</a>.</p>
</div>
<div id="ref-Zhang2016">
<p>Zhang, Yanping, Hongen Xu, and Dmitrij Frishman. 2016. “Genomic Determinants of Somatic Copy Number Alterations Across Human Cancers.” <em>Human Molecular Genetics</em>. <a href="https://doi.org/10.1093/hmg/ddv623">https://doi.org/10.1093/hmg/ddv623</a>.</p>
</div>
</div>
</div>
