---
sidebar: true
title: Structural Variation
bibliography: [../../static/library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---


# Introduction - Importance of Structural Variation

With our current knowledge about genomic variation, *the* human genome has ceased to exist as a defined entity and is instead becoming a fluid continuum [@Dear2009].

**5% of the genome** is known to display CNVs, i.e. more than SNVs.
CNVs represent around 50 Mbp of difference per human genome [@Baker2012].
The amount of the genome that have been seen hosting CNVs varies across studies.
CNVs spanned 136.6 Mb/4.8% (273 Mb/9.5%) of the genome in a recent stringent (inclusive) CNV map constructed from DGV [@Zarrei2015].
CNVs spanned 112.7 Mb/3.7% of the genome in an aCGH survey across 41 samples [@Conrad2010].
On average, 24 Mb/0.78% of the genome were CNV between two samples.
@Pang2010 compared directly the assemblies of the human genome and found **48.8 Mb** (1.2%) of the genome to be affected by SVs.
More recentlty, the 1000 Genomes Project Consortium observed on average **18.4 Mb of SV** per diploid genome [@Sudmant2015a].
mCNVs (11.3 Mb) and large deletions (5.6 Mb) contribute the most to this number.
Although SVs represent less than *0.1%* of the variant sites in a typical genome, they affect more bases [@Auton2015].

In the 1000 Genome Project, **a typical genome contains 2,100-2,500 SVs**: a thousands large deletions, 160 CNV, 915 Alu insertions, 128 L1 insertions, 51 SVA insertions, 4 nuclear mitochondrial DNA variants, and 10 inversions [@Auton2015].

Similar to SNPs, the **majority of SVs in the population are rare** (65% with VAF *<0.2%*) [@Auton2015;@Sudmant2015a].
Nevretheless, the majority of variants in one genome are common.

| Study                   | Data         | SV     | Mb    | SV/sample | Mb/sample |
| ---                     | ---          | ---    | ---   | ---       | ---       |
| @Conrad2010             | 41 aCGH      | 8,600  | 112.7 | 1,100     | 24        |
| @Pang2010               | 2 assemblies | ?      | 48.8  | ?         | 48.8      |
| @Altshuler2010          | ?            | ?      | ?     | ?         | ?         |
| @Mills2011              | 185 WGS      | 28,000 | *?*   | *?*       | *?*       |
| @Altshuler2012          | ?            | ?      | ?     | ?         | ?         |
| @Francioli2014          | 750 WGS      | 27,500 | *?*   | *?*       | *?*       |
| @Zarrei2015             | DGV          | 12,000 | 136.6 | -         | -         |
| @Zarrei2015 inclusive   | DGV          | 24,000 | 273   | -         | -         |
| @Handsaker2015          | 849 WGS      | 8,659  | *?*   | *?*       | *?*       |
| @Auton2015 Sudmant2015a | 2,504 WGS    | 69,000 | ?     | 2,300     | 18.4      |

SV estimates. *italic*: computed manually from their results.



# Different SV types

+ What are the different types of SV ?
+ What the expected/known size distribution of the SV in the genome ?
+ What is the expected/known location of SV in the genome ?
+ Is it different for different SV types ?
+ Does it look like we are missing something ?

The different SV type are commonly **defined with respect to the reference genome**.
Another definition takes into account the ancestral state of the genome.
An insertion compared to the reference genome could also be a deletion in the reference genome.
While 21% of the deletions in @Mills2011 are actually insertions in the reference genome, 60% were clear deletions relative to the ancestral state.

Simple SV types includes **insertions, deletions, duplications, inversion and translocations**.
Deletions and duplications are also called **Copy Number Variation** (CNV).
Complex SVs look like a mixture simple SVs.

## Copy Number Variation

A comprehensive study of the selection bias on Human CNVs was performed by @Nguyen2006.
They started from the null hypothesis that CNVs arise uniformly in the genome and are selectively neutral.
They found that **CNVs occur more frequently near telomeres and centromeres**, are **enriched in protein-coding genes and tandem repeats** but **not in high GC content regions**.
Moreover CNV genes display a higher synonymous and non-synonymous substitution rate and are **under-represented in Mendelian disease genes**.
In fact human **CNV genes** encodes a disproportionately large number of **secreted, olfactory and immunity proteins**.
To be precise, the enrichment in tandem repeat (e.g. micro-satellites) is specific to the CNVs within 2 Mb of the centromeres and telomeres.
They also noticed no significant enrichment of RNA genes, interspersed repeats, CpG islands or GC content.
They concluded from this that the observed patterns were not due to sequence context.

@Zarrei2015 found *immune response, olfactory receptors, starch and sucrose metabolism, vesicle and Golgi apparatus pathways* to be **enriched in CNVs**.
Conversely, **copy number stable** regions were enriched in *protein phosphorylation, signal transduction, protein degradation, transcript regulation, intra-cellular transport, cell differentiation* genes.
They also found lincRNA and promoters enriched while enhancer were under-represented.

### Deletions and insertions

Deletions and insertions are the most frequent and best characterized type of SVs in the human genome.
Data suggests that the **number of insertions/deletions increases exponentially with decreasing event size** [@Sharp2006].

As expected, many deletions display **signs of negative selection**.
Deletion are significantly under-represented in coding genes, pointing at a deleterious effect and purifying selection [@Sharp2006;@Cooper2007].
In @Mills2011, high-frequency deletions were under-represented in protein-coding sequence.
Exceptions exist, e.g. molecular and environmental interaction genes.
Rare deletions seem to be present more often than SNPs, again because of a potentially stronger purifying selection [@Sharp2006].

There are more deletions detected than duplication [@Mills2011;@Fu2010].
Still it is not clear how much this trend is affected by detection power.

### Tandem repeats

Tandem repeats are composed of **variable numbers of serially repeated cassettes** [@Sharp2006].
Some repeats units can be several Kbp long.
Several **genes** with wide CNV are located **within tandemly repeated arrays**, e.g. AMY1A, GSTM1, \\(\alpha-\\)/\\(\beta-\\) defensins.
Micro-satellites expansion has also been documented within such arrays.

Many of the tandem repeat arrays might be highly variable in copy number between individuals [@Warburton2008].

**Tandem repeats in centromere** have **essential functions** for establishing **heterochromatin formation** and **chromosome segregation** [@Warburton2008].
Their function also seem to involve RNA interference-mediated chromatin modifications.

Some large tandem repeat arrays are locus specific.
The largest found by @Warburton2008 was 246 kb large, and *"reflects the heroic efforts made in sequencing the Y chromosome"*.

Microsatellites and variable number of tandem repeats can be highly polymorphic and **used as genetic markers** [@Warburton2008].

**Short tandem repeats** (STR) or microsatellites are repetitive elements of **2-6 nucleotides** and consist of approximately a *quarter million loci* in the human genome [@Gymrek2012].
They create secondary structure **prone to replication slippage** and results in **high variability**.
STRs are involved in different **genetic disorders** and used in **forensics**, genetic anthropology [@Gymrek2012].
They have also been used to trace cell lineages in cancer samples.

The **rate of STR polymorphism is highly correlated with the repeat unit length**, as well as with the length in the reference genome [@Gymrek2012].

## Inversions

Inversions might be one of the *least characterized SV type*.
Again, detected inversions are associated with the presence of segmental duplications at their flanking regions.
Moreover, **insertion/deletion at their breakpoint location are present in many inversions**, hence this SV type is often not fully unbalanced.

Inversions are **enriched in chromosome X**, maybe due to its enrichment in large inverted repeat structures [@Sharp2006].

Although inversions are mainly balanced, they can induce phenotypic effect by disrupting coding regions spanning their breakpoints, or genes dependent on their relative positions.
Eventually it is also recognized that **inversions can confer a predisposition for further chromosomal rearrangements** [@Sharp2006;@Cooper2007].
Such effects have been observed with deletions but also translocations.

Inversion have an interesting effect on a region haplotype conservation.
Indeed, local recombination seems suppressed after an inversion.
This phenomenon is thought to derive from a failure to recover from lethal recombination in inverted regions, rather than an actual reduction in recombination frequency.
Suppressed local recombination could actually be used to detect inversions by identifying and genotyping haplotypes with abnormal segregation.


## Transchromosomal SVs

Translocations have been **challenging** to study.
Even with recent sequencing, potential translocation are often filtered to limit false-positive rate [@Sharp2006].

Some fixed transchromosomal SVs have been identified but they are likely more often deleterious, affecting meiotic segregation.
For example, olfactory receptor are located in nearly half of all human sub-telomeres.
Other genes, e.g. zinc-finger and immunoglobulin heavy-chain gene families, are also located in multiple human sub-telomeres [@Sharp2006].

Common translocations usually involve p arms of acrocentric chromosomes.
For example, 15p pericentromeric satellite III into chromosome 14 short arm.

## Complex SVs

Complex SVs are observed **in tumors** as well as in the normal population.

### Somatic SVs

In @Malhotra2013, complex SVs are defined as SVs with discordant read clusters with **3 or more breakpoints** (distant from at least 1 Mbp).
Moreover, they define **chromothripsis** as a *single mutational event with 10 or more breakpoints*.
They found complex SVs to be **common in tumors**, with 75% of them exhibiting at least one.
The breakpoints of these complex SVs were **not enriched in known SV hotspots**, segmental duplication or repetitive elements.
The incidence of chromothripsis differed across tumor types (e.g. over-represented in glioblastoma) and apparently not correlated by the number of breakpoints.
They describe chromothripsis as a more extreme manifestation of common events rather than a different mechanism.
NHEJ after DNA double-strand breaks seems to be the predominant mechanism of complex SV formation.

Complex SVs can originate from a single mutational event or the accumulation of mutations in a region.
A **few copy number states** might suggest **one single mutational event** [@Malhotra2013].
74% of single-mutational complex SVs showed copy number alteration, suggesting that many events might be *falsely detected as independent events* in array or RD-based approaches.

Complex SVs in tumors seemed to be slightly more frequent than simple SVs [@Malhotra2013].
They might appear earlier or be under stronger selection.

### Germline SVs

Chromosome Y or region 5q13.3 with the Spinal Muscular Atrophy locus, were **very difficult to assemble** and are expected to segregate divergent alleles in the population.
SMA region, 1-2 Mbp in size, has two known variants that diverge by multiple rearrangements and small insertion/deletions.
*Only one male donor was used to assemble Y chromosome in the reference genome* because of its variability and high repeated/palindromic content.

@Collins2017 describe the distribution of complex SVs and chromothripsis in individuals with ASD.
They used long-insert reads (105X, ~3.5 kb IS) and linked-reads on 689 ASD patients.
They found that complex SVs are larger and rarer and usually involve inversion (84.4%)
A morbid genome has on average 14 complex SVs.
They also observed chromoanagenesis where chromosome is pulverized and reorfanized.
Chromoanagenesis encompasses chromothripsis, chromoplexy and chromoanasynthesis.
In the morbid genome they found exmaples of balanced germline chromothripsis and somatic chromoanasynthesis.

SVs found in @Collins2017 were compared to six recent WGS studies (1KGP, Chaisson-PacBio, Brand, Pendleton-Pacbio, Handsaker-1KGP, GoNL) DGV and InvFEST.
38.1% of the SVs had not been reported previously.

## SV hotspots

CNV hotspots cluster in pericentromeric and subtelomeric regions [@Cooper2007].
However this represents only ~1/3 of all hotspots [@Cooper2007].

CNVs are **enriched in segmental duplication, exons and mobile elements** [@Cooper2007].
The enrichment in exons is surprising to me; in @Cooper2007 they computed it in term of density (proportion of affected nucleotide) that might be driven by a few gene-rich regions.
CNVs are also more likely to occur near ancestrally duplicated sequence (**duplication shadowing**) [@Cooper2007].


# Bibliography

