---
sidebar: true
title: Whole-Genome Sequencing
bibliography: [../../library-small.bib]
link-citations: true
weight: 5
output:
  blogdown::html_page:
    toc: true
---

# Bias characterization

## GC bias

@Benjamini2012 described thoroughly the **bias in read coverage** due to **GC content** of a region. 
They showed that **GC-rich and GC-poor regions** tend to have a **reduced read coverage** and that it might be due to a decreased efficiency of the DNA amplification. 
Moreover they observed different strength in different samples, highlighting the **sample-specific** aspect of GC bias. 

@Talevich2014 also mentioned an effect of GC content on *hybridization* and attributed differences in the bias strength across samples to different DNA quality and hybridization efficiency.

## Repeats

Repetitiveness of genomic regions affect directly read mapping and might decrease the amount of reads mapped with good quality. 
There could also be an effect during **library preparation**, when **Cot-1 DNA** is used to block repetitive DNA [@Talevich2014]. 
This could explain how repeat-based bias could change from sample to sample. 

## Correction

@Cheung2011 proposed a method, *BEADS* to correct sequencing bias. @Hashimoto2014 too.

# Bibliography

